{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"authorship_tag":"ABX9TyPYS7obRlysUvbjAjbzE4+U"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","execution_count":1,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"RHsFdMACkun3","executionInfo":{"status":"ok","timestamp":1710094572494,"user_tz":420,"elapsed":22584,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"5af6be10-5aa1-4675-d03c-50f5cb4977e3"},"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}],"source":["from google.colab import drive\n","drive.mount('/content/drive')"]},{"cell_type":"code","source":["! wget https://dougspeed.com/wp-content/uploads/ldak5.2.linux_.zip\n","! unzip ldak5.2.linux_.zip\n","! ./ldak5.2.linux"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"HJSqDl4HlAOe","executionInfo":{"status":"ok","timestamp":1710094615813,"user_tz":420,"elapsed":5312,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"acb0f9c0-04e9-4b7d-baae-16e50be30270"},"execution_count":2,"outputs":[{"output_type":"stream","name":"stdout","text":["--2024-03-10 18:16:50--  https://dougspeed.com/wp-content/uploads/ldak5.2.linux_.zip\n","Resolving dougspeed.com (dougspeed.com)... 35.214.77.229\n","Connecting to dougspeed.com (dougspeed.com)|35.214.77.229|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 9265672 (8.8M) [application/zip]\n","Saving to: ‘ldak5.2.linux_.zip’\n","\n","ldak5.2.linux_.zip  100%[===================>]   8.84M  2.47MB/s    in 3.6s    \n","\n","2024-03-10 18:16:54 (2.47 MB/s) - ‘ldak5.2.linux_.zip’ saved [9265672/9265672]\n","\n","Archive:  ldak5.2.linux_.zip\n","  inflating: ldak5.2.linux           \n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 0 pairs of arguments:\n","\n","Error, each command requires a main argument; here is a list of main arguments\n","\n","Heritability analysis using individual-level data:\n","\n","--cut-weights <folder> - cut predictors into sections ready for calculating weightings\n","--calc-weights <folder> - calculate weightings for one section\n","--join-weights <folder> - join weightings across sections\n","--calc-weights-all <folder> - calculate weightings for all sections in one step\n","\n","--cut-kins <folder> - cut predictors into partitions ready for calculating kinships\n","--calc-kins <folder> - calculate kinship matrix for one partition\n","--join-kins <folder> - join kinship matrices across partitions\n","--calc-kins-direct <output> - calculate kinship matrix in one step\n","\n","--reml <output> - regress phenotype on kinships and/or regions\n","--calc-blups <output> - calculate BLUP effect size estimates (using results from --reml)\n","--he <output> - perform Haseman Elston Regression (an alternative to --reml)\n","--pcgc <output> - perform PCGC Regression (recommended instead of --reml for binary traits)\n","--fast-he <output> - perform fast (but approximate) Haseman Elston Regression\n","--fast-pcgc <output> - perform fast (but approximate) PCGC Regression\n","\n","Association testing:\n","\n","--linear <output> - perform standard or mixed-model linear regression\n","--logistic <output> - perform logistic regression\n","--solve-null <output> - perform two-part mixed-model linear regression (for complex models)\n","\n","--cut-genes <folder> - cut predictors into genes (or fixed-length chunks)\n","--calc-genes-kins <folder> - calculate kinships for each gene/chunk in one partition\n","--calc-genes-reml <folder> - calculate association test for each gene in one partition\n","--join-genes-reml <folder> - join association test results across partitions (also computes more accurate p-values)\n","\n","Heritability analysis using summary statistics:\n","\n","--calc-tagging <output> - calculate tagging file for use with --sum-hers or --sum-cors\n","--join-tagging <output> - join tagging files across predictors\n","--merge-tagging <output> - combine tagging files across categories\n","--reduce-tagging <output> - extract categories from a tagging file\n","\n","--sum-hers <output> - estimate heritabilities from summary statistics\n","--sum-cors <output> - estimate genetic correlations from summary statistics\n","--calc-exps <output> - calculate expected heritability tagged by predictors (using results from --sum-hers)\n","--calc-posts <output> - calculate posterior estimate of predictor effect sizes (using results from --calc-exps)\n","\n","Prediction:\n","\n","--ridge <output> - construct a ridge regression prediction model\n","--bolt <output> - construct a Bolt prediction model\n","--bayesr <output> -  construct a BayesR prediction model\n","--elastic <output> -  construct an elastic-net prediction model\n","\n","--calc-cors <output> - calculate predictor-predictor correlations for use with --mega-prs\n","--join-cors <output> - join correlations across predictors\n","--pseudo-summaries <output> - generate partial summary statistics (used to train MegaPRS prediction models)\n","--mega-prs <output> - construct lasso, ridge regression, Bolt and BayesR prediction models from summary statistics\n","\n","Other features:\n","\n","--thin <output> - prune predictors based on pairwise correlations\n","--thin-tops <output> - prune highly-associated predictors based on pairwise correlations\n","--find-tags <output> - search for the predictors most correlated with those in a scorefile\n","--remove-tags <output> - search for all predictors correlated with those in a target file\n","\n","--filter <output> - fiter samples based on relatedness\n","--add-grm <output> - combine kinship matrices\n","--sub-grm <output> - subtract from one kinship matrix the contributions of other kinship matrices\n","--convert-gz <output> - convert kinship matrix saved in gz (old GCTA) format\n","--convert-raw <output> - convert kinship matrix saved as a text file\n","--calc-sim-grm <output> - calculate correlations between pairs of kinship matrices\n","\n","--pca <output> - compute the principal component axes of a kinship matrix\n","--calc-pca-loads <output> - calculate the principal component predictor loadings (using results from --pca)\n","--decompose <output> - eigen-decompose a kinship matrix\n","--adjust-grm <output> - adjust a kinship matrix for covariates\n","--gxemm-iid / --gxemm-free <output> - calculate environmental kinship matrices\n","\n","--calc-stats <output> - calculate predictor allele frequencies, call-rates and possibly information scores\n","--calc-scores <output> - calculate one or more polygenic risk scores\n","--make-phenos <output> - simulate phenotypes\n","--make-snps <output> - simulate SNP data\n","\n","--jackknife <output> - measure prediction accuracy, obtaining standard deviations via block jackknifing\n","--cut-folds <output> - divide samples in preparation for performing K-fold cross-validation\n","--find-gaussian <output> - estimate the selection-related parameter alpha (using results from --sum-hers)\n","\n","--make-bed / --make-sp / --make-sped / --make-speed / --make-gen <outfile> - convert to bed / sp / sped / speed / gen format\n","--condense-bed / condense-sp / condense-sped / condense-speed <outfile> - condense to bed / sp / sped / speed format\n","--calc-sim-data <outfile> - calculate concordance between two datasets\n","\n"]}]},{"cell_type":"code","source":["! wget https://s3.amazonaws.com/plink2-assets/alpha5/plink2_linux_x86_64_20240105.zip\n","! unzip plink2_linux_x86_64_20240105.zip\n","! ./plink2"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"aPnn2JvVlPOV","executionInfo":{"status":"ok","timestamp":1710094653430,"user_tz":420,"elapsed":1298,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"69db6f98-6f53-43ed-bf36-dbc02df70059"},"execution_count":3,"outputs":[{"output_type":"stream","name":"stdout","text":["--2024-03-10 18:17:32--  https://s3.amazonaws.com/plink2-assets/alpha5/plink2_linux_x86_64_20240105.zip\n","Resolving s3.amazonaws.com (s3.amazonaws.com)... 52.217.99.110, 16.182.35.168, 54.231.224.40, ...\n","Connecting to s3.amazonaws.com (s3.amazonaws.com)|52.217.99.110|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 9266484 (8.8M) [application/zip]\n","Saving to: ‘plink2_linux_x86_64_20240105.zip’\n","\n","plink2_linux_x86_64 100%[===================>]   8.84M  33.1MB/s    in 0.3s    \n","\n","2024-03-10 18:17:32 (33.1 MB/s) - ‘plink2_linux_x86_64_20240105.zip’ saved [9266484/9266484]\n","\n","Archive:  plink2_linux_x86_64_20240105.zip\n","  inflating: plink2                  \n","PLINK v2.00a5.10LM 64-bit Intel (5 Jan 2024)   www.cog-genomics.org/plink/2.0/\n","(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n","\n","  plink2 <input flag(s)...> [command flag(s)...] [other flag(s)...]\n","  plink2 --help [flag name(s)...]\n","\n","Commands include --rm-dup list, --make-bpgen, --export, --freq, --geno-counts,\n","--sample-counts, --missing, --hardy, --het, --fst, --indep-pairwise, --ld,\n","--sample-diff, --make-king, --king-cutoff, --pmerge, --pgen-diff,\n","--write-samples, --write-snplist, --make-grm-list, --pca, --glm, --adjust-file,\n","--gwas-ssf, --clump, --score, --variant-score, --genotyping-rate, --pgen-info,\n","--validate, and --zst-decompress.\n","\n","\"plink2 --help | more\" describes all functions.\n"]}]},{"cell_type":"code","source":["! cp /content/drive/MyDrive/CSE-284-Final-Project/data/1000g_by_chrom/chr19.* ."],"metadata":{"id":"1SOMgrk8llad","executionInfo":{"status":"ok","timestamp":1710094710431,"user_tz":420,"elapsed":4687,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":4,"outputs":[]},{"cell_type":"code","source":["! cp /content/drive/MyDrive/CSE-284-Final-Project/data/test_data/ldl_prs.txt ."],"metadata":{"id":"1hEzWdSolP-N","executionInfo":{"status":"ok","timestamp":1710094851310,"user_tz":420,"elapsed":531,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":5,"outputs":[]},{"cell_type":"code","source":["import pandas as pd\n","effects_df = pd.read_csv('ldl_prs.txt', sep='\\t', names=['snp', 'allele', 'effect'])\n","effects_df = effects_df.drop(columns=['allele'])\n","effects_df"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":424},"id":"3ESjx56ZmKWc","executionInfo":{"status":"ok","timestamp":1710095267605,"user_tz":420,"elapsed":198,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"9d8f5542-567a-4590-b217-48d554b85c3b"},"execution_count":8,"outputs":[{"output_type":"execute_result","data":{"text/plain":["              snp  effect\n","0          rs7412  0.5032\n","1      rs11591147  0.4325\n","2     rs117261169  0.3915\n","3     rs113345881 -0.1350\n","4      rs62117204  0.1386\n","...           ...     ...\n","7786  rs529444196 -0.0965\n","7787   rs28797529  0.0288\n","7788  rs143415507 -0.0187\n","7789  rs539832736  0.0647\n","7790  rs116629050  0.0273\n","\n","[7791 rows x 2 columns]"],"text/html":["\n","  <div id=\"df-e679a807-3cc2-47b8-a191-0559f5420548\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>snp</th>\n","      <th>effect</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>rs7412</td>\n","      <td>0.5032</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>rs11591147</td>\n","      <td>0.4325</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>rs117261169</td>\n","      <td>0.3915</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>rs113345881</td>\n","      <td>-0.1350</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>rs62117204</td>\n","      <td>0.1386</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>7786</th>\n","      <td>rs529444196</td>\n","      <td>-0.0965</td>\n","    </tr>\n","    <tr>\n","      <th>7787</th>\n","      <td>rs28797529</td>\n","      <td>0.0288</td>\n","    </tr>\n","    <tr>\n","      <th>7788</th>\n","      <td>rs143415507</td>\n","      <td>-0.0187</td>\n","    </tr>\n","    <tr>\n","      <th>7789</th>\n","      <td>rs539832736</td>\n","      <td>0.0647</td>\n","    </tr>\n","    <tr>\n","      <th>7790</th>\n","      <td>rs116629050</td>\n","      <td>0.0273</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>7791 rows × 2 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-e679a807-3cc2-47b8-a191-0559f5420548')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-e679a807-3cc2-47b8-a191-0559f5420548 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-e679a807-3cc2-47b8-a191-0559f5420548');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-402644a2-6d41-445d-9bc6-88616e9ffdac\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-402644a2-6d41-445d-9bc6-88616e9ffdac')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-402644a2-6d41-445d-9bc6-88616e9ffdac button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_dce155a1-500c-4e11-b16f-c82a4eb015ed\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('effects_df')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_dce155a1-500c-4e11-b16f-c82a4eb015ed button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('effects_df');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"effects_df","summary":"{\n  \"name\": \"effects_df\",\n  \"rows\": 7791,\n  \"fields\": [\n    {\n      \"column\": \"snp\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 7791,\n        \"samples\": [\n          \"rs116413795\",\n          \"rs62323720\",\n          \"rs145481672\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"effect\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0.07769823317580843,\n        \"min\": -1.2352,\n        \"max\": 1.059,\n        \"num_unique_values\": 2415,\n        \"samples\": [\n          -0.137,\n          0.0606,\n          0.0712\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"}},"metadata":{},"execution_count":8}]},{"cell_type":"code","source":["chr19_snps = pd.read_csv('chr19.bim', sep=\"\\t\", names=[\"chr\", \"snp\", \"pos\", \"coord\", \"alt\", \"ref\"])\n","chr19_snps"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":424},"id":"ouajtHRsr6Xz","executionInfo":{"status":"ok","timestamp":1710096479463,"user_tz":420,"elapsed":499,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"152b1d6f-6b3d-4272-c1cd-7c1658875b72"},"execution_count":24,"outputs":[{"output_type":"execute_result","data":{"text/plain":["        chr          snp  pos     coord alt ref\n","0        19  rs372156287    0     69984   A   G\n","1        19  rs199696009    0     95794   C   A\n","2        19  rs571287431    0    103810   T   A\n","3        19  rs549887024    0    130998   G   A\n","4        19  rs373248213    0    140465   T   C\n","...     ...          ...  ...       ...  ..  ..\n","283168   19  rs150483540    0  59118016   T   G\n","283169   19  rs549996506    0  59118367   G   T\n","283170   19  rs186913222    0  59118616   T   C\n","283171   19  rs141816674    0  59118779   C   G\n","283172   19  rs150801216    0  59118783   G   C\n","\n","[283173 rows x 6 columns]"],"text/html":["\n","  <div id=\"df-eabdf016-e9fb-4466-9518-dba87fc9a3f5\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>chr</th>\n","      <th>snp</th>\n","      <th>pos</th>\n","      <th>coord</th>\n","      <th>alt</th>\n","      <th>ref</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>19</td>\n","      <td>rs372156287</td>\n","      <td>0</td>\n","      <td>69984</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>19</td>\n","      <td>rs199696009</td>\n","      <td>0</td>\n","      <td>95794</td>\n","      <td>C</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>19</td>\n","      <td>rs571287431</td>\n","      <td>0</td>\n","      <td>103810</td>\n","      <td>T</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>19</td>\n","      <td>rs549887024</td>\n","      <td>0</td>\n","      <td>130998</td>\n","      <td>G</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>19</td>\n","      <td>rs373248213</td>\n","      <td>0</td>\n","      <td>140465</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>283168</th>\n","      <td>19</td>\n","      <td>rs150483540</td>\n","      <td>0</td>\n","      <td>59118016</td>\n","      <td>T</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>283169</th>\n","      <td>19</td>\n","      <td>rs549996506</td>\n","      <td>0</td>\n","      <td>59118367</td>\n","      <td>G</td>\n","      <td>T</td>\n","    </tr>\n","    <tr>\n","      <th>283170</th>\n","      <td>19</td>\n","      <td>rs186913222</td>\n","      <td>0</td>\n","      <td>59118616</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>283171</th>\n","      <td>19</td>\n","      <td>rs141816674</td>\n","      <td>0</td>\n","      <td>59118779</td>\n","      <td>C</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>283172</th>\n","      <td>19</td>\n","      <td>rs150801216</td>\n","      <td>0</td>\n","      <td>59118783</td>\n","      <td>G</td>\n","      <td>C</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>283173 rows × 6 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-eabdf016-e9fb-4466-9518-dba87fc9a3f5')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-eabdf016-e9fb-4466-9518-dba87fc9a3f5 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-eabdf016-e9fb-4466-9518-dba87fc9a3f5');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-4f9e9693-c625-45fc-ac84-0eb15cae5b4d\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-4f9e9693-c625-45fc-ac84-0eb15cae5b4d')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-4f9e9693-c625-45fc-ac84-0eb15cae5b4d button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_0f034b70-ca65-42f2-b5fc-b9a8277c43af\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('chr19_snps')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_0f034b70-ca65-42f2-b5fc-b9a8277c43af button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('chr19_snps');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"chr19_snps"}},"metadata":{},"execution_count":24}]},{"cell_type":"code","source":["effects_chr19 = effects_df.merge(chr19_snps, on='snp')\n","effects_chr19"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":424},"id":"VAHL-BEgskuM","executionInfo":{"status":"ok","timestamp":1710096548900,"user_tz":420,"elapsed":364,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"386b2f98-b80f-48ed-b8b3-d27c9072dd27"},"execution_count":25,"outputs":[{"output_type":"execute_result","data":{"text/plain":["             snp  effect  chr  pos     coord alt ref\n","0         rs7412  0.5032   19    0  45412079   T   C\n","1    rs113345881 -0.1350   19    0  45431636   G   A\n","2     rs62117204  0.1386   19    0  45242967   T   C\n","3     rs56317818 -0.0776   19    0  45359586   T   C\n","4      rs2965169 -0.0657   19    0  45251156   C   A\n","..           ...     ...  ...  ...       ...  ..  ..\n","507  rs147716408 -0.0394   19    0  44785588   C   T\n","508   rs73566656 -0.0068   19    0  46381181   A   G\n","509   rs76353780 -0.0362   19    0   5416505   C   T\n","510  rs115758240 -0.0407   19    0  11350086   A   G\n","511   rs79068664  0.0373   19    0  50007599   C   G\n","\n","[512 rows x 7 columns]"],"text/html":["\n","  <div id=\"df-807deeb4-a6e9-4472-9a0a-54add4940366\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>snp</th>\n","      <th>effect</th>\n","      <th>chr</th>\n","      <th>pos</th>\n","      <th>coord</th>\n","      <th>alt</th>\n","      <th>ref</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>rs7412</td>\n","      <td>0.5032</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45412079</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>rs113345881</td>\n","      <td>-0.1350</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45431636</td>\n","      <td>G</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>rs62117204</td>\n","      <td>0.1386</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45242967</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>rs56317818</td>\n","      <td>-0.0776</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45359586</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>rs2965169</td>\n","      <td>-0.0657</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45251156</td>\n","      <td>C</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>507</th>\n","      <td>rs147716408</td>\n","      <td>-0.0394</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>44785588</td>\n","      <td>C</td>\n","      <td>T</td>\n","    </tr>\n","    <tr>\n","      <th>508</th>\n","      <td>rs73566656</td>\n","      <td>-0.0068</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>46381181</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>509</th>\n","      <td>rs76353780</td>\n","      <td>-0.0362</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>5416505</td>\n","      <td>C</td>\n","      <td>T</td>\n","    </tr>\n","    <tr>\n","      <th>510</th>\n","      <td>rs115758240</td>\n","      <td>-0.0407</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>11350086</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>511</th>\n","      <td>rs79068664</td>\n","      <td>0.0373</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>50007599</td>\n","      <td>C</td>\n","      <td>G</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>512 rows × 7 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-807deeb4-a6e9-4472-9a0a-54add4940366')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-807deeb4-a6e9-4472-9a0a-54add4940366 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-807deeb4-a6e9-4472-9a0a-54add4940366');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-0f5b170a-471c-47b2-89ad-34b20da8d11f\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0f5b170a-471c-47b2-89ad-34b20da8d11f')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-0f5b170a-471c-47b2-89ad-34b20da8d11f button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_7e822099-d51d-4504-88e5-ff5ccf413dc9\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('effects_chr19')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_7e822099-d51d-4504-88e5-ff5ccf413dc9 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('effects_chr19');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"effects_chr19","summary":"{\n  \"name\": \"effects_chr19\",\n  \"rows\": 512,\n  \"fields\": [\n    {\n      \"column\": \"snp\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 512,\n        \"samples\": [\n          \"rs204911\",\n          \"rs72658856\",\n          \"rs35678764\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"effect\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0.0496988806025273,\n        \"min\": -0.1865,\n        \"max\": 0.5032,\n        \"num_unique_values\": 383,\n        \"samples\": [\n          0.0152,\n          -0.0212,\n          -0.0082\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"chr\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 19,\n        \"max\": 19,\n        \"num_unique_values\": 1,\n        \"samples\": [\n          19\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pos\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 0,\n        \"max\": 0,\n        \"num_unique_values\": 1,\n        \"samples\": [\n          0\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"coord\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 17725529,\n        \"min\": 717289,\n        \"max\": 59082605,\n        \"num_unique_values\": 512,\n        \"samples\": [\n          45463540\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"alt\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 4,\n        \"samples\": [\n          \"G\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"ref\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 4,\n        \"samples\": [\n          \"A\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"}},"metadata":{},"execution_count":25}]},{"cell_type":"code","source":["effects_chr19_transposed = effects_chr19.transpose()\n","effects_chr19_transposed"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":318},"id":"h5zVpwSznwci","executionInfo":{"status":"ok","timestamp":1710096572580,"user_tz":420,"elapsed":169,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"6b045fda-eefa-4360-acdb-e348c350e916"},"execution_count":26,"outputs":[{"output_type":"execute_result","data":{"text/plain":["             0            1           2           3          4            5    \\\n","snp       rs7412  rs113345881  rs62117204  rs56317818  rs2965169  rs146576912   \n","effect    0.5032       -0.135      0.1386     -0.0776    -0.0657        0.142   \n","chr           19           19          19          19         19           19   \n","pos            0            0           0           0          0            0   \n","coord   45412079     45431636    45242967    45359586   45251156     11283222   \n","alt            T            G           T           T          C            T   \n","ref            C            A           C           C          A            C   \n","\n","               6           7           8           9    ...        502  \\\n","snp     rs62119328  rs11671372  rs58542926  rs11881315  ...  rs8100933   \n","effect       -0.16      0.0507      0.0916      0.0565  ...     0.0077   \n","chr             19          19          19          19  ...         19   \n","pos              0           0           0           0  ...          0   \n","coord     45187163    11269726    19379549    10909953  ...   47540069   \n","alt              G           T           T           T  ...          A   \n","ref              A           G           C           C  ...          G   \n","\n","              503          504        505         506          507  \\\n","snp     rs3848609  rs143059942  rs2051815  rs56344893  rs147716408   \n","effect      0.015      -0.0142    -0.0061      0.0061      -0.0394   \n","chr            19           19         19          19           19   \n","pos             0            0          0           0            0   \n","coord    55753433     49213604   18835115    44282529     44785588   \n","alt             A            C          A           A            C   \n","ref             G            T          G           C            T   \n","\n","               508         509          510         511  \n","snp     rs73566656  rs76353780  rs115758240  rs79068664  \n","effect     -0.0068     -0.0362      -0.0407      0.0373  \n","chr             19          19           19          19  \n","pos              0           0            0           0  \n","coord     46381181     5416505     11350086    50007599  \n","alt              A           C            A           C  \n","ref              G           T            G           G  \n","\n","[7 rows x 512 columns]"],"text/html":["\n","  <div id=\"df-d5dd9810-e513-4811-bb58-440f111d9972\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>0</th>\n","      <th>1</th>\n","      <th>2</th>\n","      <th>3</th>\n","      <th>4</th>\n","      <th>5</th>\n","      <th>6</th>\n","      <th>7</th>\n","      <th>8</th>\n","      <th>9</th>\n","      <th>...</th>\n","      <th>502</th>\n","      <th>503</th>\n","      <th>504</th>\n","      <th>505</th>\n","      <th>506</th>\n","      <th>507</th>\n","      <th>508</th>\n","      <th>509</th>\n","      <th>510</th>\n","      <th>511</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>snp</th>\n","      <td>rs7412</td>\n","      <td>rs113345881</td>\n","      <td>rs62117204</td>\n","      <td>rs56317818</td>\n","      <td>rs2965169</td>\n","      <td>rs146576912</td>\n","      <td>rs62119328</td>\n","      <td>rs11671372</td>\n","      <td>rs58542926</td>\n","      <td>rs11881315</td>\n","      <td>...</td>\n","      <td>rs8100933</td>\n","      <td>rs3848609</td>\n","      <td>rs143059942</td>\n","      <td>rs2051815</td>\n","      <td>rs56344893</td>\n","      <td>rs147716408</td>\n","      <td>rs73566656</td>\n","      <td>rs76353780</td>\n","      <td>rs115758240</td>\n","      <td>rs79068664</td>\n","    </tr>\n","    <tr>\n","      <th>effect</th>\n","      <td>0.5032</td>\n","      <td>-0.135</td>\n","      <td>0.1386</td>\n","      <td>-0.0776</td>\n","      <td>-0.0657</td>\n","      <td>0.142</td>\n","      <td>-0.16</td>\n","      <td>0.0507</td>\n","      <td>0.0916</td>\n","      <td>0.0565</td>\n","      <td>...</td>\n","      <td>0.0077</td>\n","      <td>0.015</td>\n","      <td>-0.0142</td>\n","      <td>-0.0061</td>\n","      <td>0.0061</td>\n","      <td>-0.0394</td>\n","      <td>-0.0068</td>\n","      <td>-0.0362</td>\n","      <td>-0.0407</td>\n","      <td>0.0373</td>\n","    </tr>\n","    <tr>\n","      <th>chr</th>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>...</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","      <td>19</td>\n","    </tr>\n","    <tr>\n","      <th>pos</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>...</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>coord</th>\n","      <td>45412079</td>\n","      <td>45431636</td>\n","      <td>45242967</td>\n","      <td>45359586</td>\n","      <td>45251156</td>\n","      <td>11283222</td>\n","      <td>45187163</td>\n","      <td>11269726</td>\n","      <td>19379549</td>\n","      <td>10909953</td>\n","      <td>...</td>\n","      <td>47540069</td>\n","      <td>55753433</td>\n","      <td>49213604</td>\n","      <td>18835115</td>\n","      <td>44282529</td>\n","      <td>44785588</td>\n","      <td>46381181</td>\n","      <td>5416505</td>\n","      <td>11350086</td>\n","      <td>50007599</td>\n","    </tr>\n","    <tr>\n","      <th>alt</th>\n","      <td>T</td>\n","      <td>G</td>\n","      <td>T</td>\n","      <td>T</td>\n","      <td>C</td>\n","      <td>T</td>\n","      <td>G</td>\n","      <td>T</td>\n","      <td>T</td>\n","      <td>T</td>\n","      <td>...</td>\n","      <td>A</td>\n","      <td>A</td>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>A</td>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>ref</th>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>C</td>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>C</td>\n","      <td>A</td>\n","      <td>G</td>\n","      <td>C</td>\n","      <td>C</td>\n","      <td>...</td>\n","      <td>G</td>\n","      <td>G</td>\n","      <td>T</td>\n","      <td>G</td>\n","      <td>C</td>\n","      <td>T</td>\n","      <td>G</td>\n","      <td>T</td>\n","      <td>G</td>\n","      <td>G</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>7 rows × 512 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d5dd9810-e513-4811-bb58-440f111d9972')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-d5dd9810-e513-4811-bb58-440f111d9972 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-d5dd9810-e513-4811-bb58-440f111d9972');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-3051f3ef-60b2-4ed2-849d-74d63d7fa1fc\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-3051f3ef-60b2-4ed2-849d-74d63d7fa1fc')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-3051f3ef-60b2-4ed2-849d-74d63d7fa1fc button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_ea1b98c9-0f10-4c17-998c-3c8542bd159a\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('effects_chr19_transposed')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_ea1b98c9-0f10-4c17-998c-3c8542bd159a button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('effects_chr19_transposed');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"effects_chr19_transposed"}},"metadata":{},"execution_count":26}]},{"cell_type":"code","source":["pheno_name = \"PHENOTEST\"\n","causals = (effects_chr19_transposed\n","           .loc[[\"snp\"]]\n","           .reset_index(drop=True))\n","causals"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":130},"id":"MxGb3mAImKwK","executionInfo":{"status":"ok","timestamp":1710097023949,"user_tz":420,"elapsed":151,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"3e538878-1005-4dec-e864-e790262cc36f"},"execution_count":44,"outputs":[{"output_type":"execute_result","data":{"text/plain":["      0            1           2           3          4            5    \\\n","0  rs7412  rs113345881  rs62117204  rs56317818  rs2965169  rs146576912   \n","\n","          6           7           8           9    ...        502        503  \\\n","0  rs62119328  rs11671372  rs58542926  rs11881315  ...  rs8100933  rs3848609   \n","\n","           504        505         506          507         508         509  \\\n","0  rs143059942  rs2051815  rs56344893  rs147716408  rs73566656  rs76353780   \n","\n","           510         511  \n","0  rs115758240  rs79068664  \n","\n","[1 rows x 512 columns]"],"text/html":["\n","  <div id=\"df-862c36ba-2b75-4a4a-9f32-add01778b2bd\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>0</th>\n","      <th>1</th>\n","      <th>2</th>\n","      <th>3</th>\n","      <th>4</th>\n","      <th>5</th>\n","      <th>6</th>\n","      <th>7</th>\n","      <th>8</th>\n","      <th>9</th>\n","      <th>...</th>\n","      <th>502</th>\n","      <th>503</th>\n","      <th>504</th>\n","      <th>505</th>\n","      <th>506</th>\n","      <th>507</th>\n","      <th>508</th>\n","      <th>509</th>\n","      <th>510</th>\n","      <th>511</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>rs7412</td>\n","      <td>rs113345881</td>\n","      <td>rs62117204</td>\n","      <td>rs56317818</td>\n","      <td>rs2965169</td>\n","      <td>rs146576912</td>\n","      <td>rs62119328</td>\n","      <td>rs11671372</td>\n","      <td>rs58542926</td>\n","      <td>rs11881315</td>\n","      <td>...</td>\n","      <td>rs8100933</td>\n","      <td>rs3848609</td>\n","      <td>rs143059942</td>\n","      <td>rs2051815</td>\n","      <td>rs56344893</td>\n","      <td>rs147716408</td>\n","      <td>rs73566656</td>\n","      <td>rs76353780</td>\n","      <td>rs115758240</td>\n","      <td>rs79068664</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>1 rows × 512 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-862c36ba-2b75-4a4a-9f32-add01778b2bd')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-862c36ba-2b75-4a4a-9f32-add01778b2bd button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-862c36ba-2b75-4a4a-9f32-add01778b2bd');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","  <div id=\"id_3b40bf08-8360-48d0-a234-0b88498c9c6e\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('causals')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_3b40bf08-8360-48d0-a234-0b88498c9c6e button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('causals');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"causals"}},"metadata":{},"execution_count":44}]},{"cell_type":"code","source":["effects = (effects_chr19_transposed\n","           .loc[[\"effect\"]]\n","           .reset_index(drop=True))\n","effects"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":130},"id":"bI1Uf-YJp7Cp","executionInfo":{"status":"ok","timestamp":1710099994596,"user_tz":420,"elapsed":128,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"b8796b51-0e65-4fc1-9d47-f6afc2304133"},"execution_count":73,"outputs":[{"output_type":"execute_result","data":{"text/plain":["      0      1       2       3       4      5     6       7       8       9    \\\n","0  0.5032 -0.135  0.1386 -0.0776 -0.0657  0.142 -0.16  0.0507  0.0916  0.0565   \n","\n","   ...     502    503     504     505     506     507     508     509     510  \\\n","0  ...  0.0077  0.015 -0.0142 -0.0061  0.0061 -0.0394 -0.0068 -0.0362 -0.0407   \n","\n","      511  \n","0  0.0373  \n","\n","[1 rows x 512 columns]"],"text/html":["\n","  <div id=\"df-ab561408-f9c3-41d7-8bf8-63c030b56a6a\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>0</th>\n","      <th>1</th>\n","      <th>2</th>\n","      <th>3</th>\n","      <th>4</th>\n","      <th>5</th>\n","      <th>6</th>\n","      <th>7</th>\n","      <th>8</th>\n","      <th>9</th>\n","      <th>...</th>\n","      <th>502</th>\n","      <th>503</th>\n","      <th>504</th>\n","      <th>505</th>\n","      <th>506</th>\n","      <th>507</th>\n","      <th>508</th>\n","      <th>509</th>\n","      <th>510</th>\n","      <th>511</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>0.5032</td>\n","      <td>-0.135</td>\n","      <td>0.1386</td>\n","      <td>-0.0776</td>\n","      <td>-0.0657</td>\n","      <td>0.142</td>\n","      <td>-0.16</td>\n","      <td>0.0507</td>\n","      <td>0.0916</td>\n","      <td>0.0565</td>\n","      <td>...</td>\n","      <td>0.0077</td>\n","      <td>0.015</td>\n","      <td>-0.0142</td>\n","      <td>-0.0061</td>\n","      <td>0.0061</td>\n","      <td>-0.0394</td>\n","      <td>-0.0068</td>\n","      <td>-0.0362</td>\n","      <td>-0.0407</td>\n","      <td>0.0373</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>1 rows × 512 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ab561408-f9c3-41d7-8bf8-63c030b56a6a')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-ab561408-f9c3-41d7-8bf8-63c030b56a6a button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-ab561408-f9c3-41d7-8bf8-63c030b56a6a');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","  <div id=\"id_4c874bbc-277e-4867-b875-3a135aecf123\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('effects')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_4c874bbc-277e-4867-b875-3a135aecf123 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('effects');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"effects"}},"metadata":{},"execution_count":73}]},{"cell_type":"code","source":["! mkdir causals_effects"],"metadata":{"id":"6TirWLlIxc6w","executionInfo":{"status":"ok","timestamp":1710097830083,"user_tz":420,"elapsed":301,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":57,"outputs":[]},{"cell_type":"code","source":["repeat_size = 100\n","causals_replicate = causals.loc[causals.index.repeat(repeat_size)].reset_index(drop=True)\n","effects_replicate = effects.loc[effects.index.repeat(repeat_size)].reset_index(drop=True)\n","causals_replicate.to_csv('causals_effects/causals_all.txt', sep=\"\\t\", header=False, index=False)\n","effects_replicate.to_csv('causals_effects/effects_all.txt', sep=\"\\t\", header=False, index=False)"],"metadata":{"id":"zObfMe4lp_V7","executionInfo":{"status":"ok","timestamp":1710097847999,"user_tz":420,"elapsed":120,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":58,"outputs":[]},{"cell_type":"code","source":["! head causals_effects/causals_all.txt"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"tQhxUI81qLt8","executionInfo":{"status":"ok","timestamp":1710097874507,"user_tz":420,"elapsed":533,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"e59f384e-672d-470b-a22e-f8548615300e"},"execution_count":60,"outputs":[{"output_type":"stream","name":"stdout","text":["rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n","rs7412\trs113345881\trs62117204\trs56317818\trs2965169\trs146576912\trs62119328\trs11671372\trs58542926\trs11881315\trs2972555\trs11667610\trs142751148\trs62120565\trs17242353\trs2972564\trs56315738\trs12977255\trs204480\trs34516961\trs679574\trs141651018\trs28493980\trs12611153\trs1529711\trs421812\trs12721054\trs140181075\trs117712332\trs2015257\trs10420562\trs139198665\trs12720359\trs12460253\trs7249083\trs8112559\trs41301949\trs76936640\trs17714761\trs35136575\trs117970794\trs17248952\trs74827760\trs12974306\trs77736886\trs35845603\trs145639166\trs185574863\trs10420110\trs76288880\trs11669345\trs79322367\trs72626214\trs4015\trs150742778\trs73036542\trs112283362\trs2904241\trs142385484\trs75630274\trs75592254\trs141989097\trs75270598\trs62119326\trs62111294\trs892023\trs9749350\trs75463276\trs34757881\trs182955929\trs736926\trs75920121\trs60915609\trs28362819\trs144778381\trs374116013\trs112056827\trs3752109\trs150888326\trs116018036\trs2288450\trs12976057\trs141964576\trs10407726\trs141719134\trs4801486\trs4804592\trs191160523\trs11668554\trs553008653\trs10403723\trs79548089\trs4808595\trs916694\trs2365812\trs7255936\trs11671705\trs8736\trs11671341\trs58782346\trs112501445\trs4803840\trs2927486\trs35824797\trs58925945\trs73585751\trs3794995\trs12461923\trs148215750\trs2287802\trs10419557\trs35500721\trs149183396\trs62131099\trs2075710\trs143043127\trs10416139\trs7251911\trs3093032\trs56010181\trs78961098\trs139140823\trs140842514\trs77023463\trs10419906\trs55766194\trs10415949\trs62129356\trs76649675\trs35450632\trs34964177\trs7246871\trs1860193\trs2561563\trs577349548\trs11083786\trs12982294\trs2277970\trs12981782\trs72975595\trs12984134\trs78828368\trs73515570\trs374948969\trs111677968\trs62131083\trs35483143\trs150311873\trs78134016\trs35657593\trs141366665\trs77856404\trs35763270\trs11083699\trs62130305\trs74398580\trs12978533\trs247795\trs140534128\trs184601718\trs11085723\trs117222437\trs4808119\trs59848316\trs113357289\trs2074940\trs10420349\trs8110485\trs10401338\trs8107753\trs75983562\trs11673492\trs1233440\trs58496711\trs60785207\trs35400304\trs148700057\trs2238651\trs34099325\trs138215360\trs73046146\trs115379880\trs8109734\trs61736521\trs62113588\trs62109991\trs17262997\trs35630819\trs73048053\trs10408750\trs148152819\trs4803825\trs11879991\trs35902786\trs80123334\trs17228341\trs79007763\trs79287205\trs62104359\trs112972206\trs12052158\trs62132801\trs148058936\trs11540666\trs61593631\trs2967465\trs10411967\trs148446754\trs56264140\trs111872402\trs11665702\trs11882198\trs17605476\trs55697131\trs79040846\trs10424400\trs8112901\trs12611078\trs73053552\trs2238551\trs113886122\trs55734122\trs73931391\trs62131325\trs62118499\trs62131329\trs1055392\trs8107977\trs281417\trs11672734\trs12983010\trs11668344\trs11666216\trs1003393\trs117564136\trs74499708\trs8107329\trs56085552\trs10129\trs79544735\trs78282402\trs2599469\trs34297641\trs11085327\trs142585929\trs7252658\trs114281937\trs62111695\trs141203802\trs107878\trs73591055\trs4804048\trs836912\trs1800021\trs35334454\trs4760\trs35879596\trs16978665\trs61731191\trs16981816\trs76069409\trs117607802\trs7246079\trs78712006\trs61281091\trs113571662\trs10416849\trs59715573\trs429358\trs138175288\trs11668327\trs34827707\trs60049679\trs7246100\trs12052200\trs118170342\trs73015030\trs147540853\trs435380\trs56322906\trs17249001\trs35836101\trs4803791\trs28399607\trs36005514\trs3895773\trs4804510\trs7251161\trs34503352\trs17699238\trs1836278\trs72654437\trs2627646\trs35850102\trs3786730\trs143875794\trs11673631\trs10419998\trs76237591\trs4804149\trs62119263\trs116283165\trs139425456\trs78073763\trs204911\trs7248408\trs1640266\trs9749017\trs11669879\trs10221516\trs73936968\trs11881832\trs74359223\trs73558127\trs139794839\trs36049922\trs147434609\trs62118464\trs111435864\trs62115272\trs11083743\trs2927456\trs73051677\trs139874907\trs1235379\trs8106324\trs10419669\trs4564699\trs118054030\trs72997146\trs4802506\trs141441332\trs8106963\trs117777502\trs111273322\trs12150984\trs2548957\trs115036848\trs4803836\trs79860339\trs7247433\trs10409772\trs8109618\trs34651602\trs34824923\trs34256230\trs12978957\trs147871592\trs66725322\trs139773371\trs10409774\trs75750038\trs138534124\trs12459496\trs16979231\trs238410\trs75161053\trs34448858\trs12984096\trs80257887\trs62125172\trs17652418\trs62132778\trs3810366\trs77372956\trs59840524\trs72997195\trs191608294\trs551557788\trs138218778\trs11260029\trs2074304\trs147791730\trs62129144\trs644699\trs10421262\trs1552221\trs73572043\trs1441617\trs77565836\trs148613044\trs45575140\trs346048\trs140914890\trs12462944\trs8109764\trs11880312\trs749260\trs143097253\trs11667752\trs149513825\trs11670136\trs80307900\trs60006507\trs7246718\trs260422\trs2288419\trs11881918\trs35537738\trs75014601\trs157368\trs12462642\trs11673516\trs144159978\trs62126247\trs11880141\trs3816864\trs279174\trs62113965\trs58513885\trs112298657\trs1047333\trs73013009\trs3729859\trs4468734\trs34561079\trs7255083\trs140416360\trs151006362\trs2878162\trs113755440\trs2335534\trs145200358\trs11547373\trs6509400\trs111323668\trs115624663\trs57943942\trs2304244\trs72992965\trs370539355\trs201905366\trs10404717\trs61531062\trs77156507\trs35844933\trs140915448\trs140816487\trs45446093\trs35678764\trs2007774\trs3826994\trs55737395\trs8110775\trs140656901\trs80068834\trs113642564\trs10425863\trs114395138\trs1962978\trs62107003\trs8111715\trs139255090\trs113986398\trs189434727\trs3729767\trs2003567\trs73923128\trs138869234\trs10451476\trs4801564\trs62121713\trs111441916\trs146251426\trs34768259\trs10417167\trs73563690\trs143487272\trs111443372\trs1548546\trs186167275\trs73936819\trs147936390\trs114370247\trs75951249\trs7254990\trs2620840\trs434606\trs62135502\trs73048635\trs59380439\trs116027944\trs1382090\trs116441945\trs346051\trs59616136\trs62113384\trs60545781\trs73047189\trs10416680\trs112581039\trs74931926\trs148841287\trs34123065\trs2240656\trs72658856\trs3810333\trs146901672\trs112628847\trs4806921\trs76528607\trs144110893\trs8100933\trs3848609\trs143059942\trs2051815\trs56344893\trs147716408\trs73566656\trs76353780\trs115758240\trs79068664\n"]}]},{"cell_type":"code","source":["! head causals_effects/effects_all.txt"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"x-1fZKMeqP4v","executionInfo":{"status":"ok","timestamp":1710097879661,"user_tz":420,"elapsed":248,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"26b6536d-44b1-41e7-de9c-bb5918702287"},"execution_count":61,"outputs":[{"output_type":"stream","name":"stdout","text":["0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n","0.5032\t-0.135\t0.1386\t-0.0776\t-0.0657\t0.142\t-0.16\t0.0507\t0.0916\t0.0565\t-0.0403\t0.0422\t0.1335\t-0.0593\t-0.1077\t0.0412\t0.1065\t-0.0331\t0.0346\t0.0402\t0.027\t0.0848\t0.0516\t0.0312\t-0.036\t-0.0251\t-0.1152\t-0.0621\t0.1001\t-0.0239\t-0.035\t-0.0676\t-0.0541\t-0.0654\t-0.0813\t-0.025\t0.0525\t0.0306\t0.0196\t-0.0232\t0.0704\t-0.0339\t0.0464\t-0.0148\t-0.0456\t-0.0144\t0.0499\t-0.0476\t-0.0687\t0.025\t-0.0166\t0.0199\t0.0207\t0.0155\t-0.0421\t-0.018\t-0.0411\t-0.0123\t0.018\t0.0236\t0.0284\t-0.0219\t0.0162\t0.0267\t-0.0134\t-0.0117\t0.015\t0.0482\t0.0265\t0.066\t-0.0146\t-0.0472\t0.0131\t-0.031\t0.0368\t0.0229\t-0.0618\t0.0107\t-0.03\t-0.018\t0.0141\t0.0128\t0.0636\t0.0107\t0.0314\t0.0113\t0.0276\t0.0407\t0.0125\t0.0274\t-0.0099\t0.0271\t-0.0123\t0.0092\t0.0097\t-0.0109\t-0.0151\t-0.009\t-0.041\t-0.0891\t0.0267\t-0.0195\t-0.0739\t-0.0156\t-0.0132\t-0.0132\t-0.0109\t-0.0126\t-0.0205\t0.0091\t-0.0466\t-0.0237\t0.0526\t-0.0151\t0.0095\t-0.0871\t-0.0106\t0.015\t0.0122\t0.0131\t0.0346\t-0.0297\t-0.0275\t0.0253\t0.0145\t-0.0091\t-0.0089\t0.0094\t-0.0093\t0.0108\t-0.0169\t-0.0179\t0.0096\t-0.011\t-0.0815\t0.008\t-0.0109\t0.0232\t0.0074\t-0.0128\t0.0128\t0.0177\t-0.0394\t0.0242\t-0.0285\t-0.0313\t0.0258\t0.0726\t0.0081\t0.0109\t-0.0727\t0.0131\t0.0084\t-0.0084\t-0.0098\t0.0236\t-0.0111\t-0.0078\t0.0174\t-0.0447\t-0.0091\t0.0182\t-0.0068\t-0.009\t0.0135\t0.0072\t-0.048\t0.0076\t-0.0098\t-0.0098\t0.0098\t-0.0089\t-0.0436\t-0.0124\t0.0106\t0.0075\t0.0519\t0.0079\t-0.0223\t-0.0201\t0.01\t-0.0439\t-0.0302\t-0.0207\t-0.0215\t0.015\t0.0154\t-0.0124\t-0.0218\t-0.0094\t-0.0175\t0.0064\t0.0264\t0.0085\t0.0399\t0.0123\t0.0355\t-0.0207\t-0.0152\t0.0215\t0.0063\t-0.0126\t0.0692\t0.013\t-0.0091\t-0.0091\t0.0368\t0.0227\t0.0103\t-0.0206\t-0.007\t-0.007\t-0.0107\t0.0074\t-0.0232\t-0.0313\t0.0471\t0.0073\t0.0081\t0.0085\t0.0085\t-0.0089\t-0.0198\t-0.0109\t-0.0109\t-0.0084\t0.0088\t-0.006\t0.0064\t-0.0076\t0.0108\t0.006\t0.0104\t0.0282\t-0.0119\t-0.0115\t-0.0075\t0.0067\t-0.0059\t-0.0161\t-0.0267\t0.009\t-0.0113\t-0.0066\t-0.0361\t-0.0062\t0.0611\t-0.0193\t-0.0328\t0.0077\t-0.0203\t0.0199\t0.0065\t-0.0279\t0.0084\t-0.0084\t0.0061\t0.016\t-0.0372\t-0.0193\t0.014\t0.0117\t0.0083\t-0.0226\t-0.0094\t-0.0124\t-0.006\t0.058\t0.1912\t0.1897\t0.1353\t0.2991\t-0.1366\t0.0741\t0.0572\t0.1317\t0.1346\t0.0908\t-0.1865\t0.0571\t-0.0756\t0.0372\t0.0384\t0.0752\t-0.0676\t0.0829\t0.0272\t0.0289\t0.028\t0.0231\t-0.0223\t-0.0654\t-0.0389\t0.0436\t0.1024\t-0.0364\t-0.0508\t0.0299\t0.0282\t0.0191\t-0.0212\t-0.1289\t-0.0903\t-0.0332\t0.0445\t-0.0172\t0.0161\t0.0231\t-0.0268\t-0.0167\t-0.0544\t0.1202\t-0.0761\t-0.1003\t-0.1219\t0.0456\t-0.0669\t-0.0293\t0.0338\t-0.0144\t0.0152\t0.0287\t0.0389\t-0.0275\t-0.0127\t0.0177\t-0.0219\t0.0168\t-0.0325\t0.0208\t0.0164\t-0.0655\t0.0409\t-0.0327\t-0.0191\t-0.0142\t0.0134\t-0.0905\t0.0139\t0.0361\t0.0129\t-0.0164\t-0.0164\t-0.0208\t-0.0274\t-0.0212\t0.0202\t-0.0685\t0.0107\t-0.0631\t0.0205\t0.0357\t-0.0792\t-0.0104\t0.013\t0.0634\t-0.031\t-0.0432\t-0.0098\t-0.032\t0.0123\t-0.0128\t0.0152\t0.0094\t-0.0293\t0.0311\t0.0205\t0.0957\t-0.0205\t0.0787\t0.0102\t0.0864\t0.0275\t-0.0412\t-0.0173\t-0.0094\t-0.0094\t-0.0498\t-0.0187\t-0.0537\t-0.0208\t-0.0641\t0.012\t0.0188\t-0.0085\t-0.0203\t-0.0397\t0.0139\t0.0675\t-0.0271\t-0.0367\t-0.0082\t0.0213\t0.0092\t-0.0096\t-0.008\t-0.009\t0.0137\t-0.01\t-0.0221\t-0.0084\t0.0089\t0.0078\t-0.0876\t0.023\t-0.0112\t-0.0081\t-0.0151\t-0.009\t0.036\t-0.0215\t0.0077\t0.0177\t0.0291\t-0.0076\t-0.0128\t0.0071\t-0.0194\t-0.0104\t0.0094\t-0.0642\t-0.0089\t-0.0477\t-0.007\t0.0093\t-0.0116\t-0.0436\t-0.0074\t0.0082\t-0.0173\t0.0332\t-0.0113\t0.0487\t-0.0284\t-0.0108\t0.0112\t0.0272\t0.0254\t-0.0098\t-0.0071\t0.0084\t0.0084\t0.0075\t0.0149\t0.0162\t-0.0271\t-0.0144\t0.0148\t0.0474\t0.0078\t-0.0104\t0.0091\t0.0575\t0.0112\t0.0611\t0.0129\t0.0146\t-0.012\t0.03\t-0.0064\t-0.0064\t0.0085\t-0.0326\t-0.0241\t0.0283\t-0.0092\t-0.0503\t0.0108\t-0.058\t0.0062\t0.0285\t0.0218\t-0.0275\t0.0644\t-0.011\t-0.0061\t0.0073\t0.0073\t-0.0093\t0.0129\t-0.0173\t-0.0213\t-0.0333\t0.0786\t0.0072\t0.0099\t0.0099\t0.0091\t-0.0091\t0.028\t-0.009\t0.0101\t0.0167\t-0.0163\t-0.0093\t-0.0581\t-0.0058\t-0.0336\t-0.0139\t0.0081\t-0.0189\t0.0185\t0.0077\t0.015\t-0.0142\t-0.0061\t0.0061\t-0.0394\t-0.0068\t-0.0362\t-0.0407\t0.0373\n"]}]},{"cell_type":"code","source":["! mkdir chr19_ldl_pheno"],"metadata":{"id":"xZcmBlRExxyf","executionInfo":{"status":"ok","timestamp":1710097947137,"user_tz":420,"elapsed":141,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":62,"outputs":[]},{"cell_type":"code","source":["sh = \"\"\"\n","\n","BFILE=chr19\n","CAUSALFILE=causals_effects/causals_all.txt\n","EFFECTSFILE=causals_effects/effects_all.txt\n","NUM_PHENO=100\n","NUM_CAUSL=512\n","PARENT_DIR=\"/content/chr19_ldl_pheno\"\n","\n","\n","for POWER in {-1,-0.25}\n","\n","do\n","    for HER in {0.1,0.3,0.5,0.7,0.9}\n","\n","    do\n","\n","    OUT=${PARENT_DIR}/power=${POWER}_her=${HER}_num-causals=${NUM_CAUSL}\n","\n","        ./ldak5.2.linux \\\n","            --make-phenos $OUT \\\n","            --bfile $BFILE \\\n","            --num-phenos $NUM_PHENO \\\n","            --causals $CAUSALFILE \\\n","            --effects $EFFECTSFILE \\\n","            --power $POWER \\\n","            --her $HER \\\n","            --num-causals $NUM_CAUSL \\\n","            --ignore-weights YES\n","\n","    done\n","\n","done\n","\n","\"\"\"\n","with open('script.sh', 'w') as file:\n","  file.write(sh)\n","\n","! bash script.sh"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"6pO5MVGmqSQH","executionInfo":{"status":"ok","timestamp":1710098142248,"user_tz":420,"elapsed":27070,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"503de0d4-f14e-4636-c770-58111fc69d89"},"execution_count":63,"outputs":[{"output_type":"stream","name":"stdout","text":["-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -1\n","--her 0.1\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -1\n","--her 0.3\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -1\n","--her 0.5\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -1\n","--her 0.7\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -1\n","--her 0.9\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--effects causals_effects/effects_all.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_all.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n"]}]},{"cell_type":"code","source":["! cp -r chr19_ldl_pheno /content/drive/MyDrive/CSE-284-Final-Project/data/"],"metadata":{"id":"bqJwKC7gvomC","executionInfo":{"status":"ok","timestamp":1710098158892,"user_tz":420,"elapsed":866,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":64,"outputs":[]},{"cell_type":"code","source":["! head chr19_ldl_pheno/power=-1_her=0.1_num-causals=512.pheno"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"hWSEMaDwwB_f","executionInfo":{"status":"ok","timestamp":1710098572094,"user_tz":420,"elapsed":330,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"89b744ee-733e-47f7-c9ac-9ad3c24ef479"},"execution_count":65,"outputs":[{"output_type":"stream","name":"stdout","text":["0 HG00096 0.214735 -0.586387 1.190388 -2.553315 0.084348 -1.047627 -0.512360 -0.064191 -1.415388 0.152435 0.144013 -0.411257 -0.352695 0.309752 -0.740968 1.346553 -0.135816 -2.090853 -0.660930 -0.751440 -2.130044 -0.146256 -0.606272 1.602845 -1.135104 -0.798361 -0.051145 0.773401 -1.390134 -1.930970 -0.148289 -0.982136 -0.423887 0.042521 -0.793987 -1.270393 -0.537860 -0.314816 0.573735 0.628747 0.257875 0.570095 -1.236076 -1.113755 -0.402438 1.015649 0.297353 -0.913479 0.299884 -0.103145 -0.146835 -0.250061 -1.020750 -1.413435 1.293611 0.390204 0.636938 -0.011436 0.890836 -0.122534 -0.240916 1.220933 -0.128591 0.567709 -1.404210 -0.792828 -0.350823 -0.247983 -0.668984 0.206549 -0.899257 -0.246051 0.499500 -0.002118 -0.908138 -0.619517 -0.518779 -2.016719 -0.503736 -0.723267 -0.033540 -0.716983 -0.695527 -1.079612 -0.458206 1.282743 -2.372519 -2.039462 -1.865120 -0.658532 0.461358 -0.217413 0.089573 -1.233942 0.152930 -0.395815 -0.625800 -1.647235 -1.283727 0.437398 \n","0 HG00097 -0.638287 0.942861 -1.017351 0.990146 -0.659674 -1.889351 -0.803853 0.413517 -0.499380 0.724345 -1.649259 0.684620 -0.997724 0.189850 -1.718266 0.299091 0.216374 0.986509 -0.140799 -0.885249 -1.009474 -2.194012 -0.733400 0.602811 0.585136 -1.825058 0.884356 -0.790382 0.250648 -0.570762 -0.134655 -0.976301 -0.730559 -0.129026 0.534422 -0.060640 0.211283 -1.019919 1.415291 0.912547 -0.602652 -2.282343 -0.236939 -0.770260 0.492657 0.967230 -0.329129 1.757655 1.759822 0.223652 -1.240047 1.119082 -0.811167 -1.768581 0.923498 0.404668 0.315962 -0.799708 -0.962023 -0.150719 -1.609182 -0.249850 -0.411762 -0.657730 -0.815848 -2.125786 -0.046924 0.890259 -0.928534 0.759148 -0.209066 -0.050805 2.067712 0.470266 -0.399255 0.572554 -2.415643 0.322772 -1.578176 0.158023 -0.540669 0.652039 0.266426 0.323875 0.418047 0.509921 0.395598 0.181415 -2.651876 -0.407867 -0.999720 -0.278816 0.511088 0.930832 0.615275 -0.619625 0.508093 -0.395223 -0.440116 1.467202 \n","0 HG00099 1.294528 0.200792 1.552646 -2.346795 -0.332590 0.336371 0.351499 -1.361901 0.539569 -0.237268 0.920188 0.847971 -1.146543 -0.335359 0.352938 0.511842 -0.356038 0.550611 0.887395 -0.929530 0.344565 -0.366328 -0.031542 -1.485223 0.503591 0.263933 0.178763 0.406899 0.031328 -2.077789 1.544982 0.635280 -0.592537 -1.546991 0.797638 -0.026588 0.031492 -0.615193 1.504617 -0.312074 -1.321931 0.975000 0.799203 -0.012883 -0.234151 2.161457 0.124409 -0.153786 0.341403 -0.079476 0.207761 1.602678 0.351867 0.672943 -0.108884 -0.390900 -1.078253 -0.251645 1.187632 0.090275 -0.780957 -1.818401 -0.760282 -1.298465 0.263327 -1.124884 0.173150 1.104230 0.510891 1.709950 0.870767 -1.123228 -1.943477 1.273019 -0.317850 1.406449 0.165733 -0.333321 0.643644 1.401416 -1.233151 0.078910 2.008974 -0.511546 1.037424 0.853251 -1.039203 -0.884368 0.526239 2.574747 -1.208699 1.947982 -0.110478 0.225125 -0.022715 -0.250311 0.560139 1.054978 1.325151 -2.202582 \n","0 HG00100 1.123661 1.247301 -0.241202 1.526968 1.074802 -1.904278 -0.819497 -1.019615 0.894002 -0.000501 -0.660846 -1.099448 0.558296 0.462987 -2.257428 -1.390342 -0.668184 0.670267 -0.255505 2.218694 -0.311420 -0.951146 -1.776897 -0.021700 1.064285 -0.357031 -0.559898 -0.947863 0.138965 -0.979672 -0.198194 -0.709735 0.105574 0.298925 0.172050 -2.471586 -0.792360 -0.076948 -1.555977 -1.547852 -1.317034 -1.461805 -0.245277 -0.349987 -0.420265 -0.759039 0.380107 -0.078409 0.228323 0.158692 -0.098249 -0.438094 0.231501 -1.166369 0.055099 -2.480552 0.170585 -2.354877 0.128247 0.325877 -0.075550 1.101410 1.070731 -1.139064 0.016302 -0.455914 -1.802064 -0.372395 -0.313308 0.699457 -0.782848 0.189640 1.416594 -1.044506 0.038856 -1.109805 0.713389 0.317450 -0.876362 0.154836 -1.297414 0.083987 -1.409991 -2.702384 -1.260996 0.595026 -0.860620 0.249164 0.337384 0.354246 -1.151181 -1.245790 -0.655342 -0.613807 -0.283347 -1.121519 -0.076105 -0.374797 0.415466 1.378400 \n","0 HG00101 -1.965654 -0.818943 0.401132 -1.049124 -1.340226 -0.421136 -1.560131 -1.336912 0.641162 -1.359882 -0.693480 -0.099818 -0.898685 -0.639925 -1.367906 -0.152538 2.019814 0.182748 -1.121782 -0.770068 -0.283939 -1.972271 -1.057639 2.594665 -1.583965 -0.046977 2.217703 -1.326141 1.077887 -1.175551 -0.616619 0.067906 0.062465 -1.943398 -1.651842 0.087094 -1.469161 -0.763420 -0.746753 -0.871527 -1.301365 -1.672449 -0.714493 -1.956435 -0.358089 -0.354320 -1.017428 0.031408 0.527634 -1.488674 1.084279 0.327383 -0.050671 -1.259535 1.171625 -1.163134 0.254340 0.972455 1.339591 -0.327330 -0.357945 -0.621941 0.817511 -0.516651 -1.716765 -0.175286 -0.550698 -0.704525 0.062519 -0.649162 -0.322718 0.067753 0.137522 -0.825029 1.068856 -1.067643 -1.122738 -0.548317 -0.966192 1.259714 0.365152 -0.241223 0.725327 -0.200900 -2.026233 -1.273396 -0.736043 -0.846349 1.329355 -0.163114 -0.163066 0.336286 -0.282819 -0.852009 0.377389 0.107652 -1.213014 -1.922668 1.280579 0.500939 \n","0 HG00102 0.623893 1.259135 0.049065 0.929363 1.790477 1.056707 0.044915 0.113658 0.211677 2.261870 0.024373 0.464593 1.562253 0.129466 0.180542 0.063790 -0.397210 -0.269253 -0.223066 -0.268356 0.840552 0.332985 -0.151703 0.633098 0.937033 -0.690107 0.586588 -0.109777 0.389659 0.004958 1.119445 -0.271416 -0.265897 1.570413 0.323293 0.813662 -0.237810 -1.035004 -0.230984 0.661270 0.313825 0.596537 -0.476746 0.840728 -1.330036 0.697954 1.349221 0.731078 1.476115 0.534447 -0.450613 2.027968 -2.427093 0.223182 -0.240515 1.674236 1.413253 -0.380260 0.976686 -0.019551 -0.467242 -0.361260 0.376649 -0.048531 0.028055 1.568150 1.229562 0.117998 0.317485 -0.474773 -1.629380 1.855437 1.362156 0.025397 0.163298 0.346933 -0.025669 1.090095 -1.855890 1.700686 1.753584 1.574872 1.542490 0.332222 -1.548700 1.108274 -0.242851 -0.050136 2.013475 -0.521581 1.292797 0.287859 1.487096 -0.200835 -1.211283 0.034311 -0.978287 1.698219 -0.228841 1.343048 \n","0 HG00103 0.158690 1.626472 -0.080606 -0.190881 -0.300564 -1.884526 -1.015701 -2.082037 -1.020636 0.720109 -0.342145 -1.272063 1.495416 -1.024415 -0.101001 0.205657 0.065587 0.929766 0.702294 -0.034179 0.800367 -1.313723 -0.134114 1.066098 0.365321 -0.315840 -0.247356 -0.072969 -0.354351 1.096996 0.209022 -0.224324 -2.717821 -1.382507 0.570361 0.660674 -2.175098 0.206761 -2.154512 0.444996 0.011791 0.155406 0.560056 0.479912 -1.016466 -0.001451 -0.540830 0.678991 -0.300393 1.969477 -1.424719 -0.443076 -0.103913 -0.336774 0.631339 1.802398 0.711403 0.197864 1.326838 0.052849 -0.206416 -0.021022 1.038856 -1.177652 0.963871 -0.057088 0.960448 -0.456183 -0.445120 0.090728 -0.157742 0.556427 -1.389978 0.126522 0.342989 -0.106730 0.604921 -0.448158 1.414527 1.610822 0.112922 0.414642 -0.998645 1.432748 -0.626961 -1.445923 -0.397184 0.072589 1.095038 0.214910 0.164251 1.566410 -0.514499 -0.322836 -0.229835 -1.446058 -0.524621 0.250589 0.732252 1.116760 \n","0 HG00105 0.553191 -0.833250 0.064002 0.808094 1.256466 1.274760 -1.505223 -0.666542 0.217238 -2.132890 1.387805 -1.194009 0.827368 -1.685225 -0.267785 -0.241510 -0.425543 0.412653 1.301839 0.040544 -0.290935 -0.875325 0.299295 0.221106 -0.647062 0.964982 0.807789 1.784087 0.020882 -1.204468 0.209503 0.222306 0.646897 -0.208755 -1.035228 0.519053 1.729715 -0.508437 3.051293 -0.630736 -0.255543 -0.292253 0.522885 0.340148 -1.107554 -0.980490 -0.282060 1.726447 2.947257 1.183459 0.916893 1.144851 0.779109 0.242209 0.521988 -1.215354 0.706978 0.250719 0.503645 -2.229261 0.037106 0.366221 -0.884386 -0.286960 -0.415557 0.069093 1.990571 1.997812 0.284473 -0.042283 1.011690 -0.393372 -0.724102 -0.925927 0.525186 0.597245 1.712524 0.297896 2.190719 0.155377 -0.729918 1.453678 -0.102241 -0.231079 -1.285392 -0.131199 0.024971 -0.433159 0.639510 -0.107048 -0.816929 -1.412124 1.199749 -1.224989 -1.194248 0.230685 -0.299308 -0.794838 1.021375 0.029177 \n","0 HG00106 0.341682 -1.356905 0.543900 0.571697 -0.041318 -0.401217 0.212848 -0.421465 0.987857 -0.704451 -0.930627 -0.105155 -0.377139 1.097092 -0.427879 -2.153828 -0.175815 0.871695 0.635611 0.273283 0.067579 -0.360991 -0.192684 -0.090713 0.880748 0.363726 1.441532 0.175094 -1.275167 -0.545164 0.454662 0.417763 -0.090939 0.041573 -0.430255 -0.920049 -0.734448 0.296972 -0.773561 -2.075996 0.739464 -0.041248 -0.963882 0.541293 0.476534 0.107345 0.404776 -0.317505 0.308187 -0.922478 -1.927094 -1.329329 0.136517 -1.095499 -0.400496 -0.709339 -0.798002 -0.412841 0.133282 1.382174 1.536279 0.625511 0.778647 0.763228 0.147353 -0.129752 1.283714 0.503794 0.382993 0.863887 1.588864 0.945010 0.172316 0.959536 -2.175712 -0.600620 -0.402434 -0.218454 -0.397073 1.772543 0.587944 -0.165944 -0.885192 -1.127801 -1.358776 0.902458 0.249310 0.093309 0.467999 -1.257218 0.266038 0.019955 1.324312 -0.549599 -0.312580 -0.144643 1.418888 0.002267 0.564242 1.581688 \n","0 HG00107 0.890239 -2.475916 0.048007 -0.588310 0.547615 -1.747043 -0.276928 -0.612495 0.368219 0.549013 -1.498273 -0.590318 -0.905523 -0.161498 0.472743 0.180015 -2.177799 -1.085759 -0.062704 -1.075306 1.034153 -1.470007 -0.868849 -0.225835 0.155741 0.546145 -1.119548 -1.152872 -0.516612 -0.842169 -1.628330 -1.213880 0.605871 0.021054 -0.519657 1.685027 -0.207710 -1.379441 0.921834 1.482401 -2.850467 0.525861 1.533510 -0.645089 0.151948 -0.473810 -1.646561 -0.833912 -0.167142 0.036532 -1.589682 -1.183747 -1.078548 1.119467 -1.534396 -0.590316 0.481084 -0.437595 -0.008637 0.451141 -0.545931 1.269433 -0.535150 2.474649 -0.588997 -0.490887 -0.430437 1.220505 -1.268354 0.225823 -1.237851 0.148325 -1.086281 1.012872 -0.500461 -2.108676 0.528003 0.148934 1.023144 -0.608209 0.927792 -0.626628 -0.286763 1.203645 -0.910254 0.763441 -0.913071 -0.996936 -1.794013 1.867918 -1.017080 -0.477156 0.235385 -0.248897 -0.519333 -0.409141 0.238612 0.982173 0.622238 -1.132876 \n"]}]},{"cell_type":"code","source":["! head chr19_ldl_pheno/power=-1_her=0.9_num-causals=512.pheno"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"zpTBdeRfqTRH","executionInfo":{"status":"ok","timestamp":1710098592606,"user_tz":420,"elapsed":184,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"91b11088-c72b-4a51-b3c0-61ec467315a0"},"execution_count":66,"outputs":[{"output_type":"stream","name":"stdout","text":["0 HG00096 -1.899202 -1.041246 -1.443321 -1.732044 -1.200891 -1.060183 -1.020299 -0.547387 -1.842862 -0.907769 -1.825056 -1.391698 -1.608965 -1.555709 -1.502583 -1.149981 -1.375420 -1.379174 -1.135772 -0.992182 -1.478227 -1.422246 -1.279287 -1.430140 -1.483291 -1.915762 -1.008216 -1.610993 -1.404742 -1.981382 -1.373465 -1.076122 -1.021652 -0.956304 -1.295711 -0.772557 -1.270001 -0.836604 -1.535703 -0.810656 -0.785056 -1.205638 -0.836946 -1.278888 -1.009078 -1.160094 -1.580056 -1.533457 -1.299815 -1.701400 -1.429931 -0.592049 -1.271766 -1.323405 -2.117853 -1.378791 -1.424904 -1.143046 -1.171225 -1.597679 -1.293848 -1.119953 -0.920118 -0.912147 -1.166964 -1.058441 -1.310111 -1.642080 -1.450256 -1.528894 -1.204778 -1.429540 -1.051580 -0.988689 -1.474904 -1.556249 -1.415465 -1.356226 -2.103688 -1.315107 -1.237281 -1.219446 -1.355105 -1.611589 -1.263697 -1.352068 -1.201778 -1.278334 -1.393961 -1.301894 -1.605615 -1.015500 -2.351497 -1.232124 -1.630880 -1.163642 -1.506007 -1.393312 -1.246500 -0.809743 \n","0 HG00097 -0.379556 -0.345065 -0.112039 0.064766 0.172122 0.228262 -0.214366 -0.421242 -0.217297 -0.032437 0.066399 0.132044 -0.051028 -0.271505 -0.374500 -0.819684 -0.034834 -0.311421 -0.175942 -0.238333 -0.348811 0.217063 0.341153 -0.312744 -0.133984 -0.182666 0.236934 -0.142990 0.219950 0.116424 -0.008091 -0.214255 0.019343 -0.288326 -0.708285 -0.212347 -0.557774 0.500092 -0.337541 -0.420029 -0.209745 -0.355463 -0.337039 0.434334 0.241679 0.214913 -0.110940 0.046721 0.217309 0.614983 -0.045451 -0.103354 -0.367499 -0.297790 0.036514 -0.349255 0.010886 -0.637110 -0.947358 -0.285678 0.398866 -0.328755 0.687280 0.334291 -0.223081 -0.421902 0.052165 -0.180721 0.169125 -0.024687 0.094568 -0.495994 -0.301991 -0.245867 0.605259 -0.011702 -0.456515 0.016003 -0.643817 -0.043524 -0.704032 -0.109641 -0.101298 -0.113224 -0.169461 -0.164022 0.346957 -0.181939 -0.521000 -0.080545 -0.377359 -0.286961 -0.111480 0.111415 -0.146678 -0.788718 -0.091183 -0.583044 -0.533360 0.024159 \n","0 HG00099 0.141943 0.473072 0.158021 0.622426 -0.030839 0.313654 0.035323 0.470899 0.333639 0.302082 0.086713 0.146339 0.566243 -0.231363 -0.000065 0.948400 0.200167 0.378902 -0.241826 0.453345 0.130160 0.060377 0.404676 0.468065 0.046136 -0.022175 -0.591308 0.410334 0.183112 0.624923 -0.464972 0.202811 0.258077 0.485556 0.024920 0.486689 0.605263 0.047644 0.189086 0.511978 0.218407 -0.283105 0.030729 -0.280997 -0.045374 -0.101269 -0.249181 0.426425 0.121698 -0.190742 0.299221 0.034804 -0.470770 0.173446 0.292909 0.093574 0.222295 0.377528 -0.069246 -0.511221 0.440147 0.018533 0.038654 -0.087247 0.104222 0.124023 -0.186593 0.061808 0.055228 0.274041 0.424919 -0.761492 0.488651 0.492534 -0.030610 -0.237972 0.136881 0.286121 0.241906 -0.545988 0.134892 0.153109 -0.171401 0.384892 0.160824 -0.025808 -0.197663 -0.428543 -0.167558 0.465923 0.130143 0.246554 -0.236017 0.194480 0.199885 0.219790 -0.029022 0.305666 -0.306366 0.330933 \n","0 HG00100 -0.622421 -0.407337 -1.382916 -0.104484 -0.835572 -0.975154 -0.652388 -0.124815 -0.411298 -0.517011 -0.791366 -1.083932 -0.953880 -0.435262 -1.044622 -0.658126 -0.792755 -0.627887 -0.910846 -0.584924 -0.873623 -0.922172 -0.435548 -0.360697 -0.416605 -0.707356 -0.706950 -0.399472 -0.995818 -1.014063 -0.293622 -0.380997 -0.596370 -0.789544 -0.678562 -0.498696 -0.677925 -0.677108 -1.099567 -0.915196 -0.770809 -1.160635 -0.987924 -0.588342 -0.921882 -0.753634 -1.018873 -0.286409 -0.773637 -0.592055 -1.083637 -0.845526 -1.397421 -0.741215 -0.665897 -0.762864 -0.974068 -0.686983 -0.578788 -0.726483 -1.138721 -1.042104 -0.595280 -1.120207 -1.050620 -0.822573 -0.666290 -1.007406 0.329753 -0.667375 -0.902356 -0.764043 0.005882 -0.583696 -1.285589 -0.900854 -0.292722 -1.097654 -0.744307 -0.444253 -0.744624 -1.084951 -1.263451 -0.392221 -0.352432 -0.280332 -0.249604 -0.708402 -1.090059 -0.737327 -0.837749 -0.769424 -0.909622 -0.652720 -0.818819 -0.121479 -0.905485 -0.688437 -0.571404 -0.504763 \n","0 HG00101 -0.921933 -1.535862 -1.346244 -0.701908 -1.205896 -1.486772 -0.931368 -1.320459 -1.125926 -1.737015 -1.754664 -1.199141 -1.192042 -0.974798 -1.496633 -1.174389 -1.282848 -1.364816 -1.356320 -1.752778 -1.494813 -1.653416 -1.301052 -1.230437 -1.166391 -1.184611 -0.796465 -1.441433 -1.154737 -0.794525 -0.855750 -1.151155 -1.568101 -0.939415 -1.419636 -1.179443 -1.127263 -1.088456 -1.549530 -1.085272 -1.484902 -1.962145 -0.080234 -1.058023 -1.505291 -1.243082 -1.374256 -0.807495 -0.788739 -1.443870 -1.313722 -1.207659 -1.007884 -1.051370 -1.326709 -1.059227 -1.275982 -0.992755 -1.278022 -0.970830 -1.638637 -0.752117 -1.421717 -1.162939 -1.382306 -1.797691 -1.663943 -0.711245 -1.512932 -1.368954 -0.666436 -1.579619 -0.864894 -1.405869 -1.769277 -1.144801 -1.220010 -1.892009 -1.430801 -0.926973 -1.492251 -1.370287 -1.061602 -0.682328 -0.948765 -0.926798 -1.349958 -1.299436 -1.807964 -0.973732 -1.115329 -1.390435 -1.781292 -0.872598 -1.362881 -1.222850 -1.120062 -1.404089 -1.279816 -1.548861 \n","0 HG00102 0.792442 0.974144 1.341908 0.436392 0.997738 1.046680 1.009923 0.821979 0.452101 0.807098 1.118030 0.854383 0.753072 0.850003 0.962298 0.256192 0.542264 0.120717 0.182671 0.816292 0.948767 0.463308 -0.002332 0.752982 0.557841 0.897032 0.434847 0.928279 0.878774 0.573134 0.745855 1.122195 0.215392 0.475696 0.598586 0.933273 0.378177 0.855231 0.690539 0.644663 0.356112 1.218801 0.500574 0.765463 0.630569 0.463211 0.963380 1.023633 0.869763 0.679137 0.289767 1.049527 0.708147 1.256470 0.323217 0.768757 0.723831 1.011402 1.015434 1.193155 0.173807 0.587741 0.841276 1.533418 0.705914 1.074250 0.826465 0.798892 1.202521 0.573857 0.776669 0.619423 0.658337 0.508835 1.055186 0.738848 1.020974 0.966951 1.011923 0.655547 1.255357 0.717926 0.677095 0.350849 0.761731 0.824532 0.412257 1.029882 0.658640 0.869678 0.577573 0.217458 0.857413 0.466964 0.396239 1.234487 0.508024 1.128527 0.235294 0.546724 \n","0 HG00103 0.585911 -0.687698 0.230281 0.215792 0.039610 -0.278614 -0.442470 -0.879235 0.326995 0.000734 0.180033 0.315584 -0.071767 0.033146 0.043861 -0.247837 0.097926 -0.068779 0.482161 -0.068269 -0.347628 0.294619 -0.211238 0.271093 -0.152838 0.170708 -0.309963 0.160557 0.251566 -0.072066 -0.243770 0.191946 0.252835 -0.066411 0.486746 -0.096011 0.212012 0.071994 0.244731 0.071664 0.272329 0.161562 -0.520681 0.088159 -0.004616 0.310724 -0.049781 -0.013642 -0.259508 -0.298360 0.389049 -0.165624 0.192547 -0.108549 -0.081955 -0.166085 -0.036094 -0.120478 0.124376 -0.253690 -0.326758 0.318569 0.087681 0.008434 0.304995 0.053116 -0.056945 0.199678 -0.287813 0.168261 -0.111610 0.435714 0.268544 -0.369234 0.607206 0.229924 -0.181993 -0.138187 -0.562940 -0.099056 0.350211 0.180768 -0.537866 0.212028 -0.326686 -0.614803 0.484411 -0.069382 -0.148766 -0.053647 0.742548 -0.273228 0.093886 0.303357 0.609236 -0.010358 -0.223223 -0.349168 0.277820 -0.188250 \n","0 HG00105 -0.301709 -0.515345 -0.253119 -0.263568 -0.286359 0.153898 -0.566097 0.064934 -0.499219 -0.020925 0.228973 -0.661856 -0.435899 -0.001565 -0.243456 0.602707 0.033214 0.217800 -0.361763 -0.505390 -0.311324 -0.002468 0.254876 0.209154 -0.436321 0.036865 0.163707 -0.241951 0.167152 -0.370867 0.106399 -0.453578 0.077547 -0.407949 0.305064 -0.479526 0.233607 0.333160 0.120168 -0.008863 -0.451891 0.011611 0.488725 -0.245264 -0.675794 0.412304 0.085059 0.145982 -0.360855 -0.208730 -0.167083 -0.768591 0.381064 -0.288028 -0.433601 0.139635 0.295994 -0.148964 -0.600851 -0.173333 -0.597797 0.234829 -0.067085 0.238363 0.255021 -0.268157 -0.338237 -0.063064 -0.449667 -0.205391 0.319704 -0.076995 -0.356677 0.228624 -0.249381 -0.084862 0.115590 -0.738857 0.370792 -0.531776 -0.123655 -0.070705 0.223092 0.122151 -0.045879 0.512255 -0.295067 0.213302 0.157512 0.510952 0.243616 -0.029752 -0.724669 -0.450347 -0.194493 -0.067607 0.080973 0.423516 0.022015 0.927382 \n","0 HG00106 0.462228 -0.560828 -0.161148 -0.858381 -0.386992 -0.071992 -0.541533 -0.160354 -0.274098 0.048308 0.033717 -0.383529 0.361067 0.420466 -0.372204 -0.354835 -0.306093 0.473387 -0.333836 0.206705 -0.084203 0.007877 -0.021319 0.134263 -0.409783 0.348610 -0.150295 -0.217102 0.401627 0.137113 -0.050728 -0.081603 0.438995 -0.191827 0.185975 0.396098 -0.321910 -0.465039 -0.533360 0.406861 -0.259819 -0.178021 0.168104 -0.725231 0.120133 -0.087141 -0.115148 -0.059438 -0.051112 -0.185610 0.135427 -0.123199 -0.355560 -0.568491 -0.139714 -0.672166 -0.545693 0.151055 -0.289931 -0.263013 0.509531 -0.193248 -0.269447 -0.392788 0.335683 -0.619362 -0.436501 -0.090378 0.335339 -0.254531 -0.522956 -0.105983 -0.057856 -0.343317 0.105083 0.194887 -0.262628 -0.315608 0.264696 -0.128112 -0.310866 -0.412751 0.108636 -0.757544 0.286813 0.310114 0.031675 -0.217064 -0.075125 -0.026039 -0.059464 -0.148017 -0.443587 0.002108 -0.166718 0.105707 -0.045940 -0.011623 -0.044919 0.302027 \n","0 HG00107 -1.114487 -0.710916 -0.748805 -0.882620 -0.987595 -0.566089 -1.147765 -0.701064 -1.324007 -0.940942 -0.571452 -0.955506 -0.478425 -1.494316 -0.835007 -1.273517 -1.072123 -1.491629 -0.779884 -0.647087 -0.865479 -0.667198 -0.533423 -0.834980 -0.912430 -1.034307 -0.566126 -1.511626 -0.791156 -1.168434 -1.351734 -0.565563 -1.028257 -0.256375 -0.853611 -0.643211 -0.689859 0.273470 -0.853128 -0.315503 -0.330841 -0.686689 -0.979301 -1.424147 -0.657692 -0.804034 -1.379338 -0.721725 -0.647631 -1.224657 -0.598626 -0.902786 -0.969175 -1.052812 -0.907873 -1.146742 -0.621936 -0.991787 -0.714765 -1.264735 -0.574705 -1.089142 -1.223832 -0.627859 -0.953061 -1.009540 -0.724896 -1.133366 -0.559882 -1.306703 -0.888944 -1.557002 -1.363197 -1.338129 -1.058137 -1.361597 -0.826277 -0.808874 -0.335992 -0.908656 -0.721804 -1.118103 -0.687731 -0.839251 -0.930435 -0.723776 -0.587948 -1.470963 -0.823451 -0.795167 -1.137372 -0.370392 -0.604820 -0.800780 -0.422709 -0.917277 -1.022735 -1.304627 -1.249471 -0.447894 \n"]}]},{"cell_type":"code","source":["! head chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=512.pheno"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"Xk1lc29JqS8O","executionInfo":{"status":"ok","timestamp":1710098610960,"user_tz":420,"elapsed":353,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"b8b7c6e4-b442-4b17-efef-f1604783f05b"},"execution_count":67,"outputs":[{"output_type":"stream","name":"stdout","text":["0 HG00096 -1.107044 -1.215778 -1.704484 -0.995185 -1.365799 -0.621652 -0.676746 -1.512593 -1.780164 -1.439749 -0.554559 -1.151093 -0.893698 -1.347664 -1.616704 -0.758645 -0.968776 -0.754217 -1.749669 -1.205287 -1.242741 -1.169732 -1.659914 -1.205588 -1.091213 -1.795153 -1.257381 -1.495473 -1.420209 -1.619440 -1.255575 -1.512802 -0.891579 -1.356832 -1.406791 -1.883521 -0.806860 -1.113352 -1.661203 -1.185024 -1.192756 -0.825975 -1.023094 -2.000873 -1.270047 -1.145541 -1.576674 -1.648591 -0.913374 -1.323303 -1.056818 -1.172338 -1.244784 -1.991998 -1.495054 -1.557855 -1.480354 -1.612820 -0.898757 -1.189610 -1.511485 -1.593118 -1.249223 -1.876525 -0.631540 -0.521159 -1.637079 -1.374904 -1.677736 -1.415298 -0.881230 -1.253788 -1.465289 -1.571255 -1.116468 -1.605500 -1.460020 -1.811784 -0.971336 -1.193030 -1.643454 -1.607149 -1.462569 -1.256204 -1.458407 -0.947535 -1.083158 -1.856140 -1.581749 -1.553019 -1.284812 -1.485012 -1.249776 -1.057600 -1.041404 -1.730156 -1.030367 -1.350577 -1.667788 -1.540609 \n","0 HG00097 0.061141 0.276746 -0.336033 -0.243652 0.105273 -0.328281 -0.074516 -0.270201 0.192229 -0.791691 -0.951750 -0.300054 -0.083742 -0.250512 -0.137375 -0.374368 -0.572081 0.251400 -0.308336 0.149873 -0.123529 0.012472 -0.316619 -0.529751 -0.393948 -0.382940 -0.573619 0.260762 -0.185121 -0.049752 -0.637312 0.016416 -0.678693 -0.286577 -0.617917 -0.153151 -0.216164 -0.045142 -0.375768 -0.311106 -0.181270 -0.229560 -0.229545 -0.142868 0.095700 -0.265193 -0.151682 -0.313258 -0.211641 -0.565644 0.091299 -0.154014 -0.071658 -0.651919 -0.443481 0.154333 -0.414663 0.365385 -0.514566 -0.274972 -0.034924 -0.073351 -0.189251 0.438308 -0.495570 -0.035999 -0.415048 -0.237424 0.045525 -0.483347 0.067485 0.127279 -0.241243 0.102586 -0.002260 -0.237804 -0.179298 0.149135 -0.614031 -1.000021 -0.134801 -0.062443 -0.528452 -0.357641 -0.381305 0.566733 0.059838 -0.688483 -0.195769 -0.587166 -0.610648 -0.702705 -0.522796 0.012212 0.000561 -0.241975 -0.867900 -0.241677 -0.328935 -0.365468 \n","0 HG00099 -0.124759 -0.265935 -0.263927 0.298597 -0.050895 -0.207945 -0.377366 -0.365726 -0.189529 0.585752 0.043218 -0.294363 0.175254 0.501686 0.122216 -0.150422 -0.566792 0.185545 0.095513 0.008488 0.390959 0.189105 -0.598935 0.074493 0.031883 -0.175638 -0.508223 -0.106279 -0.758334 -0.021333 -0.288481 -0.210967 -0.061121 -0.128388 0.878389 0.293252 -0.667002 -0.217126 -0.122252 -0.114341 -0.033190 -0.337508 -0.132296 -0.131396 -0.414811 -0.481030 -0.027083 -0.466382 -0.511348 -0.063210 -0.018804 -0.271288 -0.371973 -0.382976 -0.217461 0.003931 0.325228 -0.111006 0.156069 -0.451189 -0.158994 0.244402 -0.348767 0.033445 0.135337 0.035766 -0.666425 -0.104265 -0.583698 0.308480 -0.094303 0.541069 -0.348413 -0.166695 0.323397 0.208019 0.544924 0.325606 -0.118404 0.143895 -0.033716 -0.406252 -0.119657 -0.303954 0.639299 0.322040 -0.354490 0.096512 0.280529 0.352890 -0.012029 -0.324316 -0.031047 -0.003712 0.392446 -0.077712 0.239459 0.120101 -0.080219 -0.579251 \n","0 HG00100 0.352577 -0.121414 -0.462982 -0.584993 -0.957590 -0.605769 0.113541 -0.671174 -0.248349 -0.533817 -0.271653 -0.702703 -0.499299 -0.853636 -0.275426 -0.476259 -0.784868 -0.508783 -0.045880 -1.008572 -0.252044 -0.730449 -0.506167 0.380471 -1.078689 -0.557336 -0.768373 -0.230971 -0.063862 0.341556 -0.872769 -0.366940 -0.344180 -0.629376 -0.142823 -0.226765 -0.637475 -0.351212 -0.030845 -1.006158 0.143496 -0.485736 -0.662264 -0.477761 -0.448272 -0.381438 -0.864687 -0.894791 -0.892767 -0.396727 -0.356570 -0.441736 -0.881756 -1.027540 -0.944602 -0.600848 -0.727500 -0.750777 -0.956822 -0.427504 -0.444945 -0.166557 -0.766292 -0.267348 -0.707823 -1.072072 -0.560737 -0.186675 0.094045 -0.258952 -0.656884 -1.445599 -0.417892 -0.504271 -0.558456 -0.520294 -0.889403 -0.214366 0.125777 -0.865879 -0.880477 -0.389407 -0.172116 -0.516642 -0.200934 -0.405045 -0.031256 -0.065160 -1.080455 -0.577414 -0.828473 -0.329563 -0.427509 -0.566451 -0.514774 -0.294353 -0.509005 -0.593444 -0.143696 -1.285120 \n","0 HG00101 -0.892331 -0.497496 -1.297303 -1.422271 -0.840788 -0.780878 -0.913317 -0.719586 -1.047514 -0.704806 -1.050653 -0.920153 -1.558383 -1.440560 -0.030863 -0.523785 -0.792022 -0.599672 -0.703285 -1.099319 -0.167003 -0.658450 -1.252940 -1.529545 -0.406747 -0.390330 -1.190454 -0.914427 -1.345319 -0.829103 -1.104011 -1.166383 -1.486877 -1.116061 -0.872120 -0.379000 -1.669611 -1.296991 -0.687847 -0.307406 -1.246294 -0.923490 -0.760080 -0.936811 -1.417227 -0.998062 -0.845242 -1.023279 -0.981842 -1.021436 -0.615336 -1.970481 -0.593660 -0.812354 -1.334352 -0.508988 -0.780203 -0.295663 -1.029922 -0.844462 -1.250250 -0.837642 -0.988226 -1.000216 -1.435683 -0.890193 -1.593804 -0.926972 -0.637251 -1.524277 -0.877668 -0.635593 -1.221966 -0.584194 -0.445566 -1.487782 -0.802923 -0.902334 -1.326299 -0.839052 -1.098779 -1.267037 -1.080707 -1.521145 -0.937567 -0.968392 -0.990192 -0.883634 -1.151284 -0.940739 -1.629606 -0.804379 -0.880948 -0.684539 -0.227283 -0.983990 -1.128809 -1.032425 -0.990394 -1.194459 \n","0 HG00102 1.018474 0.888480 0.599291 0.547443 0.357456 0.457148 -0.038816 0.624746 0.811574 0.453314 1.016054 1.462925 0.378512 0.519540 0.607520 0.118541 1.129229 0.713688 0.958878 0.932095 0.501086 0.580289 0.755989 0.257780 0.951111 0.631294 1.076932 0.110714 0.973620 0.525798 0.789428 0.727906 0.735208 0.307116 0.019540 0.766241 0.963814 0.817407 0.677679 0.351601 0.467430 0.729490 0.583704 1.492542 0.447528 0.449802 0.865082 0.824112 0.740891 0.757498 1.267994 0.438970 0.643686 0.406297 0.591952 0.672958 0.933139 0.864949 1.374556 1.140941 0.676469 0.404532 0.266042 1.121187 0.779454 0.918365 0.519445 0.854096 0.591108 0.640881 0.872392 1.098707 0.712358 0.416566 0.249432 0.855037 0.574666 1.136544 1.171027 0.771648 0.572949 1.049630 0.401161 0.691093 0.413992 0.734896 0.567091 1.224263 0.868028 0.023743 0.849412 1.416825 0.576066 0.736445 0.664086 0.333053 0.795659 0.773475 0.939130 0.612823 \n","0 HG00103 0.752945 0.873346 0.429859 0.777275 0.906057 0.576875 0.980707 0.300652 0.460118 1.218236 0.604523 0.627684 0.487275 0.565768 1.108942 0.851888 0.532626 0.353453 0.946771 0.607216 1.107249 0.912821 0.637681 0.673613 0.574630 0.766061 0.675549 0.554015 0.647998 0.066572 0.499863 0.629211 0.810496 0.547175 -0.012965 0.825919 0.617043 1.060100 0.392845 0.476258 0.599076 0.538957 0.261891 0.988222 1.135107 0.633469 0.333131 0.881153 0.632326 0.548048 0.448182 0.411921 0.592376 0.683784 0.553926 1.375359 0.574162 0.724186 0.698726 0.684579 0.249535 0.669949 0.429277 0.470450 0.282817 0.469268 0.385445 0.870161 1.323589 0.455328 0.518803 0.092269 0.182345 0.879943 0.196990 1.033700 0.313854 0.695577 0.812476 0.571895 0.545718 1.033388 0.660181 0.222641 1.071359 0.882909 0.508606 0.367952 0.612818 0.809423 0.639064 0.573228 0.519208 0.598806 0.072485 0.843397 0.625420 0.734864 0.597824 0.570963 \n","0 HG00105 0.046831 -0.465255 0.288171 0.275767 0.383178 -0.185571 -0.202654 0.274781 0.283854 0.422048 -0.499559 -0.122286 -0.078891 0.770432 0.086727 0.335076 0.454321 0.063173 0.189350 0.217395 0.865436 0.681184 -0.747194 0.281058 0.063541 0.232111 -0.002534 0.536476 0.521571 0.625690 -0.016472 0.038393 0.361513 0.211695 0.134955 -0.396814 0.192938 -0.126624 -0.149869 -0.131128 0.413952 0.149316 0.100873 0.360873 -0.176355 0.303386 -0.049914 0.078858 -0.112714 -0.294187 0.282875 0.106899 -0.075914 0.678643 0.342697 0.197425 0.566503 0.722072 0.005958 -0.047182 0.338456 0.033726 0.070334 0.437648 0.552332 0.536775 0.471310 0.073399 0.363597 0.442180 0.727358 0.579465 0.164481 -0.302208 0.170308 -0.075565 -0.281107 0.031484 0.597472 0.280398 -0.096211 -0.041776 0.266481 -0.129649 0.443176 -0.578901 0.140625 0.420120 0.645807 -0.129645 0.404185 0.424202 -0.026973 0.239478 -0.084217 1.030710 0.062388 -0.055284 0.606994 -0.177444 \n","0 HG00106 0.298394 0.300589 0.347745 0.197673 -0.477462 0.610304 0.089345 0.177671 0.912129 -0.033185 -0.274867 -0.274250 -0.356277 0.020981 0.167387 0.230535 -0.098930 0.176154 -0.011547 0.561812 0.364790 0.214907 0.139901 0.418060 0.566949 0.005401 0.452226 -0.028420 0.607501 -0.086121 -0.571194 -0.385099 0.027436 -0.026012 0.561418 -0.317307 -0.511473 -0.295209 0.315759 0.199775 0.120935 0.129784 0.293428 -0.106558 -0.197191 0.397908 0.667036 -0.736511 0.241337 -0.961309 0.502901 0.139847 0.005197 0.369545 -0.114791 -0.220588 -0.320954 0.233563 0.052506 -0.431468 0.693725 0.319125 -0.255582 -0.165648 0.263419 0.118339 -0.097729 -0.088505 0.337203 0.142103 0.853913 0.406476 0.234815 -0.169002 0.670210 0.325474 0.025314 -0.049616 -0.051029 0.367551 -0.347968 0.238458 -0.267991 -0.064425 -0.403787 0.234542 0.182050 -0.373895 0.037639 0.258193 0.062222 0.088180 -0.274287 -0.237486 0.253232 -0.247182 0.505613 0.174075 -0.063950 0.059682 \n","0 HG00107 -1.185676 -1.091113 -0.548848 -1.079809 -0.036904 -1.088533 -1.447718 -0.790887 -1.455847 -0.365173 -1.221407 -0.751149 -1.324334 -1.275528 -1.182172 -1.042591 -1.217286 -1.231422 -1.227643 -0.986069 -0.728270 -0.992761 -0.704762 -0.960856 -0.955238 -0.841983 -0.562027 -1.155898 -0.668335 -1.290368 -1.264784 -1.466118 -1.016216 -1.298809 -1.102390 -0.707673 -0.640984 -1.291776 -0.918258 -1.317889 -0.885637 -1.308606 -1.228161 -1.225284 -0.988796 -1.380014 -0.900450 -1.209252 -0.309784 -0.790194 -1.520265 -1.148825 -1.188862 -1.139752 -1.034282 -1.444227 -1.096655 -1.242395 -0.904198 -0.945496 -1.377664 -1.166032 -0.779194 -0.362705 -1.083888 -0.857579 -0.949042 -1.692479 -1.159324 -0.747696 -0.773295 -1.160039 -1.169195 -1.132996 -1.334045 -0.859734 -0.368845 -0.604910 -1.421193 -1.103916 -0.461326 -1.153506 -0.929239 -1.014743 -1.232879 -0.654811 -1.124434 -0.466901 -1.376415 -1.065113 -1.199436 -0.519731 -0.649717 -1.477272 -1.104984 -1.071814 -1.003096 -1.168123 -1.117870 -1.272265 \n"]}]},{"cell_type":"code","source":["num_causals = 10\n","effects_chr19.sample(n=num_causals, random_state=1)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":363},"id":"j3xh6SF13ynx","executionInfo":{"status":"ok","timestamp":1710099543950,"user_tz":420,"elapsed":153,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"e027a50c-cbff-4f68-c388-dc6e6e77f8e8"},"execution_count":68,"outputs":[{"output_type":"execute_result","data":{"text/plain":["             snp  effect  chr  pos     coord alt ref\n","47   rs185574863 -0.0476   19    0  11020435   C   A\n","345   rs34256230 -0.0212   19    0  10596480   A   G\n","284   rs36005514 -0.0676   19    0  11190764   A   G\n","221   rs55734122 -0.0089   19    0  58733610   T   G\n","503    rs3848609  0.0150   19    0  55753433   A   G\n","168   rs10401338 -0.0098   19    0   4464014   G   A\n","223   rs62131325 -0.0109   19    0    770714   T   C\n","189   rs10408750 -0.0094   19    0  19063806   G   A\n","378    rs1441617 -0.0187   19    0  45056421   A   G\n","192   rs11879991  0.0264   19    0  11282983   C   A"],"text/html":["\n","  <div id=\"df-77d3141a-accc-401e-a023-abc7a170683b\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>snp</th>\n","      <th>effect</th>\n","      <th>chr</th>\n","      <th>pos</th>\n","      <th>coord</th>\n","      <th>alt</th>\n","      <th>ref</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>47</th>\n","      <td>rs185574863</td>\n","      <td>-0.0476</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>11020435</td>\n","      <td>C</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>345</th>\n","      <td>rs34256230</td>\n","      <td>-0.0212</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>10596480</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>284</th>\n","      <td>rs36005514</td>\n","      <td>-0.0676</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>11190764</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>221</th>\n","      <td>rs55734122</td>\n","      <td>-0.0089</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>58733610</td>\n","      <td>T</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>503</th>\n","      <td>rs3848609</td>\n","      <td>0.0150</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>55753433</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>168</th>\n","      <td>rs10401338</td>\n","      <td>-0.0098</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>4464014</td>\n","      <td>G</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>223</th>\n","      <td>rs62131325</td>\n","      <td>-0.0109</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>770714</td>\n","      <td>T</td>\n","      <td>C</td>\n","    </tr>\n","    <tr>\n","      <th>189</th>\n","      <td>rs10408750</td>\n","      <td>-0.0094</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>19063806</td>\n","      <td>G</td>\n","      <td>A</td>\n","    </tr>\n","    <tr>\n","      <th>378</th>\n","      <td>rs1441617</td>\n","      <td>-0.0187</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>45056421</td>\n","      <td>A</td>\n","      <td>G</td>\n","    </tr>\n","    <tr>\n","      <th>192</th>\n","      <td>rs11879991</td>\n","      <td>0.0264</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>11282983</td>\n","      <td>C</td>\n","      <td>A</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-77d3141a-accc-401e-a023-abc7a170683b')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-77d3141a-accc-401e-a023-abc7a170683b button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-77d3141a-accc-401e-a023-abc7a170683b');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-d56db234-f3ca-4d84-9601-4d028e6d7f9e\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-d56db234-f3ca-4d84-9601-4d028e6d7f9e')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-d56db234-f3ca-4d84-9601-4d028e6d7f9e button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","summary":"{\n  \"name\": \"effects_chr19\",\n  \"rows\": 10,\n  \"fields\": [\n    {\n      \"column\": \"snp\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"rs1441617\",\n          \"rs34256230\",\n          \"rs10401338\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"effect\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0.027095102058408184,\n        \"min\": -0.0676,\n        \"max\": 0.0264,\n        \"num_unique_values\": 10,\n        \"samples\": [\n          -0.0187,\n          -0.0212,\n          -0.0098\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"chr\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 19,\n        \"max\": 19,\n        \"num_unique_values\": 1,\n        \"samples\": [\n          19\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pos\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 0,\n        \"max\": 0,\n        \"num_unique_values\": 1,\n        \"samples\": [\n          0\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"coord\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 21764032,\n        \"min\": 770714,\n        \"max\": 58733610,\n        \"num_unique_values\": 10,\n        \"samples\": [\n          45056421\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"alt\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 4,\n        \"samples\": [\n          \"A\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"ref\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 3,\n        \"samples\": [\n          \"A\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"}},"metadata":{},"execution_count":68}]},{"cell_type":"code","source":["sample_size = 100\n","\n","for num_causals in [1, 10, 50, 100, 250]:\n","\n","    sample_causals_list = list()\n","    sample_effects_list = list()\n","\n","    for i in range(sample_size):\n","        sampled_snps = (effects_chr19[['snp', 'effect']]\n","                        .sample(n=num_causals, random_state=i)\n","                        .transpose())\n","        sampled_causal_row = (sampled_snps\n","                            .loc[[\"snp\"]]\n","                            .reset_index(drop=True)\n","                            .set_axis(range(sampled_snps.shape[1]), axis=1))\n","        sampled_effect_row = (sampled_snps\n","                            .loc[[\"effect\"]]\n","                            .reset_index(drop=True)\n","                            .set_axis(range(sampled_snps.shape[1]), axis=1))\n","        sample_causals_list.append(sampled_causal_row)\n","        sample_effects_list.append(sampled_effect_row)\n","\n","    sample_causals_df = pd.concat(sample_causals_list, axis=0, ignore_index=True)\n","    sample_effects_df = pd.concat(sample_effects_list, axis=0, ignore_index=True)\n","\n","    sample_causals_df.to_csv('causals_effects/causals_num-causals='+str(num_causals)+'.txt',\n","                             sep=\"\\t\", header=False, index=False)\n","    sample_effects_df.to_csv('causals_effects/effects_num-causals='+str(num_causals)+'.txt',\n","                             sep=\"\\t\", header=False, index=False)\n"],"metadata":{"id":"13heWw033y06","executionInfo":{"status":"ok","timestamp":1710101379768,"user_tz":420,"elapsed":2061,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":90,"outputs":[]},{"cell_type":"code","source":["sample_causals_df"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":444},"id":"0vBt_Kv63zML","executionInfo":{"status":"ok","timestamp":1710100796197,"user_tz":420,"elapsed":132,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"d6f6420d-5004-4ff3-9dff-792b89b2f305"},"execution_count":84,"outputs":[{"output_type":"execute_result","data":{"text/plain":["            0            1            2            3            4    \\\n","0    rs28399607  rs112501445   rs76936640  rs145639166     rs644699   \n","1   rs185574863   rs34256230   rs36005514   rs55734122    rs3848609   \n","2      rs238410  rs141441332  rs116283165   rs62118464   rs73051677   \n","3    rs56315738    rs2288450   rs62109991    rs4564699   rs74931926   \n","4   rs148613044  rs553008653   rs11879991    rs2927486   rs11665702   \n","..          ...          ...          ...          ...          ...   \n","95   rs75161053    rs4803825    rs8109764   rs73563690   rs11666216   \n","96   rs11671372   rs10451476  rs140915448   rs41301949  rs551557788   \n","97  rs139140823   rs34768259     rs238410    rs2238551     rs204480   \n","98  rs112283362  rs138218778   rs79322367   rs11673631  rs143043127   \n","99   rs79860339  rs117970794   rs17262997    rs2620840   rs10420562   \n","\n","            5            6            7            8            9    ...  \\\n","0    rs10403723      rs10129    rs2972564   rs73591055    rs1003393  ...   \n","1    rs10401338   rs62131325   rs10408750    rs1441617   rs11879991  ...   \n","2     rs2074940  rs142585929   rs35763270   rs34561079   rs35678764  ...   \n","3     rs8100933   rs10415949  rs113357289  rs117777502   rs34964177  ...   \n","4     rs7246718   rs35334454  rs144159978   rs58542926    rs8110775  ...   \n","..          ...          ...          ...          ...          ...  ...   \n","95    rs7254990  rs113357289    rs4808595   rs58513885  rs143043127  ...   \n","96  rs112056827  rs140842514  rs117970794  rs141964576    rs2007774  ...   \n","97  rs143059942     rs435380    rs8109764   rs56315738   rs10420349  ...   \n","98    rs3810333   rs59715573  rs577349548   rs11665702   rs58513885  ...   \n","99   rs58496711   rs11669345    rs3093032   rs73048053   rs11880312  ...   \n","\n","            240          241          242          243          244  \\\n","0    rs78961098     rs429358  rs149183396   rs11670136    rs4804149   \n","1    rs17714761  rs141366665   rs10409774   rs10417167   rs59848316   \n","2     rs8107753   rs11667610  rs147871592   rs10417167    rs4804510   \n","3    rs72654437   rs11673516  rs142385484   rs55766194     rs281417   \n","4     rs2335534   rs34964177   rs75920121   rs10221516   rs17262997   \n","..          ...          ...          ...          ...          ...   \n","95   rs10420562     rs916694    rs8107329   rs11085327   rs62132801   \n","96  rs146901672   rs62111695   rs62119326    rs1441617   rs72992965   \n","97  rs117607802   rs76649675     rs421812   rs11668344   rs72997195   \n","98  rs141651018   rs10419557   rs34256230   rs58542926  rs117222437   \n","99    rs1548546   rs61593631   rs12983010  rs148152819     rs346051   \n","\n","            245          246          247          248          249  \n","0    rs17699238   rs12982294  rs148152819    rs1047333  rs115379880  \n","1     rs8110485   rs60545781   rs10411967  rs142585929   rs73048053  \n","2   rs115379880  rs111435864   rs10415949   rs61593631   rs11670136  \n","3     rs4803791   rs74827760    rs2965169  rs139140823   rs10401338  \n","4    rs58782346   rs74359223   rs73015030   rs45446093   rs11667752  \n","..          ...          ...          ...          ...          ...  \n","95    rs7249083   rs72992965  rs114370247   rs34099325  rs149513825  \n","96   rs34099325   rs62109991  rs113986398   rs12461923   rs10417167  \n","97   rs35850102  rs140181075    rs1640266    rs4803840    rs4802506  \n","98  rs139140823   rs35500721    rs8109764   rs72658856   rs11671341  \n","99    rs2967465   rs76069409   rs75983562    rs2335534  rs370539355  \n","\n","[100 rows x 250 columns]"],"text/html":["\n","  <div id=\"df-272874c2-0eb9-46c3-936f-ad48fc8d75bc\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>0</th>\n","      <th>1</th>\n","      <th>2</th>\n","      <th>3</th>\n","      <th>4</th>\n","      <th>5</th>\n","      <th>6</th>\n","      <th>7</th>\n","      <th>8</th>\n","      <th>9</th>\n","      <th>...</th>\n","      <th>240</th>\n","      <th>241</th>\n","      <th>242</th>\n","      <th>243</th>\n","      <th>244</th>\n","      <th>245</th>\n","      <th>246</th>\n","      <th>247</th>\n","      <th>248</th>\n","      <th>249</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>rs28399607</td>\n","      <td>rs112501445</td>\n","      <td>rs76936640</td>\n","      <td>rs145639166</td>\n","      <td>rs644699</td>\n","      <td>rs10403723</td>\n","      <td>rs10129</td>\n","      <td>rs2972564</td>\n","      <td>rs73591055</td>\n","      <td>rs1003393</td>\n","      <td>...</td>\n","      <td>rs78961098</td>\n","      <td>rs429358</td>\n","      <td>rs149183396</td>\n","      <td>rs11670136</td>\n","      <td>rs4804149</td>\n","      <td>rs17699238</td>\n","      <td>rs12982294</td>\n","      <td>rs148152819</td>\n","      <td>rs1047333</td>\n","      <td>rs115379880</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>rs185574863</td>\n","      <td>rs34256230</td>\n","      <td>rs36005514</td>\n","      <td>rs55734122</td>\n","      <td>rs3848609</td>\n","      <td>rs10401338</td>\n","      <td>rs62131325</td>\n","      <td>rs10408750</td>\n","      <td>rs1441617</td>\n","      <td>rs11879991</td>\n","      <td>...</td>\n","      <td>rs17714761</td>\n","      <td>rs141366665</td>\n","      <td>rs10409774</td>\n","      <td>rs10417167</td>\n","      <td>rs59848316</td>\n","      <td>rs8110485</td>\n","      <td>rs60545781</td>\n","      <td>rs10411967</td>\n","      <td>rs142585929</td>\n","      <td>rs73048053</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>rs238410</td>\n","      <td>rs141441332</td>\n","      <td>rs116283165</td>\n","      <td>rs62118464</td>\n","      <td>rs73051677</td>\n","      <td>rs2074940</td>\n","      <td>rs142585929</td>\n","      <td>rs35763270</td>\n","      <td>rs34561079</td>\n","      <td>rs35678764</td>\n","      <td>...</td>\n","      <td>rs8107753</td>\n","      <td>rs11667610</td>\n","      <td>rs147871592</td>\n","      <td>rs10417167</td>\n","      <td>rs4804510</td>\n","      <td>rs115379880</td>\n","      <td>rs111435864</td>\n","      <td>rs10415949</td>\n","      <td>rs61593631</td>\n","      <td>rs11670136</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>rs56315738</td>\n","      <td>rs2288450</td>\n","      <td>rs62109991</td>\n","      <td>rs4564699</td>\n","      <td>rs74931926</td>\n","      <td>rs8100933</td>\n","      <td>rs10415949</td>\n","      <td>rs113357289</td>\n","      <td>rs117777502</td>\n","      <td>rs34964177</td>\n","      <td>...</td>\n","      <td>rs72654437</td>\n","      <td>rs11673516</td>\n","      <td>rs142385484</td>\n","      <td>rs55766194</td>\n","      <td>rs281417</td>\n","      <td>rs4803791</td>\n","      <td>rs74827760</td>\n","      <td>rs2965169</td>\n","      <td>rs139140823</td>\n","      <td>rs10401338</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>rs148613044</td>\n","      <td>rs553008653</td>\n","      <td>rs11879991</td>\n","      <td>rs2927486</td>\n","      <td>rs11665702</td>\n","      <td>rs7246718</td>\n","      <td>rs35334454</td>\n","      <td>rs144159978</td>\n","      <td>rs58542926</td>\n","      <td>rs8110775</td>\n","      <td>...</td>\n","      <td>rs2335534</td>\n","      <td>rs34964177</td>\n","      <td>rs75920121</td>\n","      <td>rs10221516</td>\n","      <td>rs17262997</td>\n","      <td>rs58782346</td>\n","      <td>rs74359223</td>\n","      <td>rs73015030</td>\n","      <td>rs45446093</td>\n","      <td>rs11667752</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>95</th>\n","      <td>rs75161053</td>\n","      <td>rs4803825</td>\n","      <td>rs8109764</td>\n","      <td>rs73563690</td>\n","      <td>rs11666216</td>\n","      <td>rs7254990</td>\n","      <td>rs113357289</td>\n","      <td>rs4808595</td>\n","      <td>rs58513885</td>\n","      <td>rs143043127</td>\n","      <td>...</td>\n","      <td>rs10420562</td>\n","      <td>rs916694</td>\n","      <td>rs8107329</td>\n","      <td>rs11085327</td>\n","      <td>rs62132801</td>\n","      <td>rs7249083</td>\n","      <td>rs72992965</td>\n","      <td>rs114370247</td>\n","      <td>rs34099325</td>\n","      <td>rs149513825</td>\n","    </tr>\n","    <tr>\n","      <th>96</th>\n","      <td>rs11671372</td>\n","      <td>rs10451476</td>\n","      <td>rs140915448</td>\n","      <td>rs41301949</td>\n","      <td>rs551557788</td>\n","      <td>rs112056827</td>\n","      <td>rs140842514</td>\n","      <td>rs117970794</td>\n","      <td>rs141964576</td>\n","      <td>rs2007774</td>\n","      <td>...</td>\n","      <td>rs146901672</td>\n","      <td>rs62111695</td>\n","      <td>rs62119326</td>\n","      <td>rs1441617</td>\n","      <td>rs72992965</td>\n","      <td>rs34099325</td>\n","      <td>rs62109991</td>\n","      <td>rs113986398</td>\n","      <td>rs12461923</td>\n","      <td>rs10417167</td>\n","    </tr>\n","    <tr>\n","      <th>97</th>\n","      <td>rs139140823</td>\n","      <td>rs34768259</td>\n","      <td>rs238410</td>\n","      <td>rs2238551</td>\n","      <td>rs204480</td>\n","      <td>rs143059942</td>\n","      <td>rs435380</td>\n","      <td>rs8109764</td>\n","      <td>rs56315738</td>\n","      <td>rs10420349</td>\n","      <td>...</td>\n","      <td>rs117607802</td>\n","      <td>rs76649675</td>\n","      <td>rs421812</td>\n","      <td>rs11668344</td>\n","      <td>rs72997195</td>\n","      <td>rs35850102</td>\n","      <td>rs140181075</td>\n","      <td>rs1640266</td>\n","      <td>rs4803840</td>\n","      <td>rs4802506</td>\n","    </tr>\n","    <tr>\n","      <th>98</th>\n","      <td>rs112283362</td>\n","      <td>rs138218778</td>\n","      <td>rs79322367</td>\n","      <td>rs11673631</td>\n","      <td>rs143043127</td>\n","      <td>rs3810333</td>\n","      <td>rs59715573</td>\n","      <td>rs577349548</td>\n","      <td>rs11665702</td>\n","      <td>rs58513885</td>\n","      <td>...</td>\n","      <td>rs141651018</td>\n","      <td>rs10419557</td>\n","      <td>rs34256230</td>\n","      <td>rs58542926</td>\n","      <td>rs117222437</td>\n","      <td>rs139140823</td>\n","      <td>rs35500721</td>\n","      <td>rs8109764</td>\n","      <td>rs72658856</td>\n","      <td>rs11671341</td>\n","    </tr>\n","    <tr>\n","      <th>99</th>\n","      <td>rs79860339</td>\n","      <td>rs117970794</td>\n","      <td>rs17262997</td>\n","      <td>rs2620840</td>\n","      <td>rs10420562</td>\n","      <td>rs58496711</td>\n","      <td>rs11669345</td>\n","      <td>rs3093032</td>\n","      <td>rs73048053</td>\n","      <td>rs11880312</td>\n","      <td>...</td>\n","      <td>rs1548546</td>\n","      <td>rs61593631</td>\n","      <td>rs12983010</td>\n","      <td>rs148152819</td>\n","      <td>rs346051</td>\n","      <td>rs2967465</td>\n","      <td>rs76069409</td>\n","      <td>rs75983562</td>\n","      <td>rs2335534</td>\n","      <td>rs370539355</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>100 rows × 250 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-272874c2-0eb9-46c3-936f-ad48fc8d75bc')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-272874c2-0eb9-46c3-936f-ad48fc8d75bc button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-272874c2-0eb9-46c3-936f-ad48fc8d75bc');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-1dad7f4f-6ce0-47c2-b4b3-b8d2f6b92582\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-1dad7f4f-6ce0-47c2-b4b3-b8d2f6b92582')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-1dad7f4f-6ce0-47c2-b4b3-b8d2f6b92582 button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_9cdae674-1176-4dfb-a19e-8343750ebc6f\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('sample_causals_df')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_9cdae674-1176-4dfb-a19e-8343750ebc6f button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('sample_causals_df');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"sample_causals_df"}},"metadata":{},"execution_count":84}]},{"cell_type":"code","source":["sh = \"\"\"\n","\n","BFILE=chr19\n","NUM_PHENO=100\n","PARENT_DIR=\"/content/chr19_ldl_pheno\"\n","\n","for NUM_CAUSL in {1,10,50,100,250}\n","\n","do\n","\n","    CAUSALFILE=causals_effects/causals_num-causals=${NUM_CAUSL}.txt\n","    EFFECTSFILE=causals_effects/effects_num-causals=${NUM_CAUSL}.txt\n","\n","    for POWER in {-1,-0.25}\n","\n","    do\n","        for HER in {0.1,0.3,0.5,0.7,0.9}\n","\n","        do\n","\n","        OUT=${PARENT_DIR}/power=${POWER}_her=${HER}_num-causals=${NUM_CAUSL}\n","\n","            ./ldak5.2.linux \\\n","                --make-phenos $OUT \\\n","                --bfile $BFILE \\\n","                --num-phenos $NUM_PHENO \\\n","                --causals $CAUSALFILE \\\n","                --effects $EFFECTSFILE \\\n","                --power $POWER \\\n","                --her $HER \\\n","                --num-causals $NUM_CAUSL \\\n","                --ignore-weights YES\n","\n","        done\n","\n","    done\n","\n","done\n","\n","\"\"\"\n","with open('script.sh', 'w') as file:\n","  file.write(sh)\n","\n","! bash script.sh"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"8QMus0BcqSoB","executionInfo":{"status":"ok","timestamp":1710101491952,"user_tz":420,"elapsed":105057,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"e9c2616a-326d-412c-8a2a-fa1aae64bd98"},"execution_count":91,"outputs":[{"output_type":"stream","name":"stdout","text":["-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -1\n","--her 0.1\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -1\n","--her 0.3\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -1\n","--her 0.5\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -1\n","--her 0.7\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -1\n","--her 0.9\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--effects causals_effects/effects_num-causals=1.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=1.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -1\n","--her 0.1\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -1\n","--her 0.3\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -1\n","--her 0.5\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -1\n","--her 0.7\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -1\n","--her 0.9\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--effects causals_effects/effects_num-causals=10.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=10.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -1\n","--her 0.1\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -1\n","--her 0.3\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -1\n","--her 0.5\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -1\n","--her 0.7\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -1\n","--her 0.9\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--effects causals_effects/effects_num-causals=50.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=50.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -1\n","--her 0.1\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -1\n","--her 0.3\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -1\n","--her 0.5\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -1\n","--her 0.7\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -1\n","--her 0.9\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--effects causals_effects/effects_num-causals=100.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=100.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -1\n","--her 0.1\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.1_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -1\n","--her 0.3\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.3_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -1\n","--her 0.5\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.5_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -1\n","--her 0.7\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.7_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -1\n","--her 0.9\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-1_her=0.9_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.1_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.3_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.5_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.7_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 9 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--effects causals_effects/effects_num-causals=250.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) are specified in causals_effects/effects_num-causals=250.txt\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno/power=-0.25_her=0.9_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n"]}]},{"cell_type":"code","source":["! cp -r chr19_ldl_pheno/* /content/drive/MyDrive/CSE-284-Final-Project/data/chr19_ldl_pheno/"],"metadata":{"id":"Tf1mDtQe_H8I","executionInfo":{"status":"ok","timestamp":1710101504183,"user_tz":420,"elapsed":5330,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":92,"outputs":[]},{"cell_type":"code","source":["! cp -r causals_effects/* /content/drive/MyDrive/CSE-284-Final-Project/data/chr19_ldl_causals_effects/"],"metadata":{"id":"PJP7BgkV_L58","executionInfo":{"status":"ok","timestamp":1710101506053,"user_tz":420,"elapsed":637,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":93,"outputs":[]},{"cell_type":"code","source":["! mkdir chr19_ldl_pheno_random_effects"],"metadata":{"id":"1qammoK7lJ1L","executionInfo":{"status":"ok","timestamp":1710111383729,"user_tz":420,"elapsed":296,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":94,"outputs":[]},{"cell_type":"code","source":["sh = \"\"\"\n","\n","BFILE=chr19\n","CAUSALFILE=causals_effects/causals_all.txt\n","NUM_PHENO=100\n","NUM_CAUSL=512\n","PARENT_DIR=\"/content/chr19_ldl_pheno_random_effects\"\n","\n","\n","for POWER in {-1,-0.25}\n","\n","do\n","    for HER in {0.1,0.3,0.5,0.7,0.9}\n","\n","    do\n","\n","    OUT=${PARENT_DIR}/power=${POWER}_her=${HER}_num-causals=${NUM_CAUSL}\n","\n","        ./ldak5.2.linux \\\n","            --make-phenos $OUT \\\n","            --bfile $BFILE \\\n","            --num-phenos $NUM_PHENO \\\n","            --causals $CAUSALFILE \\\n","            --power $POWER \\\n","            --her $HER \\\n","            --num-causals $NUM_CAUSL \\\n","            --ignore-weights YES\n","\n","    done\n","\n","done\n","\n","\"\"\"\n","with open('script.sh', 'w') as file:\n","  file.write(sh)\n","\n","! bash script.sh"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"eFwKPrfNlOxE","executionInfo":{"status":"ok","timestamp":1710111480207,"user_tz":420,"elapsed":32343,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"0fc1b672-74d8-4111-d87a-64ad16e575a5"},"execution_count":95,"outputs":[{"output_type":"stream","name":"stdout","text":["-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -1\n","--her 0.1\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -1\n","--her 0.3\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -1\n","--her 0.5\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -1\n","--her 0.7\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -1\n","--her 0.9\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=512\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_all.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 512\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 512 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_all.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_all.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=512.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=512.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=512.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n"]}]},{"cell_type":"code","source":["sh = \"\"\"\n","\n","BFILE=chr19\n","NUM_PHENO=100\n","PARENT_DIR=\"/content/chr19_ldl_pheno_random_effects\"\n","\n","for NUM_CAUSL in {1,10,50,100,250}\n","\n","do\n","\n","    CAUSALFILE=causals_effects/causals_num-causals=${NUM_CAUSL}.txt\n","\n","    for POWER in {-1,-0.25}\n","\n","    do\n","        for HER in {0.1,0.3,0.5,0.7,0.9}\n","\n","        do\n","\n","        OUT=${PARENT_DIR}/power=${POWER}_her=${HER}_num-causals=${NUM_CAUSL}\n","\n","            ./ldak5.2.linux \\\n","                --make-phenos $OUT \\\n","                --bfile $BFILE \\\n","                --num-phenos $NUM_PHENO \\\n","                --causals $CAUSALFILE \\\n","                --power $POWER \\\n","                --her $HER \\\n","                --num-causals $NUM_CAUSL \\\n","                --ignore-weights YES\n","\n","        done\n","\n","    done\n","\n","done\n","\n","\"\"\"\n","with open('script.sh', 'w') as file:\n","  file.write(sh)\n","\n","! bash script.sh"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"SgCaqRq0lPLH","executionInfo":{"status":"ok","timestamp":1710111652434,"user_tz":420,"elapsed":125555,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"91c5c2f3-261d-4ae1-8ce6-0792b4d4459c"},"execution_count":96,"outputs":[{"output_type":"stream","name":"stdout","text":["-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -1\n","--her 0.1\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -1\n","--her 0.3\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -1\n","--her 0.5\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -1\n","--her 0.7\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -1\n","--her 0.9\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=1\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=1.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 1\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 1 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=1.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=1.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=1.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=1.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=1.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -1\n","--her 0.1\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -1\n","--her 0.3\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -1\n","--her 0.5\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -1\n","--her 0.7\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -1\n","--her 0.9\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=10\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=10.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 10\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 10 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=10.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=10.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=10.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=10.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=10.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -1\n","--her 0.1\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -1\n","--her 0.3\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -1\n","--her 0.5\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -1\n","--her 0.7\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -1\n","--her 0.9\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=50\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=50.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 50\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 50 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=50.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=50.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=50.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=50.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=50.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -1\n","--her 0.1\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -1\n","--her 0.3\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -1\n","--her 0.5\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -1\n","--her 0.7\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -1\n","--her 0.9\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=100\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=100.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 100\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 100 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=100.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=100.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=100.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=100.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=100.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -1\n","--her 0.1\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.1_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -1\n","--her 0.3\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.3_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -1\n","--her 0.5\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.5_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -1\n","--her 0.7\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.7_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -1\n","--her 0.9\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-1.0000/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -0.25\n","--her 0.1\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.1000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.1_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -0.25\n","--her 0.3\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.3000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.3_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -0.25\n","--her 0.5\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.5000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.5_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -0.25\n","--her 0.7\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.7000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.7_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","LDAK - Software for obtaining Linkage Disequilibrium Adjusted Kinships and Loads More\n","Version 5.2 - Help pages at http://www.ldak.org\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","There are 8 pairs of arguments:\n","--make-phenos /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=250\n","--bfile chr19\n","--num-phenos 100\n","--causals causals_effects/causals_num-causals=250.txt\n","--power -0.25\n","--her 0.9\n","--num-causals 250\n","--ignore-weights YES\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Making 100 phenotypes, each with heritability 0.9000 and 250 causal predictors\n","\n","Predictors will be centred then scaled by V^(-0.2500/2) (option \"--power\"), where V=2*MAF*(1-MAF), the expected variance assuming Hardy-Weinberg Equilibrium; to instead scale based on observed variance use \"--hwe-stand NO\"\n","\n","Causal predictors are specified in causals_effects/causals_num-causals=250.txt\n","\n","Effect sizes (for scaled predictors) will be drawn from a Gaussian distribution; if you would prefer to specify them, use \"--effects\"\n","\n","To generate a binary phenotype, use \"--prev\" to provide the prevalence of cases\n","\n","To generate correlated pairs of phenotypes, use \"--bivar\"\n","\n","It appears this system has multiple processors available; to run the parallel version of LDAK, use \"--max-threads\" (this will only reduce runtime for some commands)\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","\n","Reading IDs for 2504 samples from chr19.fam\n","\n","Reading details for 283173 predictors from chr19.bim\n","\n","Data contain 2504 samples and 283173 predictors\n","\n","Reading causal predictors from causals_effects/causals_num-causals=250.txt\n","Making phenotypes for Chunk 1 of 1\n","\n","Phenotypes saved in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=250.pheno, with breeding values in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=250.breed and effect sizes in /content/chr19_ldl_pheno_random_effects/power=-0.25_her=0.9_num-causals=250.effects\n","\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n","Mission completed. All your basepair are belong to us :)\n","-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --\n"]}]},{"cell_type":"code","source":["! head chr19_ldl_pheno/power=-1_her=0.9_num-causals=512.pheno"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"SBmSMyH4mIU2","executionInfo":{"status":"ok","timestamp":1710111687676,"user_tz":420,"elapsed":150,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"731a421d-0381-46fc-b96d-100c8a1494c7"},"execution_count":100,"outputs":[{"output_type":"stream","name":"stdout","text":["0 HG00096 -1.899202 -1.041246 -1.443321 -1.732044 -1.200891 -1.060183 -1.020299 -0.547387 -1.842862 -0.907769 -1.825056 -1.391698 -1.608965 -1.555709 -1.502583 -1.149981 -1.375420 -1.379174 -1.135772 -0.992182 -1.478227 -1.422246 -1.279287 -1.430140 -1.483291 -1.915762 -1.008216 -1.610993 -1.404742 -1.981382 -1.373465 -1.076122 -1.021652 -0.956304 -1.295711 -0.772557 -1.270001 -0.836604 -1.535703 -0.810656 -0.785056 -1.205638 -0.836946 -1.278888 -1.009078 -1.160094 -1.580056 -1.533457 -1.299815 -1.701400 -1.429931 -0.592049 -1.271766 -1.323405 -2.117853 -1.378791 -1.424904 -1.143046 -1.171225 -1.597679 -1.293848 -1.119953 -0.920118 -0.912147 -1.166964 -1.058441 -1.310111 -1.642080 -1.450256 -1.528894 -1.204778 -1.429540 -1.051580 -0.988689 -1.474904 -1.556249 -1.415465 -1.356226 -2.103688 -1.315107 -1.237281 -1.219446 -1.355105 -1.611589 -1.263697 -1.352068 -1.201778 -1.278334 -1.393961 -1.301894 -1.605615 -1.015500 -2.351497 -1.232124 -1.630880 -1.163642 -1.506007 -1.393312 -1.246500 -0.809743 \n","0 HG00097 -0.379556 -0.345065 -0.112039 0.064766 0.172122 0.228262 -0.214366 -0.421242 -0.217297 -0.032437 0.066399 0.132044 -0.051028 -0.271505 -0.374500 -0.819684 -0.034834 -0.311421 -0.175942 -0.238333 -0.348811 0.217063 0.341153 -0.312744 -0.133984 -0.182666 0.236934 -0.142990 0.219950 0.116424 -0.008091 -0.214255 0.019343 -0.288326 -0.708285 -0.212347 -0.557774 0.500092 -0.337541 -0.420029 -0.209745 -0.355463 -0.337039 0.434334 0.241679 0.214913 -0.110940 0.046721 0.217309 0.614983 -0.045451 -0.103354 -0.367499 -0.297790 0.036514 -0.349255 0.010886 -0.637110 -0.947358 -0.285678 0.398866 -0.328755 0.687280 0.334291 -0.223081 -0.421902 0.052165 -0.180721 0.169125 -0.024687 0.094568 -0.495994 -0.301991 -0.245867 0.605259 -0.011702 -0.456515 0.016003 -0.643817 -0.043524 -0.704032 -0.109641 -0.101298 -0.113224 -0.169461 -0.164022 0.346957 -0.181939 -0.521000 -0.080545 -0.377359 -0.286961 -0.111480 0.111415 -0.146678 -0.788718 -0.091183 -0.583044 -0.533360 0.024159 \n","0 HG00099 0.141943 0.473072 0.158021 0.622426 -0.030839 0.313654 0.035323 0.470899 0.333639 0.302082 0.086713 0.146339 0.566243 -0.231363 -0.000065 0.948400 0.200167 0.378902 -0.241826 0.453345 0.130160 0.060377 0.404676 0.468065 0.046136 -0.022175 -0.591308 0.410334 0.183112 0.624923 -0.464972 0.202811 0.258077 0.485556 0.024920 0.486689 0.605263 0.047644 0.189086 0.511978 0.218407 -0.283105 0.030729 -0.280997 -0.045374 -0.101269 -0.249181 0.426425 0.121698 -0.190742 0.299221 0.034804 -0.470770 0.173446 0.292909 0.093574 0.222295 0.377528 -0.069246 -0.511221 0.440147 0.018533 0.038654 -0.087247 0.104222 0.124023 -0.186593 0.061808 0.055228 0.274041 0.424919 -0.761492 0.488651 0.492534 -0.030610 -0.237972 0.136881 0.286121 0.241906 -0.545988 0.134892 0.153109 -0.171401 0.384892 0.160824 -0.025808 -0.197663 -0.428543 -0.167558 0.465923 0.130143 0.246554 -0.236017 0.194480 0.199885 0.219790 -0.029022 0.305666 -0.306366 0.330933 \n","0 HG00100 -0.622421 -0.407337 -1.382916 -0.104484 -0.835572 -0.975154 -0.652388 -0.124815 -0.411298 -0.517011 -0.791366 -1.083932 -0.953880 -0.435262 -1.044622 -0.658126 -0.792755 -0.627887 -0.910846 -0.584924 -0.873623 -0.922172 -0.435548 -0.360697 -0.416605 -0.707356 -0.706950 -0.399472 -0.995818 -1.014063 -0.293622 -0.380997 -0.596370 -0.789544 -0.678562 -0.498696 -0.677925 -0.677108 -1.099567 -0.915196 -0.770809 -1.160635 -0.987924 -0.588342 -0.921882 -0.753634 -1.018873 -0.286409 -0.773637 -0.592055 -1.083637 -0.845526 -1.397421 -0.741215 -0.665897 -0.762864 -0.974068 -0.686983 -0.578788 -0.726483 -1.138721 -1.042104 -0.595280 -1.120207 -1.050620 -0.822573 -0.666290 -1.007406 0.329753 -0.667375 -0.902356 -0.764043 0.005882 -0.583696 -1.285589 -0.900854 -0.292722 -1.097654 -0.744307 -0.444253 -0.744624 -1.084951 -1.263451 -0.392221 -0.352432 -0.280332 -0.249604 -0.708402 -1.090059 -0.737327 -0.837749 -0.769424 -0.909622 -0.652720 -0.818819 -0.121479 -0.905485 -0.688437 -0.571404 -0.504763 \n","0 HG00101 -0.921933 -1.535862 -1.346244 -0.701908 -1.205896 -1.486772 -0.931368 -1.320459 -1.125926 -1.737015 -1.754664 -1.199141 -1.192042 -0.974798 -1.496633 -1.174389 -1.282848 -1.364816 -1.356320 -1.752778 -1.494813 -1.653416 -1.301052 -1.230437 -1.166391 -1.184611 -0.796465 -1.441433 -1.154737 -0.794525 -0.855750 -1.151155 -1.568101 -0.939415 -1.419636 -1.179443 -1.127263 -1.088456 -1.549530 -1.085272 -1.484902 -1.962145 -0.080234 -1.058023 -1.505291 -1.243082 -1.374256 -0.807495 -0.788739 -1.443870 -1.313722 -1.207659 -1.007884 -1.051370 -1.326709 -1.059227 -1.275982 -0.992755 -1.278022 -0.970830 -1.638637 -0.752117 -1.421717 -1.162939 -1.382306 -1.797691 -1.663943 -0.711245 -1.512932 -1.368954 -0.666436 -1.579619 -0.864894 -1.405869 -1.769277 -1.144801 -1.220010 -1.892009 -1.430801 -0.926973 -1.492251 -1.370287 -1.061602 -0.682328 -0.948765 -0.926798 -1.349958 -1.299436 -1.807964 -0.973732 -1.115329 -1.390435 -1.781292 -0.872598 -1.362881 -1.222850 -1.120062 -1.404089 -1.279816 -1.548861 \n","0 HG00102 0.792442 0.974144 1.341908 0.436392 0.997738 1.046680 1.009923 0.821979 0.452101 0.807098 1.118030 0.854383 0.753072 0.850003 0.962298 0.256192 0.542264 0.120717 0.182671 0.816292 0.948767 0.463308 -0.002332 0.752982 0.557841 0.897032 0.434847 0.928279 0.878774 0.573134 0.745855 1.122195 0.215392 0.475696 0.598586 0.933273 0.378177 0.855231 0.690539 0.644663 0.356112 1.218801 0.500574 0.765463 0.630569 0.463211 0.963380 1.023633 0.869763 0.679137 0.289767 1.049527 0.708147 1.256470 0.323217 0.768757 0.723831 1.011402 1.015434 1.193155 0.173807 0.587741 0.841276 1.533418 0.705914 1.074250 0.826465 0.798892 1.202521 0.573857 0.776669 0.619423 0.658337 0.508835 1.055186 0.738848 1.020974 0.966951 1.011923 0.655547 1.255357 0.717926 0.677095 0.350849 0.761731 0.824532 0.412257 1.029882 0.658640 0.869678 0.577573 0.217458 0.857413 0.466964 0.396239 1.234487 0.508024 1.128527 0.235294 0.546724 \n","0 HG00103 0.585911 -0.687698 0.230281 0.215792 0.039610 -0.278614 -0.442470 -0.879235 0.326995 0.000734 0.180033 0.315584 -0.071767 0.033146 0.043861 -0.247837 0.097926 -0.068779 0.482161 -0.068269 -0.347628 0.294619 -0.211238 0.271093 -0.152838 0.170708 -0.309963 0.160557 0.251566 -0.072066 -0.243770 0.191946 0.252835 -0.066411 0.486746 -0.096011 0.212012 0.071994 0.244731 0.071664 0.272329 0.161562 -0.520681 0.088159 -0.004616 0.310724 -0.049781 -0.013642 -0.259508 -0.298360 0.389049 -0.165624 0.192547 -0.108549 -0.081955 -0.166085 -0.036094 -0.120478 0.124376 -0.253690 -0.326758 0.318569 0.087681 0.008434 0.304995 0.053116 -0.056945 0.199678 -0.287813 0.168261 -0.111610 0.435714 0.268544 -0.369234 0.607206 0.229924 -0.181993 -0.138187 -0.562940 -0.099056 0.350211 0.180768 -0.537866 0.212028 -0.326686 -0.614803 0.484411 -0.069382 -0.148766 -0.053647 0.742548 -0.273228 0.093886 0.303357 0.609236 -0.010358 -0.223223 -0.349168 0.277820 -0.188250 \n","0 HG00105 -0.301709 -0.515345 -0.253119 -0.263568 -0.286359 0.153898 -0.566097 0.064934 -0.499219 -0.020925 0.228973 -0.661856 -0.435899 -0.001565 -0.243456 0.602707 0.033214 0.217800 -0.361763 -0.505390 -0.311324 -0.002468 0.254876 0.209154 -0.436321 0.036865 0.163707 -0.241951 0.167152 -0.370867 0.106399 -0.453578 0.077547 -0.407949 0.305064 -0.479526 0.233607 0.333160 0.120168 -0.008863 -0.451891 0.011611 0.488725 -0.245264 -0.675794 0.412304 0.085059 0.145982 -0.360855 -0.208730 -0.167083 -0.768591 0.381064 -0.288028 -0.433601 0.139635 0.295994 -0.148964 -0.600851 -0.173333 -0.597797 0.234829 -0.067085 0.238363 0.255021 -0.268157 -0.338237 -0.063064 -0.449667 -0.205391 0.319704 -0.076995 -0.356677 0.228624 -0.249381 -0.084862 0.115590 -0.738857 0.370792 -0.531776 -0.123655 -0.070705 0.223092 0.122151 -0.045879 0.512255 -0.295067 0.213302 0.157512 0.510952 0.243616 -0.029752 -0.724669 -0.450347 -0.194493 -0.067607 0.080973 0.423516 0.022015 0.927382 \n","0 HG00106 0.462228 -0.560828 -0.161148 -0.858381 -0.386992 -0.071992 -0.541533 -0.160354 -0.274098 0.048308 0.033717 -0.383529 0.361067 0.420466 -0.372204 -0.354835 -0.306093 0.473387 -0.333836 0.206705 -0.084203 0.007877 -0.021319 0.134263 -0.409783 0.348610 -0.150295 -0.217102 0.401627 0.137113 -0.050728 -0.081603 0.438995 -0.191827 0.185975 0.396098 -0.321910 -0.465039 -0.533360 0.406861 -0.259819 -0.178021 0.168104 -0.725231 0.120133 -0.087141 -0.115148 -0.059438 -0.051112 -0.185610 0.135427 -0.123199 -0.355560 -0.568491 -0.139714 -0.672166 -0.545693 0.151055 -0.289931 -0.263013 0.509531 -0.193248 -0.269447 -0.392788 0.335683 -0.619362 -0.436501 -0.090378 0.335339 -0.254531 -0.522956 -0.105983 -0.057856 -0.343317 0.105083 0.194887 -0.262628 -0.315608 0.264696 -0.128112 -0.310866 -0.412751 0.108636 -0.757544 0.286813 0.310114 0.031675 -0.217064 -0.075125 -0.026039 -0.059464 -0.148017 -0.443587 0.002108 -0.166718 0.105707 -0.045940 -0.011623 -0.044919 0.302027 \n","0 HG00107 -1.114487 -0.710916 -0.748805 -0.882620 -0.987595 -0.566089 -1.147765 -0.701064 -1.324007 -0.940942 -0.571452 -0.955506 -0.478425 -1.494316 -0.835007 -1.273517 -1.072123 -1.491629 -0.779884 -0.647087 -0.865479 -0.667198 -0.533423 -0.834980 -0.912430 -1.034307 -0.566126 -1.511626 -0.791156 -1.168434 -1.351734 -0.565563 -1.028257 -0.256375 -0.853611 -0.643211 -0.689859 0.273470 -0.853128 -0.315503 -0.330841 -0.686689 -0.979301 -1.424147 -0.657692 -0.804034 -1.379338 -0.721725 -0.647631 -1.224657 -0.598626 -0.902786 -0.969175 -1.052812 -0.907873 -1.146742 -0.621936 -0.991787 -0.714765 -1.264735 -0.574705 -1.089142 -1.223832 -0.627859 -0.953061 -1.009540 -0.724896 -1.133366 -0.559882 -1.306703 -0.888944 -1.557002 -1.363197 -1.338129 -1.058137 -1.361597 -0.826277 -0.808874 -0.335992 -0.908656 -0.721804 -1.118103 -0.687731 -0.839251 -0.930435 -0.723776 -0.587948 -1.470963 -0.823451 -0.795167 -1.137372 -0.370392 -0.604820 -0.800780 -0.422709 -0.917277 -1.022735 -1.304627 -1.249471 -0.447894 \n"]}]},{"cell_type":"code","source":["! head chr19_ldl_pheno_random_effects/power=-1_her=0.9_num-causals=512.pheno"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"SVPs6rRMmCBJ","executionInfo":{"status":"ok","timestamp":1710111691467,"user_tz":420,"elapsed":330,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}},"outputId":"156c8b00-42fe-4f95-dded-28b3a9a4cd7b"},"execution_count":101,"outputs":[{"output_type":"stream","name":"stdout","text":["0 HG00096 0.728274 -0.286426 0.532203 0.286817 0.012243 -0.079409 0.747995 -0.706898 1.244686 1.499962 0.220641 -0.470038 0.730225 0.125194 0.913198 1.402940 0.525353 1.138104 -0.268501 -0.781316 -0.466639 -0.181354 1.317090 -1.773108 0.856765 0.180438 -0.264528 -0.333990 -0.486910 -0.238062 -0.979106 -0.747700 -1.138810 -0.372377 -0.952487 1.707171 -0.125791 -0.377210 -0.450395 0.137840 -0.900669 1.649553 -0.715846 0.921151 1.727334 1.440694 0.299657 0.258258 -1.485421 -0.263642 -0.340887 0.426608 0.538906 0.120987 0.023283 -0.393799 -0.468902 1.178976 -1.251889 -1.260099 0.825476 -0.753166 -1.139262 -0.020048 1.774720 -0.512823 -0.746409 -0.924417 -0.756409 -1.890953 0.873399 -1.029151 -1.280232 0.827302 -0.957846 -0.632498 -0.428972 0.657202 0.419497 -1.196183 -1.825019 0.486940 -0.879750 -0.620582 0.507303 0.905536 0.794128 -0.826317 -0.002879 0.421013 -1.046894 -0.498125 -0.206345 -0.527610 -0.381658 -0.272473 0.129269 -0.508985 -1.334881 0.917522 \n","0 HG00097 0.773943 0.021560 0.590923 0.001904 -0.679421 1.148073 -0.939429 -0.562241 -0.069633 1.390658 0.554102 0.679277 -0.016296 -0.609638 -0.930162 1.481271 -0.006797 1.784849 0.318916 0.462004 -1.347352 0.477881 -0.065829 2.498596 1.494483 0.696617 -0.129211 -0.326781 0.518657 -0.330699 0.425595 -1.019618 -0.638263 -1.199118 0.537341 0.176222 -1.213769 -0.017108 1.984175 -0.251493 2.221398 -0.174517 0.802028 0.009397 0.520463 0.456120 0.445405 1.446013 -1.021416 1.037540 -1.251340 0.694057 0.261494 0.454521 -2.575559 -0.540687 1.522359 1.662510 -0.372316 0.312606 1.606119 -1.307642 0.320228 -0.348220 -1.708647 -2.237612 -0.066423 0.922176 -1.063812 0.026704 0.434052 2.432018 -1.205133 -2.573751 0.635838 0.379010 -0.830773 1.359527 -1.055770 1.322608 2.758155 -0.626160 0.737140 0.608970 -1.123905 -0.154138 -0.838192 -0.467102 -1.147147 -0.675785 -0.295191 -0.505890 -0.918925 0.497117 -0.739104 0.759259 -0.235982 -1.120795 1.556877 0.760440 \n","0 HG00099 -1.836380 1.333452 0.464629 -0.428367 0.088747 2.197910 -0.225901 -0.234321 0.762251 0.445940 -1.000573 1.093876 0.684903 0.525945 0.133885 -1.318998 -1.232561 0.513245 1.245458 -0.589064 -0.561049 -1.873333 -1.505838 -0.762307 0.217452 0.401399 0.351647 2.243966 -0.018719 0.103545 -0.207917 -0.811819 -0.606582 -0.298976 -0.712827 -0.941303 -0.495388 -0.893715 0.244653 1.697292 -0.399739 0.123847 1.050787 0.042951 1.974027 0.007695 1.251737 -1.026533 0.034764 -0.229774 -1.675184 0.277061 0.616932 -0.399045 -0.768745 -0.162725 0.093124 0.684676 1.943664 -0.145283 -0.211766 -0.079473 -0.714086 -0.764369 -1.126008 -1.248871 0.944157 0.047489 -0.817845 -0.956721 0.230220 -1.336210 -1.603551 -0.989176 0.235825 0.002666 -0.270977 -0.083170 2.599908 -0.186056 -0.070958 -0.360967 0.612671 -0.279208 0.531438 1.748674 2.019783 1.191499 1.420323 1.112337 0.217332 -1.293896 0.549096 1.294593 -0.865404 -0.896304 1.481186 0.580952 1.798204 0.796308 \n","0 HG00100 -0.226601 1.218781 -0.642803 0.567437 0.108107 -1.194151 -1.443352 -0.502415 0.000651 2.167453 1.126100 -0.849757 1.912663 1.116674 -0.355592 -0.458807 0.124534 -0.086439 1.616760 1.424207 -1.800924 -0.749266 -0.303012 -0.175523 -0.574253 -0.723685 1.413890 0.262603 0.651908 -0.031703 -1.662142 -1.108057 -0.519955 0.042094 -0.243816 1.693987 -1.627640 0.726669 -0.742610 -0.233503 0.404069 0.096460 1.542869 0.189573 0.817256 -0.242230 0.177749 -0.452882 -0.631793 1.337970 -0.292573 0.228831 0.884300 -0.679648 -0.825059 -0.603834 -0.218751 -1.038333 -0.345062 -2.262123 0.108170 0.072817 -0.123277 0.256742 0.926276 0.620782 -2.252814 -1.088574 -1.259438 -2.283045 0.857796 1.607823 -0.572082 2.759763 1.249533 -0.543088 1.527922 -1.372459 1.117938 -0.675776 0.208292 0.038209 1.495299 -0.634849 0.186472 -0.076862 1.119461 -0.201049 -1.049032 -0.067333 -1.247134 1.296607 0.575811 0.926814 1.750149 -1.475150 0.298945 -0.272982 -0.087288 -1.665253 \n","0 HG00101 -0.148141 0.702527 -0.286303 1.640414 -0.402651 0.103563 0.808265 0.759659 1.003960 0.091448 -1.093641 -0.875699 -2.991989 -0.424300 -0.986872 -0.475587 0.551646 2.020389 -0.339850 2.785458 -1.586653 -1.299944 -1.689637 0.327745 1.212814 -1.537874 0.245907 0.595114 -1.690097 0.006610 0.048913 0.013519 0.704347 0.586714 2.108926 0.447339 0.951249 -0.448871 -1.143152 -0.256635 0.533253 1.337403 1.729399 -0.296529 0.586255 1.433862 -0.672089 1.368270 -1.038766 0.287326 -0.078258 -0.260583 -0.590455 -0.646735 0.509444 0.136893 -0.488343 0.475126 1.576169 0.660220 1.990986 1.207837 2.824280 -1.697204 -1.093916 0.201200 0.619543 -1.170985 -1.219977 -0.920090 0.818438 0.972410 0.881932 0.612940 -0.738142 -0.579179 1.196929 -0.393756 1.148241 -0.035538 0.766988 -0.730303 0.482248 0.196162 1.859637 1.062713 1.020459 -0.606839 0.794153 1.340428 0.701803 -1.333595 1.798924 2.096759 -0.532954 1.190743 -0.056139 -0.151213 2.534863 -2.508104 \n","0 HG00102 -1.153474 -0.133176 0.403937 2.869687 0.471179 -0.616755 0.943978 1.047253 -1.050113 -0.850521 0.863929 -0.051803 0.367596 -0.019250 0.397124 -0.429421 -0.885871 0.021471 1.228083 0.650706 -0.406759 -1.697229 -0.071715 0.340345 0.673324 -1.796388 0.649661 0.762270 -0.453092 0.064049 1.314466 1.932872 0.898561 -0.044606 1.839850 0.119532 -0.112139 -0.063383 -0.273070 -0.860160 0.626055 0.043600 -0.770078 -0.236360 1.528201 0.855514 2.065378 -0.580744 2.298632 -0.889093 0.288412 0.485126 -0.304940 -1.174962 -0.046473 -0.613603 2.025679 -0.828821 -0.166389 -0.228697 0.369272 1.074717 0.888424 0.207216 -1.451205 -0.845180 0.447617 0.157632 -0.560550 -0.389384 -0.997902 1.057493 -1.489562 0.766116 1.308001 0.689764 0.746447 1.096462 -0.946155 -0.246295 0.272506 -0.777912 0.688939 1.657566 1.678274 0.675302 0.183569 0.403201 -0.460525 0.023776 -0.568632 0.396115 -1.034947 0.231176 0.547799 -0.726820 0.814219 -0.576671 0.045563 -0.578714 \n","0 HG00103 -2.268862 -1.052555 -0.039782 0.281021 0.580325 0.238607 1.471054 -0.648345 -0.180118 1.569566 0.907766 0.264174 0.768173 -0.679915 -0.551778 -0.857984 -1.457092 0.719292 0.378513 0.230659 0.441067 0.286218 -0.683580 0.473672 0.901414 -0.098822 -0.284152 0.329456 -0.423231 -1.165698 -0.466848 -2.175364 0.009077 -0.077772 0.799616 -0.451492 -0.929382 0.799503 -0.849394 -0.116475 0.818608 1.405437 -2.004351 -0.355989 1.107447 -0.745908 -1.100004 -1.689418 0.333117 0.539550 1.361824 -0.189299 0.154131 0.311100 0.360600 -0.662708 0.728080 0.329552 1.356124 0.556709 -0.270720 -0.768515 -1.281135 1.660305 1.355265 0.127025 -1.616685 -0.715798 -0.717243 0.382096 0.464640 -0.024648 -0.630772 -1.081409 0.072610 1.343685 1.226717 -0.840397 0.948529 0.251045 1.740236 0.642885 0.593741 -0.557756 0.284290 1.144262 -1.265387 0.026654 1.339989 -0.545501 -0.906550 -0.701169 1.174239 -0.306384 -1.127198 -0.628653 0.452281 1.317396 -0.240665 -0.048269 \n","0 HG00105 1.027641 -0.649207 0.390102 2.649143 -1.072077 -0.601109 0.721914 -0.815829 0.799310 0.159079 0.166521 -0.416628 1.775886 -1.069950 2.079567 -1.518592 0.283245 1.452664 -2.305957 1.616546 -0.887177 1.017914 0.907182 0.658926 2.127195 -2.767974 -0.883922 0.410204 -2.823542 0.230110 -0.542890 0.878293 -1.645623 -0.927992 -1.730437 -0.241454 0.994863 -1.173865 1.480533 -0.923319 0.336414 0.959772 -0.518241 -0.285893 1.913778 2.130897 -0.492774 0.172899 -0.664981 -1.083506 -1.604139 2.220671 1.427044 0.625190 0.870649 -1.003520 2.101000 0.583202 -0.901243 -0.355303 -1.581638 0.196020 -0.337744 -2.718672 0.338808 0.870692 -0.365193 -2.708567 -0.663074 1.017331 1.960011 0.651076 0.845085 0.715416 0.077345 0.573598 1.943174 -0.984296 -0.862987 1.148465 0.340290 -0.266764 -1.080952 -0.945819 0.379646 0.929687 -0.232325 -0.953446 -1.623135 -2.281272 1.205545 -0.356915 0.760012 0.146469 1.197158 2.370884 -0.953288 1.395329 -1.265836 -1.547223 \n","0 HG00106 1.122411 -0.704796 -0.733577 -0.417244 0.258951 -0.058950 0.035901 -2.086222 0.679700 -0.477814 -0.132950 -0.613615 0.857358 -2.063092 -0.293953 -0.355936 -2.112654 -0.231798 0.152016 0.588626 -1.137712 -0.658318 1.016997 0.569832 0.572236 0.679211 0.724400 0.697220 -0.002052 0.633847 -0.006529 0.472650 -0.412308 0.278503 0.889999 -0.539106 -1.048502 0.720407 1.088384 -0.518957 0.837088 -0.910189 0.321500 -2.068451 0.764790 -0.961244 0.095842 -0.232765 -1.091576 -0.300629 2.057085 -0.327492 -1.394796 -0.389619 -0.712785 0.270279 0.967750 0.484355 -0.563020 -2.053524 0.612134 -0.973758 -0.016765 1.674979 0.305473 -0.061582 -0.780799 0.089166 0.375744 -1.613810 -0.400264 0.977782 -0.756442 0.682159 1.221105 0.580549 -0.523391 0.366108 0.738137 -1.033574 1.936483 -0.263201 -0.290853 0.321511 0.032990 -0.024425 0.199273 -0.730833 1.069531 -0.174533 -2.233814 -1.172636 -0.334279 1.037885 -0.964511 0.084135 0.638153 -1.287064 1.746723 0.159726 \n","0 HG00107 0.120566 0.378628 0.354385 0.335924 -0.897227 0.020770 -1.025172 0.162177 1.394531 -0.452057 -0.074504 0.251184 1.803024 -0.293938 -0.225821 0.864807 0.539390 -0.169215 0.251810 -0.989343 -2.100730 0.096437 -0.454267 1.351278 -0.305342 -0.731993 1.847661 0.160700 0.439017 0.981909 0.362510 0.219498 0.377652 -0.052197 -0.559459 1.142669 -0.113460 -0.222180 -0.463040 -0.791559 -0.488777 -0.585033 -2.114913 0.197941 1.216037 0.197912 -0.983317 -0.219720 -0.557817 -1.463518 -0.757770 0.600218 -0.934831 -1.993677 -1.677950 0.254556 -0.571512 0.381154 1.303888 -0.542476 -0.334915 0.190485 -1.763567 1.232829 -0.047250 2.093644 0.118842 -2.096547 -2.435929 -0.731157 -0.665608 0.120576 -0.964300 -0.502794 0.318942 -0.337288 1.639717 0.214786 2.865793 -0.861415 0.856943 -1.041708 2.179611 -1.742910 0.961668 -0.853595 1.755695 1.188252 -0.139601 -0.535823 -2.146615 -0.142368 -0.996788 0.305435 1.505131 -0.277769 0.962695 -1.081758 2.422881 -1.113747 \n"]}]},{"cell_type":"code","source":["! cp -r chr19_ldl_pheno_random_effects/* /content/drive/MyDrive/CSE-284-Final-Project/data/chr19_ldl_pheno_random_effects/"],"metadata":{"id":"Kz2mTqoElmKi","executionInfo":{"status":"ok","timestamp":1710111658509,"user_tz":420,"elapsed":5435,"user":{"displayName":"Keng-Chi Chang","userId":"08877207485459642010"}}},"execution_count":99,"outputs":[]}]}