{"cells":[{"cell_type":"markdown","source":["# Clumping and Thresholding (C + T)\n","## Data\n","- Superpopulations: 'SAS', 'EUR', 'EAS', 'AMR', 'AFR'\n","- Genotypes: Chromosome 19 (1000 Genomes)\n","- Phenotypes (LDAK Simulation)\n","  - Power: -0.25, -1\n","  - Heritability: 0.1, 0.3, 0.5, 0.7, 0.9\n","  - Number of Causal SNPs: 1, 10, 100, 250, 512\n","  - Number of Phenotypes (Traits): 20\n","  - Effect Sizes: fixed (use LDL PRS data)\n","- Data Split\n","  - 70% Training\n","  - 15% Validation\n","  - 15% Testing\n","\n","## References\n","- Clumping and Thresholding: UCSD CSE 284 Lecture Slides, PS3, and PS4\n","- Phenotype Simulation: https://dougspeed.com/simulations\n","- LDL PRS Data: https://www.nature.com/articles/s41586-021-04064-3\n"],"metadata":{"id":"tzcSCKBTYLUM"}},{"cell_type":"code","execution_count":1,"metadata":{"id":"ipfNy5BN61rV","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1710190296777,"user_tz":420,"elapsed":1422,"user":{"displayName":"Po-Chun Wu","userId":"13047107782036402947"}},"outputId":"410c660c-4a0e-489d-9647-0938b72d0dc1"},"outputs":[{"output_type":"stream","name":"stdout","text":["Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"]}],"source":["from google.colab import drive\n","drive.mount('/content/drive')"]},{"cell_type":"code","execution_count":2,"metadata":{"id":"h6q31sXow8et","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1710190341185,"user_tz":420,"elapsed":42933,"user":{"displayName":"Po-Chun Wu","userId":"13047107782036402947"}},"outputId":"4147fe22-c77b-440b-a1c2-3f0d94f9a823"},"outputs":[{"output_type":"stream","name":"stdout","text":["--2024-03-11 20:51:38--  https://s3.amazonaws.com/plink2-assets/alpha5/plink2_linux_x86_64_20240105.zip\n","Resolving s3.amazonaws.com (s3.amazonaws.com)... 52.216.61.216, 52.217.106.78, 52.216.152.86, ...\n","Connecting to s3.amazonaws.com (s3.amazonaws.com)|52.216.61.216|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 9266484 (8.8M) [application/zip]\n","Saving to: ‘plink2_linux_x86_64_20240105.zip.1’\n","\n","plink2_linux_x86_64 100%[===================>]   8.84M  10.6MB/s    in 0.8s    \n","\n","2024-03-11 20:51:39 (10.6 MB/s) - ‘plink2_linux_x86_64_20240105.zip.1’ saved [9266484/9266484]\n","\n","Archive:  plink2_linux_x86_64_20240105.zip\n","replace plink2? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n","  inflating: plink2                  \n","PLINK v2.00a5.10LM 64-bit Intel (5 Jan 2024)   www.cog-genomics.org/plink/2.0/\n","(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n","\n","  plink2 <input flag(s)...> [command flag(s)...] [other flag(s)...]\n","  plink2 --help [flag name(s)...]\n","\n","Commands include --rm-dup list, --make-bpgen, --export, --freq, --geno-counts,\n","--sample-counts, --missing, --hardy, --het, --fst, --indep-pairwise, --ld,\n","--sample-diff, --make-king, --king-cutoff, --pmerge, --pgen-diff,\n","--write-samples, --write-snplist, --make-grm-list, --pca, --glm, --adjust-file,\n","--gwas-ssf, --clump, --score, --variant-score, --genotyping-rate, --pgen-info,\n","--validate, and --zst-decompress.\n","\n","\"plink2 --help | more\" describes all functions.\n"]}],"source":["! wget https://s3.amazonaws.com/plink2-assets/alpha5/plink2_linux_x86_64_20240105.zip\n","! unzip plink2_linux_x86_64_20240105.zip\n","! ./plink2"]},{"cell_type":"code","execution_count":3,"metadata":{"id":"4QPf_plu8c7O","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1710190346163,"user_tz":420,"elapsed":2505,"user":{"displayName":"Po-Chun Wu","userId":"13047107782036402947"}},"outputId":"06617e15-378e-420d-d6f4-fdee509f5986"},"outputs":[{"output_type":"stream","name":"stdout","text":["Populating the interactive namespace from numpy and matplotlib\n"]}],"source":["# Import libraries\n","%pylab inline\n","import os\n","import csv\n","import scipy.stats\n","import numpy as np\n","import pandas as pd\n","import matplotlib.pyplot as plt"]},{"cell_type":"code","execution_count":6,"metadata":{"id":"e_7qXiCENvGm","executionInfo":{"status":"ok","timestamp":1710193019530,"user_tz":420,"elapsed":444,"user":{"displayName":"Po-Chun Wu","userId":"13047107782036402947"}}},"outputs":[],"source":["POP_LIST = ['SAS', 'EUR', 'EAS', 'AMR', 'AFR']\n","POWER_LIST = [-0.25, -1]\n","HER_LIST = [0.1, 0.3, 0.5, 0.7, 0.9]\n","NUM_CAUSALS_LIST = [1, 10, 100, 250, 512]\n","PHENO_NUM_START = 1\n","PHENO_NUM_END = 20"]},{"cell_type":"code","execution_count":7,"metadata":{"id":"qph4kZ5jjIbB","colab":{"base_uri":"https://localhost:8080/"},"outputId":"64979508-9643-416d-cd28-25a6932513cb","executionInfo":{"status":"ok","timestamp":1710196636191,"user_tz":420,"elapsed":462625,"user":{"displayName":"Po-Chun Wu","userId":"13047107782036402947"}}},"outputs":[{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["PLINK v2.00a5.10LM 64-bit Intel (5 Jan 2024)   www.cog-genomics.org/plink/2.0/\n","(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n","Logging to AMR_train.log.\n","Options in effect:\n","  --bfile /content/chr19\n","  --keep AMR_train_IID_only.txt\n","  --make-bed\n","  --out AMR_train\n","\n","Start time: Mon Mar 11 21:37:05 2024\n","12978 MiB RAM detected, ~11545 available; reserving 6489 MiB for main\n","workspace.\n","Using up to 2 compute threads.\n","2504 samples (1270 females, 1234 males; 2497 founders) loaded from\n","/content/chr19.fam.\n","283173 variants loaded from /content/chr19.bim.\n","Note: No phenotype data present.\n","--keep: 245 samples remaining.\n","245 samples (126 females, 119 males; 245 founders) remaining after main\n","filters.\n","Writing AMR_train.fam ... done.\n","Writing AMR_train.bim ... done.\n","Writing AMR_train.bed ... 0%\b\b23%\b\b\b46%\b\b\b69%\b\b\b92%\b\b\bdone.\n","End time: Mon Mar 11 21:37:06 2024\n","PLINK v2.00a5.10LM 64-bit Intel (5 Jan 2024)   www.cog-genomics.org/plink/2.0/\n","(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n","Logging to AMR_val.log.\n","Options in effect:\n","  --bfile /content/chr19\n","  --keep AMR_val_IID_only.txt\n","  --make-bed\n","  --out AMR_val\n","\n","Start time: Mon Mar 11 21:37:06 2024\n","12978 MiB RAM detected, ~11530 available; reserving 6489 MiB for main\n","workspace.\n","Using up to 2 compute threads.\n","2504 samples (1270 females, 1234 males; 2497 founders) loaded from\n","/content/chr19.fam.\n","283173 variants loaded from /content/chr19.bim.\n","Note: No phenotype data present.\n","--keep: 47 samples remaining.\n","47 samples (23 females, 24 males; 47 founders) remaining after main filters.\n","Writing AMR_val.fam ... done.\n","Writing AMR_val.bim ... done.\n","Writing AMR_val.bed ... 0%\b\b23%\b\b\b46%\b\b\b69%\b\b\b92%\b\b\bdone.\n","End time: Mon Mar 11 21:37:07 2024\n","PLINK v2.00a5.10LM 64-bit Intel (5 Jan 2024)   www.cog-genomics.org/plink/2.0/\n","(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n","Logging to AMR_test.log.\n","Options in effect:\n","  --bfile /content/chr19\n","  --keep AMR_test_IID_only.txt\n","  --make-bed\n","  --out AMR_test\n","\n","Start time: Mon Mar 11 21:37:07 2024\n","12978 MiB RAM detected, ~11548 available; reserving 6489 MiB for main\n","workspace.\n","Using up to 2 compute threads.\n","2504 samples (1270 females, 1234 males; 2497 founders) loaded from\n","/content/chr19.fam.\n","283173 variants loaded from /content/chr19.bim.\n","Note: No phenotype data present.\n","--keep: 55 samples remaining.\n","55 samples (28 females, 27 males; 55 founders) remaining after main filters.\n","Writing AMR_test.fam ... done.\n","Writing AMR_test.bim ... done.\n","Writing AMR_test.bed ... 0%\b\b23%\b\b\b46%\b\b\b69%\b\b\b92%\b\b\bdone.\n","End time: Mon Mar 11 21:37:07 2024\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6863669971233877\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.658973434715183e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6302099678490773\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0514474891516073\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141724 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141724 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9005043370339112\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002190404429987215\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5858342367369334\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007333873783287051\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6894616212078695\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.049721941224837106\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8812520225108533\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004037569156006645\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7852693608992571\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.05600460260303446\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8793104027790917\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00023245921147294746\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8247845954100136\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09473081986837953\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7555195444095896\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=4.2702793562446006e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8053415513374002\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07568038456474058\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141842 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141842 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.40982974225143626\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=8.346981535032721e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8846495499042618\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05632892429913966\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8330101840174148\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=2.0191151662232883e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.901585979936243\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023569419375970704\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9196355600148848\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03274791563618113\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5959009524524952\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009230540619901383\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.47100589208210863\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.024929595327879\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8103037584319102\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.9937712236594455e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8205750518213402\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06659243295152113\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.1_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.1_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  29.141500 -0.411467          1\n","1     HG01052  39.416000 -0.651482          1\n","2     HG01064  17.843500 -0.895190          1\n","3     HG01069   5.454260  1.935899          1\n","4     HG01079  28.908300 -0.700593          1\n","...       ...        ...       ...        ...\n","1095  NA19734   9.288270 -0.749206         20\n","1096  NA19741  20.639400 -0.101993         20\n","1097  NA19752 -17.736500 -2.216098         20\n","1098  NA19759  29.119900  0.249379         20\n","1099  NA19770  -0.592406  1.206749         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.1            1          1  0.686367   \n","1                         AMR  -0.25  0.1            1          2  0.630210   \n","2                         AMR  -0.25  0.1            1          3  0.900504   \n","3                         AMR  -0.25  0.1            1          4  0.585834   \n","4                         AMR  -0.25  0.1            1          5  0.689462   \n","5                         AMR  -0.25  0.1            1          6  0.881252   \n","6                         AMR  -0.25  0.1            1          7  0.785269   \n","7                         AMR  -0.25  0.1            1          8  0.879310   \n","8                         AMR  -0.25  0.1            1          9  0.824785   \n","9                         AMR  -0.25  0.1            1         10  0.755520   \n","10                        AMR  -0.25  0.1            1         11  0.805342   \n","11                        AMR  -0.25  0.1            1         12  0.409830   \n","12                        AMR  -0.25  0.1            1         13  0.884650   \n","13                        AMR  -0.25  0.1            1         14  0.833010   \n","14                        AMR  -0.25  0.1            1         15  0.901586   \n","15                        AMR  -0.25  0.1            1         16  0.919636   \n","16                        AMR  -0.25  0.1            1         17  0.595901   \n","17                        AMR  -0.25  0.1            1         18  0.471006   \n","18                        AMR  -0.25  0.1            1         19  0.810304   \n","19                        AMR  -0.25  0.1            1         20  0.820575   \n","\n","    val      test  \n","0    -1  0.000047  \n","1    -1  0.051447  \n","2    -1  0.002190  \n","3    -1  0.007334  \n","4    -1  0.049722  \n","5    -1  0.004038  \n","6    -1  0.056005  \n","7    -1  0.000232  \n","8    -1  0.094731  \n","9    -1  0.000043  \n","10   -1  0.075680  \n","11   -1  0.000083  \n","12   -1  0.056329  \n","13   -1  0.000020  \n","14   -1  0.023569  \n","15   -1  0.032748  \n","16   -1  0.009231  \n","17   -1  0.024930  \n","18   -1  0.000005  \n","19   -1  0.066592  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6863247770440383\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.011334210753565797\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8991258349088799\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=7.836892425957998e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6829837303278435\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0026246667560536224\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7052828821323507\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016581293098917184\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7515842474902596\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004912487375383282\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7572038979330068\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.042474907687624214\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8162384608770696\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003229832078756597\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8912502273105851\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.034178564277521355\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7943136718153149\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0030471380011293\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9141574769542702\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03731632385098541\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9088767057909101\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0017683610385163659\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4295340671924069\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032815808648987365\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.543640120924005\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01913559817611561\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9257276131543887\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.166191989395098e-08\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5020011525776611\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03228125635125472\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4955473373217039\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016203460516385926\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8367333103289835\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0007899468789580796\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9093212367659691\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00014684315464375116\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9180791356457974\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009004323403393083\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8941613569498883\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06252967953910997\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.1_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.1_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  26.40570 -0.138899          1\n","1     HG01052  18.77080 -0.514488          1\n","2     HG01064  22.77610  0.918510          1\n","3     HG01069   9.16733  0.425325          1\n","4     HG01079  28.50010  0.537220          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -1.31554 -1.648347         20\n","1096  NA19741   2.06260 -0.485029         20\n","1097  NA19752   6.13574  0.477829         20\n","1098  NA19759  12.09440 -0.079882         20\n","1099  NA19770 -13.82810 -1.061384         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.1           10          1  0.686325   \n","1                         AMR  -0.25  0.1           10          2  0.899126   \n","2                         AMR  -0.25  0.1           10          3  0.682984   \n","3                         AMR  -0.25  0.1           10          4  0.705283   \n","4                         AMR  -0.25  0.1           10          5  0.751584   \n","5                         AMR  -0.25  0.1           10          6  0.757204   \n","6                         AMR  -0.25  0.1           10          7  0.816238   \n","7                         AMR  -0.25  0.1           10          8  0.891250   \n","8                         AMR  -0.25  0.1           10          9  0.794314   \n","9                         AMR  -0.25  0.1           10         10  0.914157   \n","10                        AMR  -0.25  0.1           10         11  0.908877   \n","11                        AMR  -0.25  0.1           10         12  0.429534   \n","12                        AMR  -0.25  0.1           10         13  0.543640   \n","13                        AMR  -0.25  0.1           10         14  0.925728   \n","14                        AMR  -0.25  0.1           10         15  0.502001   \n","15                        AMR  -0.25  0.1           10         16  0.495547   \n","16                        AMR  -0.25  0.1           10         17  0.836733   \n","17                        AMR  -0.25  0.1           10         18  0.909321   \n","18                        AMR  -0.25  0.1           10         19  0.918079   \n","19                        AMR  -0.25  0.1           10         20  0.894161   \n","\n","    val          test  \n","0    -1  1.133421e-02  \n","1    -1  7.836892e-05  \n","2    -1  2.624667e-03  \n","3    -1  1.658129e-02  \n","4    -1  4.912487e-03  \n","5    -1  4.247491e-02  \n","6    -1  3.229832e-03  \n","7    -1  3.417856e-02  \n","8    -1  3.047138e-03  \n","9    -1  3.731632e-02  \n","10   -1  1.768361e-03  \n","11   -1  3.281581e-03  \n","12   -1  1.913560e-02  \n","13   -1  5.166192e-08  \n","14   -1  3.228126e-02  \n","15   -1  1.620346e-02  \n","16   -1  7.899469e-04  \n","17   -1  1.468432e-04  \n","18   -1  9.004323e-03  \n","19   -1  6.252968e-02  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8843215525589617\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00047953652594442584\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8937939609917017\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006006191304344115\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.675399953068564\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.01192883873176158\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7160908272839018\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00344156589439036\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5240342078862164\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05922143624904978\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.663774218114472\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002127138527306462\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8411830681323573\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0041146873399612585\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8427699515941695\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03148322123960873\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9125593869860079\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0041280014378384565\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8341024450761063\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025623960038308417\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.818566921128935\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026760083958079408\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6591970312310284\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017723630784724526\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8455000559531162\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017601914561558664\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.734697276406074\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.136859391215108e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7601353078961995\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00990419514167683\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8653659228638061\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0009070845007590261\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8621076504754591\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0001324207470194331\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7623634941979899\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.10421980269913782\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5748668680916683\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08635635209516047\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.837503715964229\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0010223835384062646\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.1_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.1_num-causals=100_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   0.856458  2.282394          1\n","1     HG01052  17.516100 -0.944464          1\n","2     HG01064   9.127800  0.989812          1\n","3     HG01069  -3.232950 -0.129926          1\n","4     HG01079  16.056200 -0.299781          1\n","...       ...        ...       ...        ...\n","1095  NA19734  -3.585720 -1.056622         20\n","1096  NA19741   6.610920  0.538659         20\n","1097  NA19752  -9.055370  0.034906         20\n","1098  NA19759   0.388936 -1.028117         20\n","1099  NA19770 -17.137100 -0.354321         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.1          100          1  0.884322   \n","1                         AMR  -0.25  0.1          100          2  0.893794   \n","2                         AMR  -0.25  0.1          100          3  0.675400   \n","3                         AMR  -0.25  0.1          100          4  0.716091   \n","4                         AMR  -0.25  0.1          100          5  0.524034   \n","5                         AMR  -0.25  0.1          100          6  0.663774   \n","6                         AMR  -0.25  0.1          100          7  0.841183   \n","7                         AMR  -0.25  0.1          100          8  0.842770   \n","8                         AMR  -0.25  0.1          100          9  0.912559   \n","9                         AMR  -0.25  0.1          100         10  0.834102   \n","10                        AMR  -0.25  0.1          100         11  0.818567   \n","11                        AMR  -0.25  0.1          100         12  0.659197   \n","12                        AMR  -0.25  0.1          100         13  0.845500   \n","13                        AMR  -0.25  0.1          100         14  0.734697   \n","14                        AMR  -0.25  0.1          100         15  0.760135   \n","15                        AMR  -0.25  0.1          100         16  0.865366   \n","16                        AMR  -0.25  0.1          100         17  0.862108   \n","17                        AMR  -0.25  0.1          100         18  0.762363   \n","18                        AMR  -0.25  0.1          100         19  0.574867   \n","19                        AMR  -0.25  0.1          100         20  0.837504   \n","\n","    val      test  \n","0    -1  0.000480  \n","1    -1  0.006006  \n","2    -1  0.011929  \n","3    -1  0.003442  \n","4    -1  0.059221  \n","5    -1  0.002127  \n","6    -1  0.004115  \n","7    -1  0.031483  \n","8    -1  0.004128  \n","9    -1  0.025624  \n","10   -1  0.026760  \n","11   -1  0.017724  \n","12   -1  0.017602  \n","13   -1  0.000005  \n","14   -1  0.009904  \n","15   -1  0.000907  \n","16   -1  0.000132  \n","17   -1  0.104220  \n","18   -1  0.086356  \n","19   -1  0.001022  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8869584791361423\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00014533785884521014\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.776318297163563\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0030363463364659967\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8904731501919185\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=7.689831211485175e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6113626173001284\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009478974677595715\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8884160224580695\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006436909142499616\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7021842132263628\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06010884437042299\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4766134858804241\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07854857201827162\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7529336045784838\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00652169001366794\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5847328098631454\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02692438422680929\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9132277278556311\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007306933854387518\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8361205950059934\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.052035313809996e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8857017760706367\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0014329495038495521\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6827504218884433\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.017620653678123707\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5537221934250696\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0015357100995907675\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.714137112299149\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002144622348124225\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8874953725572634\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008599542341252304\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8637969509338839\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004343802046426828\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7787341259832176\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02445835642050631\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7497746388024237\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0013947728748982323\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9217888243739225\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.373056798714814e-07\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.1_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.1_num-causals=250_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  20.418400  2.098320          1\n","1     HG01052  15.742400 -0.633637          1\n","2     HG01064  18.284000 -0.556495          1\n","3     HG01069  18.623500 -0.578952          1\n","4     HG01079  11.495100  0.437964          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.550078 -0.648941         20\n","1096  NA19741   0.575975 -0.634843         20\n","1097  NA19752  14.154300  0.272276         20\n","1098  NA19759  19.774300  0.874575         20\n","1099  NA19770   2.860420 -0.297002         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.1          250          1  0.886958   \n","1                         AMR  -0.25  0.1          250          2  0.776318   \n","2                         AMR  -0.25  0.1          250          3  0.890473   \n","3                         AMR  -0.25  0.1          250          4  0.611363   \n","4                         AMR  -0.25  0.1          250          5  0.888416   \n","5                         AMR  -0.25  0.1          250          6  0.702184   \n","6                         AMR  -0.25  0.1          250          7  0.476613   \n","7                         AMR  -0.25  0.1          250          8  0.752934   \n","8                         AMR  -0.25  0.1          250          9  0.584733   \n","9                         AMR  -0.25  0.1          250         10  0.913228   \n","10                        AMR  -0.25  0.1          250         11  0.836121   \n","11                        AMR  -0.25  0.1          250         12  0.885702   \n","12                        AMR  -0.25  0.1          250         13  0.682750   \n","13                        AMR  -0.25  0.1          250         14  0.553722   \n","14                        AMR  -0.25  0.1          250         15  0.714137   \n","15                        AMR  -0.25  0.1          250         16  0.887495   \n","16                        AMR  -0.25  0.1          250         17  0.863797   \n","17                        AMR  -0.25  0.1          250         18  0.778734   \n","18                        AMR  -0.25  0.1          250         19  0.749775   \n","19                        AMR  -0.25  0.1          250         20  0.921789   \n","\n","    val          test  \n","0    -1  1.453379e-04  \n","1    -1  3.036346e-03  \n","2    -1  7.689831e-05  \n","3    -1  9.478975e-03  \n","4    -1  6.436909e-03  \n","5    -1  6.010884e-02  \n","6    -1  7.854857e-02  \n","7    -1  6.521690e-03  \n","8    -1  2.692438e-02  \n","9    -1  7.306934e-03  \n","10   -1  5.052035e-05  \n","11   -1  1.432950e-03  \n","12   -1  1.762065e-02  \n","13   -1  1.535710e-03  \n","14   -1  2.144622e-03  \n","15   -1  8.599542e-03  \n","16   -1  4.343802e-03  \n","17   -1  2.445836e-02  \n","18   -1  1.394773e-03  \n","19   -1  3.373057e-07  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6461736303064762\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0018443052208554395\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8825029999997575\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.01833062618289443\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6393675311793595\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023978733282366892\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5113602290416416\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.007180785285007315\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8589422447013207\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.019376219224622564\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.930973196527081\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=6.981647145666961e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5703191595813345\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11300752198816588\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.892047497578343\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00018215326714321781\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8570512549270226\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0031013447007493507\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6736085471491549\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006090508879662949\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.913529779111706\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0019451593695946079\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8497889223628191\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008134838465805011\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9025268172761043\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.027879092780882436\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6664748406108835\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0023017022995289603\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6356430875742884\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006782338577572782\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6303126178501677\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06812069004369888\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141836 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141836 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.46526597028394967\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0023680021015321487\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6060195107900013\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013808863612950681\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.82730432184027\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03349029486371669\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5765387194797791\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006472291381825815\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.1_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.1_num-causals=512_pheno=1-20.csv\n","          IID  predict    actual  pheno_num\n","0     HG00732  14.6884 -0.069521          1\n","1     HG01052  25.2560 -1.456309          1\n","2     HG01064  20.1708 -0.835636          1\n","3     HG01069  12.3943 -0.686153          1\n","4     HG01079  20.6501 -1.279049          1\n","...       ...      ...       ...        ...\n","1095  NA19734 -25.5377 -1.793706         20\n","1096  NA19741 -30.0360 -1.101185         20\n","1097  NA19752  30.7929  0.001157         20\n","1098  NA19759 -49.4605  0.147343         20\n","1099  NA19770  23.5815  0.562027         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.1          512          1  0.646174   \n","1                         AMR  -0.25  0.1          512          2  0.882503   \n","2                         AMR  -0.25  0.1          512          3  0.639368   \n","3                         AMR  -0.25  0.1          512          4  0.511360   \n","4                         AMR  -0.25  0.1          512          5  0.858942   \n","5                         AMR  -0.25  0.1          512          6  0.930973   \n","6                         AMR  -0.25  0.1          512          7  0.570319   \n","7                         AMR  -0.25  0.1          512          8  0.892047   \n","8                         AMR  -0.25  0.1          512          9  0.857051   \n","9                         AMR  -0.25  0.1          512         10  0.673609   \n","10                        AMR  -0.25  0.1          512         11  0.913530   \n","11                        AMR  -0.25  0.1          512         12  0.849789   \n","12                        AMR  -0.25  0.1          512         13  0.902527   \n","13                        AMR  -0.25  0.1          512         14  0.666475   \n","14                        AMR  -0.25  0.1          512         15  0.635643   \n","15                        AMR  -0.25  0.1          512         16  0.630313   \n","16                        AMR  -0.25  0.1          512         17  0.465266   \n","17                        AMR  -0.25  0.1          512         18  0.606020   \n","18                        AMR  -0.25  0.1          512         19  0.827304   \n","19                        AMR  -0.25  0.1          512         20  0.576539   \n","\n","    val      test  \n","0    -1  0.001844  \n","1    -1  0.018331  \n","2    -1  0.023979  \n","3    -1  0.007181  \n","4    -1  0.019376  \n","5    -1  0.000070  \n","6    -1  0.113008  \n","7    -1  0.000182  \n","8    -1  0.003101  \n","9    -1  0.006091  \n","10   -1  0.001945  \n","11   -1  0.008135  \n","12   -1  0.027879  \n","13   -1  0.002302  \n","14   -1  0.006782  \n","15   -1  0.068121  \n","16   -1  0.002368  \n","17   -1  0.013809  \n","18   -1  0.033490  \n","19   -1  0.006472  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8801222825125266\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0015100253861724763\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.48838614653200435\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0006025812662547389\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6477792227774022\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010050355557372692\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.830415736739755\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0127201070948958\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9166910973467908\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.8550045863750214e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7715293837626506\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013787159690625442\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7085629469335558\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025007659684385543\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8897849753877227\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.051271433016319055\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4913304486255475\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01086075112879851\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6014274727687423\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024669411444481638\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5451423553633368\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025757604746537648\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4282916614236254\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0011610703894173984\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8032933897384078\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0013485178259734633\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8022208828887467\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014977796376490934\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6957750469502474\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04210348712067007\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5391226954432775\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01717084540236416\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9238274627392927\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.012190615474633456\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5053138218282623\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.09336634433432091\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5371139352362485\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03186894106463726\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5178755502352965\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04112248641628478\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.3_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.3_num-causals=1_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  11.41860 -0.908949          1\n","1     HG01052  12.72340 -0.248359          1\n","2     HG01064   8.10691 -0.077254          1\n","3     HG01069  -1.56388 -0.913443          1\n","4     HG01079   6.25817  0.423618          1\n","...       ...       ...       ...        ...\n","1095  NA19734   4.68808 -0.473077         20\n","1096  NA19741  51.92780  1.553869         20\n","1097  NA19752 -74.78960 -0.645275         20\n","1098  NA19759  67.46670  0.254821         20\n","1099  NA19770 -20.33380 -0.076133         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.3            1          1  0.880122   \n","1                         AMR  -0.25  0.3            1          2  0.488386   \n","2                         AMR  -0.25  0.3            1          3  0.647779   \n","3                         AMR  -0.25  0.3            1          4  0.830416   \n","4                         AMR  -0.25  0.3            1          5  0.916691   \n","5                         AMR  -0.25  0.3            1          6  0.771529   \n","6                         AMR  -0.25  0.3            1          7  0.708563   \n","7                         AMR  -0.25  0.3            1          8  0.889785   \n","8                         AMR  -0.25  0.3            1          9  0.491330   \n","9                         AMR  -0.25  0.3            1         10  0.601427   \n","10                        AMR  -0.25  0.3            1         11  0.545142   \n","11                        AMR  -0.25  0.3            1         12  0.428292   \n","12                        AMR  -0.25  0.3            1         13  0.803293   \n","13                        AMR  -0.25  0.3            1         14  0.802221   \n","14                        AMR  -0.25  0.3            1         15  0.695775   \n","15                        AMR  -0.25  0.3            1         16  0.539123   \n","16                        AMR  -0.25  0.3            1         17  0.923827   \n","17                        AMR  -0.25  0.3            1         18  0.505314   \n","18                        AMR  -0.25  0.3            1         19  0.537114   \n","19                        AMR  -0.25  0.3            1         20  0.517876   \n","\n","    val      test  \n","0    -1  0.001510  \n","1    -1  0.000603  \n","2    -1  0.010050  \n","3    -1  0.012720  \n","4    -1  0.000002  \n","5    -1  0.013787  \n","6    -1  0.025008  \n","7    -1  0.051271  \n","8    -1  0.010861  \n","9    -1  0.024669  \n","10   -1  0.025758  \n","11   -1  0.001161  \n","12   -1  0.001349  \n","13   -1  0.014978  \n","14   -1  0.042103  \n","15   -1  0.017171  \n","16   -1  0.012191  \n","17   -1  0.093366  \n","18   -1  0.031869  \n","19   -1  0.041122  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6695541840521063\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0025541623099478044\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141820 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141820 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4376671974829838\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01559624436128301\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.657588338181299\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023885534416102323\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141708 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141708 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8321847912177891\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.10717379382382716\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5508706361602168\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1045600741016455\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6860172738151907\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0028868032001278087\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141837 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141837 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6790423348527393\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.06416296851366281\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8565641495802366\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014567548978039763\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8311940607677378\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016142642073321008\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6231932437141378\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04398850196353625\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9066834728561197\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00021137854507077201\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141829 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141829 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.41724052010843193\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.05422510563901975\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.486335429777449\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.08243263950995706\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5361638651908933\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.001087173202621675\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.712655144638123\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.015379433695534907\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141813 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141813 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48035503708481414\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003521518441278819\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8765987754185608\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00016430893586905123\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8117977292399844\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.013125423621303351\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8276531463018308\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004374598372043716\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7418434373612512\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006931656478883122\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.3_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.3_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  21.70380 -0.567840          1\n","1     HG01052  17.42900  0.715505          1\n","2     HG01064  13.32300  0.178649          1\n","3     HG01069  -1.10137 -0.494317          1\n","4     HG01079  27.33550  1.740132          1\n","...       ...       ...       ...        ...\n","1095  NA19734   3.17627 -0.577095         20\n","1096  NA19741  -5.63410 -0.550374         20\n","1097  NA19752  26.02110  0.223152         20\n","1098  NA19759 -18.51770 -0.105933         20\n","1099  NA19770  13.73430 -1.012808         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.3           10          1  0.669554   \n","1                         AMR  -0.25  0.3           10          2  0.437667   \n","2                         AMR  -0.25  0.3           10          3  0.657588   \n","3                         AMR  -0.25  0.3           10          4  0.832185   \n","4                         AMR  -0.25  0.3           10          5  0.550871   \n","5                         AMR  -0.25  0.3           10          6  0.686017   \n","6                         AMR  -0.25  0.3           10          7  0.679042   \n","7                         AMR  -0.25  0.3           10          8  0.856564   \n","8                         AMR  -0.25  0.3           10          9  0.831194   \n","9                         AMR  -0.25  0.3           10         10  0.623193   \n","10                        AMR  -0.25  0.3           10         11  0.906683   \n","11                        AMR  -0.25  0.3           10         12  0.417241   \n","12                        AMR  -0.25  0.3           10         13  0.486335   \n","13                        AMR  -0.25  0.3           10         14  0.536164   \n","14                        AMR  -0.25  0.3           10         15  0.712655   \n","15                        AMR  -0.25  0.3           10         16  0.480355   \n","16                        AMR  -0.25  0.3           10         17  0.876599   \n","17                        AMR  -0.25  0.3           10         18  0.811798   \n","18                        AMR  -0.25  0.3           10         19  0.827653   \n","19                        AMR  -0.25  0.3           10         20  0.741843   \n","\n","    val      test  \n","0    -1  0.002554  \n","1    -1  0.015596  \n","2    -1  0.023886  \n","3    -1  0.107174  \n","4    -1  0.104560  \n","5    -1  0.002887  \n","6    -1  0.064163  \n","7    -1  0.014568  \n","8    -1  0.016143  \n","9    -1  0.043989  \n","10   -1  0.000211  \n","11   -1  0.054225  \n","12   -1  0.082433  \n","13   -1  0.001087  \n","14   -1  0.015379  \n","15   -1  0.003522  \n","16   -1  0.000164  \n","17   -1  0.013125  \n","18   -1  0.004375  \n","19   -1  0.006932  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8452067473278133\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06392271333591294\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8369336331639916\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.050769877148555936\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6988465390207991\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00504063585494132\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.531122511975341\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009771446499812722\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8810557538498957\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02548499051180201\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7450158479597306\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008239435427616723\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9036429637273411\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.039563612664284147\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8746006907464056\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06519596744701527\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7929805207933985\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.020423995838785403\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6723663032979457\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0036097251014110053\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7212539932337098\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008564836391073022\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4222880559612361\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0056204433698431805\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8718230038733769\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09693781348157199\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5210694760142767\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.024769525542809662\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6073370618490078\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08181228601803492\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8003694257580848\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08639315226272978\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8870901226150478\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.042718295145400625\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6228839061541546\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0007441488611519977\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.47717928571729257\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06243837684717239\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8992531596302779\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01098312157282959\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.3_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.3_num-causals=100_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   8.720830  1.616577          1\n","1     HG01052   3.285850  0.687103          1\n","2     HG01064   3.423410 -0.329484          1\n","3     HG01069  -0.990267 -0.756232          1\n","4     HG01079  18.420100  0.893430          1\n","...       ...        ...       ...        ...\n","1095  NA19734   1.769320 -0.718298         20\n","1096  NA19741  -4.957690  0.758557         20\n","1097  NA19752   5.837960  0.728147         20\n","1098  NA19759   5.212840 -1.979975         20\n","1099  NA19770   7.248140  0.081360         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.3          100          1  0.845207   \n","1                         AMR  -0.25  0.3          100          2  0.836934   \n","2                         AMR  -0.25  0.3          100          3  0.698847   \n","3                         AMR  -0.25  0.3          100          4  0.531123   \n","4                         AMR  -0.25  0.3          100          5  0.881056   \n","5                         AMR  -0.25  0.3          100          6  0.745016   \n","6                         AMR  -0.25  0.3          100          7  0.903643   \n","7                         AMR  -0.25  0.3          100          8  0.874601   \n","8                         AMR  -0.25  0.3          100          9  0.792981   \n","9                         AMR  -0.25  0.3          100         10  0.672366   \n","10                        AMR  -0.25  0.3          100         11  0.721254   \n","11                        AMR  -0.25  0.3          100         12  0.422288   \n","12                        AMR  -0.25  0.3          100         13  0.871823   \n","13                        AMR  -0.25  0.3          100         14  0.521069   \n","14                        AMR  -0.25  0.3          100         15  0.607337   \n","15                        AMR  -0.25  0.3          100         16  0.800369   \n","16                        AMR  -0.25  0.3          100         17  0.887090   \n","17                        AMR  -0.25  0.3          100         18  0.622884   \n","18                        AMR  -0.25  0.3          100         19  0.477179   \n","19                        AMR  -0.25  0.3          100         20  0.899253   \n","\n","    val      test  \n","0    -1  0.063923  \n","1    -1  0.050770  \n","2    -1  0.005041  \n","3    -1  0.009771  \n","4    -1  0.025485  \n","5    -1  0.008239  \n","6    -1  0.039564  \n","7    -1  0.065196  \n","8    -1  0.020424  \n","9    -1  0.003610  \n","10   -1  0.008565  \n","11   -1  0.005620  \n","12   -1  0.096938  \n","13   -1  0.024770  \n","14   -1  0.081812  \n","15   -1  0.086393  \n","16   -1  0.042718  \n","17   -1  0.000744  \n","18   -1  0.062438  \n","19   -1  0.010983  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6516908734299025\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.014838277026742817\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6737517972675084\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.018691601248965215\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8373333809987573\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0461271071702322\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6409395947388832\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07380153478585068\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6856941962389883\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002620718769841843\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6782816816388412\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.14765042533629993\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.656414380588168\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0011503904708917094\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8698921774527608\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030487877204542907\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141702 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141702 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8775144597146471\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017423841171393218\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5905952507057559\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003339340426502892\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8549453884061651\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=6.559446201309409e-07\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5486095521308214\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.186346978771952e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8926313710460381\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0014560724387305253\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7133100270572666\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.046414712544824405\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8198237236886073\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005444837785796841\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141714 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141714 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8919382377398418\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.058177923401267415\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6549725418203329\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005612991920757027\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8679748517810337\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00018572577312842145\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8912261850891784\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016431207620801656\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7375416172600919\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004455269625431346\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.3_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.3_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  22.57130 -0.645767          1\n","1     HG01052  13.58680 -0.363156          1\n","2     HG01064  10.71490 -0.827478          1\n","3     HG01069   2.64249  0.459998          1\n","4     HG01079  20.89130  0.847382          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -3.29495  0.488572         20\n","1096  NA19741 -23.40440  0.676620         20\n","1097  NA19752  28.64970  0.795924         20\n","1098  NA19759 -13.17350 -1.307914         20\n","1099  NA19770  14.27420  2.758219         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.3          250          1  0.651691   \n","1                         AMR  -0.25  0.3          250          2  0.673752   \n","2                         AMR  -0.25  0.3          250          3  0.837333   \n","3                         AMR  -0.25  0.3          250          4  0.640940   \n","4                         AMR  -0.25  0.3          250          5  0.685694   \n","5                         AMR  -0.25  0.3          250          6  0.678282   \n","6                         AMR  -0.25  0.3          250          7  0.656414   \n","7                         AMR  -0.25  0.3          250          8  0.869892   \n","8                         AMR  -0.25  0.3          250          9  0.877514   \n","9                         AMR  -0.25  0.3          250         10  0.590595   \n","10                        AMR  -0.25  0.3          250         11  0.854945   \n","11                        AMR  -0.25  0.3          250         12  0.548610   \n","12                        AMR  -0.25  0.3          250         13  0.892631   \n","13                        AMR  -0.25  0.3          250         14  0.713310   \n","14                        AMR  -0.25  0.3          250         15  0.819824   \n","15                        AMR  -0.25  0.3          250         16  0.891938   \n","16                        AMR  -0.25  0.3          250         17  0.654973   \n","17                        AMR  -0.25  0.3          250         18  0.867975   \n","18                        AMR  -0.25  0.3          250         19  0.891226   \n","19                        AMR  -0.25  0.3          250         20  0.737542   \n","\n","    val          test  \n","0    -1  1.483828e-02  \n","1    -1  1.869160e-02  \n","2    -1  4.612711e-02  \n","3    -1  7.380153e-02  \n","4    -1  2.620719e-03  \n","5    -1  1.476504e-01  \n","6    -1  1.150390e-03  \n","7    -1  3.048788e-02  \n","8    -1  1.742384e-02  \n","9    -1  3.339340e-03  \n","10   -1  6.559446e-07  \n","11   -1  5.186347e-05  \n","12   -1  1.456072e-03  \n","13   -1  4.641471e-02  \n","14   -1  5.444838e-03  \n","15   -1  5.817792e-02  \n","16   -1  5.612992e-03  \n","17   -1  1.857258e-04  \n","18   -1  1.643121e-02  \n","19   -1  4.455270e-03  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.696866997341503\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.9928060943003636e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7942058788018787\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0654670537783744\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9078914199300794\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=7.431338196973992e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5401143875792306\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0005105176738802806\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8583455709911783\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00827755104757376\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7836677843439184\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004414849824238638\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.48246778705895504\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0003148833579182287\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9000786070497425\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.028314840067810227\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141704 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141704 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9155095213566463\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00018390160795195572\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8216092855140829\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.029517245216239267\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6403150493481541\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005668652779473219\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8607511581887517\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08948750005296349\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7000998982756144\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.000609150911575657\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6874676291668051\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.018714777622304418\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5207206334923117\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0019084658758129192\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4729894723115079\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.529739801520874e-07\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9248366367284303\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07303250448796858\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8264640767762609\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.5545736710612113e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6239055736345245\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03897161427696421\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7034351340884292\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017635601007257726\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.3_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.3_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  33.66730  0.150327          1\n","1     HG01052  22.43480 -0.236247          1\n","2     HG01064  15.63200 -0.327305          1\n","3     HG01069   7.04244 -0.425427          1\n","4     HG01079  37.71070  0.037393          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -7.88222 -0.892920         20\n","1096  NA19741 -20.90130  0.725302         20\n","1097  NA19752  41.83690  0.092701         20\n","1098  NA19759 -19.12280 -0.629710         20\n","1099  NA19770  28.10780  0.688178         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.3          512          1  0.696867   \n","1                         AMR  -0.25  0.3          512          2  0.794206   \n","2                         AMR  -0.25  0.3          512          3  0.907891   \n","3                         AMR  -0.25  0.3          512          4  0.540114   \n","4                         AMR  -0.25  0.3          512          5  0.858346   \n","5                         AMR  -0.25  0.3          512          6  0.783668   \n","6                         AMR  -0.25  0.3          512          7  0.482468   \n","7                         AMR  -0.25  0.3          512          8  0.900079   \n","8                         AMR  -0.25  0.3          512          9  0.915510   \n","9                         AMR  -0.25  0.3          512         10  0.821609   \n","10                        AMR  -0.25  0.3          512         11  0.640315   \n","11                        AMR  -0.25  0.3          512         12  0.860751   \n","12                        AMR  -0.25  0.3          512         13  0.700100   \n","13                        AMR  -0.25  0.3          512         14  0.687468   \n","14                        AMR  -0.25  0.3          512         15  0.520721   \n","15                        AMR  -0.25  0.3          512         16  0.472989   \n","16                        AMR  -0.25  0.3          512         17  0.924837   \n","17                        AMR  -0.25  0.3          512         18  0.826464   \n","18                        AMR  -0.25  0.3          512         19  0.623906   \n","19                        AMR  -0.25  0.3          512         20  0.703435   \n","\n","    val          test  \n","0    -1  3.992806e-05  \n","1    -1  6.546705e-02  \n","2    -1  7.431338e-06  \n","3    -1  5.105177e-04  \n","4    -1  8.277551e-03  \n","5    -1  4.414850e-03  \n","6    -1  3.148834e-04  \n","7    -1  2.831484e-02  \n","8    -1  1.839016e-04  \n","9    -1  2.951725e-02  \n","10   -1  5.668653e-03  \n","11   -1  8.948750e-02  \n","12   -1  6.091509e-04  \n","13   -1  1.871478e-02  \n","14   -1  1.908466e-03  \n","15   -1  3.529740e-07  \n","16   -1  7.303250e-02  \n","17   -1  1.554574e-06  \n","18   -1  3.897161e-02  \n","19   -1  1.763560e-02  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7070934429388529\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01880949741242471\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6299350690437533\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.030643108918152e-07\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8482506788900941\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.677618588997073e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6860679971622907\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.04872293887229104\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8160080171141115\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00010205366397412505\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7176601297408353\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.015425539318142239\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5855116141188461\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026360852922894213\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5958231757717766\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.010265842415000807\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6048492625646708\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.44709469088954346\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6519208836034589\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02935004958434565\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6549036730589706\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008409772327739214\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46408160756535555\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.06293352071774021\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8109228398356279\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.07793242644447558\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7584180143057936\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026310874720886713\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.558211496937136\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08347093214740256\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5965250529467312\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0014046117929324595\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7954706355377502\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00011434628664393109\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5369332475629957\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.42565067607809615\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5271316674566886\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.4127768910864886\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6378279945563103\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09680960403249939\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.5_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.5_num-causals=1_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  48.10780  0.043274          1\n","1     HG01052  28.26730 -1.523244          1\n","2     HG01064   6.54305  0.019412          1\n","3     HG01069  18.02250 -1.306138          1\n","4     HG01079  47.38230 -0.103761          1\n","...       ...       ...       ...        ...\n","1095  NA19734   2.46970  0.470679         20\n","1096  NA19741  30.03390  0.285645         20\n","1097  NA19752 -37.52180 -0.186395         20\n","1098  NA19759  48.93210  0.411421         20\n","1099  NA19770  -6.97923  0.825756         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.5            1          1  0.707093   \n","1                         AMR  -0.25  0.5            1          2  0.629935   \n","2                         AMR  -0.25  0.5            1          3  0.848251   \n","3                         AMR  -0.25  0.5            1          4  0.686068   \n","4                         AMR  -0.25  0.5            1          5  0.816008   \n","5                         AMR  -0.25  0.5            1          6  0.717660   \n","6                         AMR  -0.25  0.5            1          7  0.585512   \n","7                         AMR  -0.25  0.5            1          8  0.595823   \n","8                         AMR  -0.25  0.5            1          9  0.604849   \n","9                         AMR  -0.25  0.5            1         10  0.651921   \n","10                        AMR  -0.25  0.5            1         11  0.654904   \n","11                        AMR  -0.25  0.5            1         12  0.464082   \n","12                        AMR  -0.25  0.5            1         13  0.810923   \n","13                        AMR  -0.25  0.5            1         14  0.758418   \n","14                        AMR  -0.25  0.5            1         15  0.558211   \n","15                        AMR  -0.25  0.5            1         16  0.596525   \n","16                        AMR  -0.25  0.5            1         17  0.795471   \n","17                        AMR  -0.25  0.5            1         18  0.536933   \n","18                        AMR  -0.25  0.5            1         19  0.527132   \n","19                        AMR  -0.25  0.5            1         20  0.637828   \n","\n","    val          test  \n","0    -1  1.880950e-02  \n","1    -1  4.030643e-07  \n","2    -1  3.677619e-05  \n","3    -1  4.872294e-02  \n","4    -1  1.020537e-04  \n","5    -1  1.542554e-02  \n","6    -1  2.636085e-02  \n","7    -1  1.026584e-02  \n","8    -1  4.470947e-01  \n","9    -1  2.935005e-02  \n","10   -1  8.409772e-03  \n","11   -1  6.293352e-02  \n","12   -1  7.793243e-02  \n","13   -1  2.631087e-02  \n","14   -1  8.347093e-02  \n","15   -1  1.404612e-03  \n","16   -1  1.143463e-04  \n","17   -1  4.256507e-01  \n","18   -1  4.127769e-01  \n","19   -1  9.680960e-02  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5252103836148755\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.03239156604735146\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45286963727447255\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023956012336215356\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8777789533288114\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=6.196426212322446e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6784077199901093\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0034387337386930015\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141856 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141856 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5421613451066415\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.184389367016112\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5996888206849642\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.01269968554972993\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7964748422367757\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.004868070640409753\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8885749345129914\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0249547647061068\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.907048196360265\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0017294306494379194\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4348405688819575\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.24154782583273313\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7944862878176296\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.028401377853631008\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.43119402892385184\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.18041625877287235\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5819153684421913\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.3455660693112658\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5268726844851386\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.13817061115807258\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6099662794153148\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0013118476765203793\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45829914378016057\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0035590650146256966\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7447588801896969\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09544440374892948\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5396533114458053\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.47962751847815455\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8855942313855398\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006343701974803434\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8263806689957258\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06066634041700289\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.5_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.5_num-causals=10_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   1.296260 -0.838049          1\n","1     HG01052   2.879990 -0.119381          1\n","2     HG01064   2.752920  1.534319          1\n","3     HG01069   0.175173 -1.208327          1\n","4     HG01079   3.945660  0.098076          1\n","...       ...        ...       ...        ...\n","1095  NA19734   4.463010 -0.320091         20\n","1096  NA19741   3.034930 -0.406117         20\n","1097  NA19752  18.617900 -0.009293         20\n","1098  NA19759  -2.954260 -1.161096         20\n","1099  NA19770  10.056300  0.566767         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.5           10          1  0.525210   \n","1                         AMR  -0.25  0.5           10          2  0.452870   \n","2                         AMR  -0.25  0.5           10          3  0.877779   \n","3                         AMR  -0.25  0.5           10          4  0.678408   \n","4                         AMR  -0.25  0.5           10          5  0.542161   \n","5                         AMR  -0.25  0.5           10          6  0.599689   \n","6                         AMR  -0.25  0.5           10          7  0.796475   \n","7                         AMR  -0.25  0.5           10          8  0.888575   \n","8                         AMR  -0.25  0.5           10          9  0.907048   \n","9                         AMR  -0.25  0.5           10         10  0.434841   \n","10                        AMR  -0.25  0.5           10         11  0.794486   \n","11                        AMR  -0.25  0.5           10         12  0.431194   \n","12                        AMR  -0.25  0.5           10         13  0.581915   \n","13                        AMR  -0.25  0.5           10         14  0.526873   \n","14                        AMR  -0.25  0.5           10         15  0.609966   \n","15                        AMR  -0.25  0.5           10         16  0.458299   \n","16                        AMR  -0.25  0.5           10         17  0.744759   \n","17                        AMR  -0.25  0.5           10         18  0.539653   \n","18                        AMR  -0.25  0.5           10         19  0.885594   \n","19                        AMR  -0.25  0.5           10         20  0.826381   \n","\n","    val      test  \n","0    -1  0.032392  \n","1    -1  0.023956  \n","2    -1  0.000062  \n","3    -1  0.003439  \n","4    -1  0.184389  \n","5    -1  0.012700  \n","6    -1  0.004868  \n","7    -1  0.024955  \n","8    -1  0.001729  \n","9    -1  0.241548  \n","10   -1  0.028401  \n","11   -1  0.180416  \n","12   -1  0.345566  \n","13   -1  0.138171  \n","14   -1  0.001312  \n","15   -1  0.003559  \n","16   -1  0.095444  \n","17   -1  0.479628  \n","18   -1  0.006344  \n","19   -1  0.060666  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6440697571302457\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008077733303095273\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8998316458964223\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031769644702519\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7697059772099109\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.16412838448264522\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4688529374306313\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09673916775584973\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9126020417984377\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0867048136174586\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8096280323352039\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0019768250840964444\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6858120166939264\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0011838271638929805\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9011023235683161\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0015744676381286803\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8272101741072172\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007958600129178093\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6118353967275598\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014849573481252166\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8614508838631025\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0014910955257157396\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4086541680213606\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.06139681066919294\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141838 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141838 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6105458649100189\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009697463125687758\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45588504421689874\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.16451802257897327\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.592885722095042\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08736423637635285\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9305225983381048\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02648941981895614\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6892027448567972\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.13416210530390474\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141812 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141812 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5951539154015998\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.037845195634109687\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.47917007437118664\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.03783267376640592\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7424112267032249\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03153266488078826\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.5_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.5_num-causals=100_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  41.44720  1.299897          1\n","1     HG01052  24.47360 -0.778169          1\n","2     HG01064  14.23160 -0.779213          1\n","3     HG01069  -3.76412 -0.556813          1\n","4     HG01079  38.69830  0.173218          1\n","...       ...       ...       ...        ...\n","1095  NA19734 -11.91090 -0.784978         20\n","1096  NA19741 -30.04030  0.832539         20\n","1097  NA19752  26.82390 -0.758902         20\n","1098  NA19759 -32.54430  0.068836         20\n","1099  NA19770   5.27097 -0.025141         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.5          100          1  0.644070   \n","1                         AMR  -0.25  0.5          100          2  0.899832   \n","2                         AMR  -0.25  0.5          100          3  0.769706   \n","3                         AMR  -0.25  0.5          100          4  0.468853   \n","4                         AMR  -0.25  0.5          100          5  0.912602   \n","5                         AMR  -0.25  0.5          100          6  0.809628   \n","6                         AMR  -0.25  0.5          100          7  0.685812   \n","7                         AMR  -0.25  0.5          100          8  0.901102   \n","8                         AMR  -0.25  0.5          100          9  0.827210   \n","9                         AMR  -0.25  0.5          100         10  0.611835   \n","10                        AMR  -0.25  0.5          100         11  0.861451   \n","11                        AMR  -0.25  0.5          100         12  0.408654   \n","12                        AMR  -0.25  0.5          100         13  0.610546   \n","13                        AMR  -0.25  0.5          100         14  0.455885   \n","14                        AMR  -0.25  0.5          100         15  0.592886   \n","15                        AMR  -0.25  0.5          100         16  0.930523   \n","16                        AMR  -0.25  0.5          100         17  0.689203   \n","17                        AMR  -0.25  0.5          100         18  0.595154   \n","18                        AMR  -0.25  0.5          100         19  0.479170   \n","19                        AMR  -0.25  0.5          100         20  0.742411   \n","\n","    val      test  \n","0    -1  0.000808  \n","1    -1  0.031770  \n","2    -1  0.164128  \n","3    -1  0.096739  \n","4    -1  0.086705  \n","5    -1  0.001977  \n","6    -1  0.001184  \n","7    -1  0.001574  \n","8    -1  0.007959  \n","9    -1  0.014850  \n","10   -1  0.001491  \n","11   -1  0.061397  \n","12   -1  0.000970  \n","13   -1  0.164518  \n","14   -1  0.087364  \n","15   -1  0.026489  \n","16   -1  0.134162  \n","17   -1  0.037845  \n","18   -1  0.037833  \n","19   -1  0.031533  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6410508263838188\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.051219685284794064\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46980240009395857\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010625963099661486\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9111771450555576\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06675418107533349\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141721 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141721 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6122106440324778\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.040678312624645975\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8972031825710348\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09337645091207583\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5576638839914956\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0121034780557812\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141847 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141847 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5027096603999794\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.005524771162041805\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8070959090088993\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0028332158671433745\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4796512620816351\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0005012386036410245\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8720807591908194\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.029734480959748934\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.822367003258614\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0161619673510226\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7179544259350634\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.007683942993477178\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6509813854433073\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.13097019452943548\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4663273609351101\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.060830739548979246\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6553314820947006\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04342278407089681\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6675773573397541\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03644698865609068\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.756923397840594\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007362291596639729\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6765376278138936\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00986421076700352\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6475716431786191\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005826949414127831\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141852 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141852 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5528284123546345\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004986539219648309\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.5_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.5_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  37.01050  0.590774          1\n","1     HG01052  26.46340 -1.305040          1\n","2     HG01064   4.28288 -1.280357          1\n","3     HG01069  19.69710  0.973132          1\n","4     HG01079  40.89860  0.586098          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -6.07672 -0.881599         20\n","1096  NA19741 -27.54240  0.202982         20\n","1097  NA19752  48.15660  0.883424         20\n","1098  NA19759 -47.26210 -0.294477         20\n","1099  NA19770  39.94960 -0.319164         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.5          250          1  0.641051   \n","1                         AMR  -0.25  0.5          250          2  0.469802   \n","2                         AMR  -0.25  0.5          250          3  0.911177   \n","3                         AMR  -0.25  0.5          250          4  0.612211   \n","4                         AMR  -0.25  0.5          250          5  0.897203   \n","5                         AMR  -0.25  0.5          250          6  0.557664   \n","6                         AMR  -0.25  0.5          250          7  0.502710   \n","7                         AMR  -0.25  0.5          250          8  0.807096   \n","8                         AMR  -0.25  0.5          250          9  0.479651   \n","9                         AMR  -0.25  0.5          250         10  0.872081   \n","10                        AMR  -0.25  0.5          250         11  0.822367   \n","11                        AMR  -0.25  0.5          250         12  0.717954   \n","12                        AMR  -0.25  0.5          250         13  0.650981   \n","13                        AMR  -0.25  0.5          250         14  0.466327   \n","14                        AMR  -0.25  0.5          250         15  0.655331   \n","15                        AMR  -0.25  0.5          250         16  0.667577   \n","16                        AMR  -0.25  0.5          250         17  0.756923   \n","17                        AMR  -0.25  0.5          250         18  0.676538   \n","18                        AMR  -0.25  0.5          250         19  0.647572   \n","19                        AMR  -0.25  0.5          250         20  0.552828   \n","\n","    val      test  \n","0    -1  0.051220  \n","1    -1  0.010626  \n","2    -1  0.066754  \n","3    -1  0.040678  \n","4    -1  0.093376  \n","5    -1  0.012103  \n","6    -1  0.005525  \n","7    -1  0.002833  \n","8    -1  0.000501  \n","9    -1  0.029734  \n","10   -1  0.016162  \n","11   -1  0.007684  \n","12   -1  0.130970  \n","13   -1  0.060831  \n","14   -1  0.043423  \n","15   -1  0.036447  \n","16   -1  0.007362  \n","17   -1  0.009864  \n","18   -1  0.005827  \n","19   -1  0.004987  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.715541170596821\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005760268106021339\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.85747453897407\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009960178473042976\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7240750837763272\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06889976968159191\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5851522320081906\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0058115613928926505\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5222024651580652\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024363893910535107\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6437202230475779\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032789190384676314\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5885474605755736\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0004237951676699694\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7655991937202749\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004449103780000135\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6200095408938873\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.02755209256136454\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9031694073076709\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032765474754993993\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7934743567496426\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031193600905577795\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5675440666683109\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03308657425853773\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6334551967278641\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025864134323096342\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.602548086209814\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05820360481353557\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5404146070471093\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.05114310498324304\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.696744931592007\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07557532337716405\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6254252103275806\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11494537358382441\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8923492685517871\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03098977071232555\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48794482995558486\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07939782824061885\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5614362468066564\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.060309595402998364\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.5_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.5_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  32.86920 -0.566430          1\n","1     HG01052   6.42959 -1.812823          1\n","2     HG01064  -1.80193 -0.454080          1\n","3     HG01069  10.52220 -1.916545          1\n","4     HG01079  22.17530  0.740419          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -4.85296 -1.618708         20\n","1096  NA19741 -38.41510 -0.031226         20\n","1097  NA19752  47.12110  0.664967         20\n","1098  NA19759 -47.61760 -0.783847         20\n","1099  NA19770  35.22910  0.171587         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.5          512          1  0.715541   \n","1                         AMR  -0.25  0.5          512          2  0.857475   \n","2                         AMR  -0.25  0.5          512          3  0.724075   \n","3                         AMR  -0.25  0.5          512          4  0.585152   \n","4                         AMR  -0.25  0.5          512          5  0.522202   \n","5                         AMR  -0.25  0.5          512          6  0.643720   \n","6                         AMR  -0.25  0.5          512          7  0.588547   \n","7                         AMR  -0.25  0.5          512          8  0.765599   \n","8                         AMR  -0.25  0.5          512          9  0.620010   \n","9                         AMR  -0.25  0.5          512         10  0.903169   \n","10                        AMR  -0.25  0.5          512         11  0.793474   \n","11                        AMR  -0.25  0.5          512         12  0.567544   \n","12                        AMR  -0.25  0.5          512         13  0.633455   \n","13                        AMR  -0.25  0.5          512         14  0.602548   \n","14                        AMR  -0.25  0.5          512         15  0.540415   \n","15                        AMR  -0.25  0.5          512         16  0.696745   \n","16                        AMR  -0.25  0.5          512         17  0.625425   \n","17                        AMR  -0.25  0.5          512         18  0.892349   \n","18                        AMR  -0.25  0.5          512         19  0.487945   \n","19                        AMR  -0.25  0.5          512         20  0.561436   \n","\n","    val      test  \n","0    -1  0.005760  \n","1    -1  0.000996  \n","2    -1  0.068900  \n","3    -1  0.005812  \n","4    -1  0.024364  \n","5    -1  0.003279  \n","6    -1  0.000424  \n","7    -1  0.004449  \n","8    -1  0.027552  \n","9    -1  0.003277  \n","10   -1  0.031194  \n","11   -1  0.033087  \n","12   -1  0.025864  \n","13   -1  0.058204  \n","14   -1  0.051143  \n","15   -1  0.075575  \n","16   -1  0.114945  \n","17   -1  0.030990  \n","18   -1  0.079398  \n","19   -1  0.060310  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5894731864859601\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010878577128067475\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5240323371814034\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008964877418608122\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6330121430280928\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06749382818794357\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6141009918600309\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006602316261541321\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6632990118627323\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0034408039886083845\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.657044987843951\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7446448455725078\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.586057832030568\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.5657063858914985\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8785507821047387\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00023832216620031662\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6853908567731899\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.45578679698331925\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5958197527866177\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.013110133895792785\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9003130282494517\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00912043498510622\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141854 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141854 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4793182553023048\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.2499158154787134\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8427872703411949\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8972929258108205\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.673801065615654\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014959557003856153\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.599818899892432\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6195718531094194\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141825 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141825 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48957551369492136\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009115773566940222\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7060222806584695\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08122458040551263\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6160828001203519\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.43366888946829835\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6374300557119835\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.3318987282106831\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6170498120102\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7372210812328021\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.7_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.7_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  56.650800  0.593505          1\n","1     HG01052  46.475400  0.233370          1\n","2     HG01064  30.230400  1.353036          1\n","3     HG01069  10.571400 -0.284351          1\n","4     HG01079  57.268200  0.490792          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.000000  0.589980         20\n","1096  NA19741   0.000000  1.257316         20\n","1097  NA19752  -2.724440 -1.602550         20\n","1098  NA19759  -0.441678  0.295498         20\n","1099  NA19770  -0.441678  0.423695         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.7            1          1  0.589473   \n","1                         AMR  -0.25  0.7            1          2  0.524032   \n","2                         AMR  -0.25  0.7            1          3  0.633012   \n","3                         AMR  -0.25  0.7            1          4  0.614101   \n","4                         AMR  -0.25  0.7            1          5  0.663299   \n","5                         AMR  -0.25  0.7            1          6  0.657045   \n","6                         AMR  -0.25  0.7            1          7  0.586058   \n","7                         AMR  -0.25  0.7            1          8  0.878551   \n","8                         AMR  -0.25  0.7            1          9  0.685391   \n","9                         AMR  -0.25  0.7            1         10  0.595820   \n","10                        AMR  -0.25  0.7            1         11  0.900313   \n","11                        AMR  -0.25  0.7            1         12  0.479318   \n","12                        AMR  -0.25  0.7            1         13  0.842787   \n","13                        AMR  -0.25  0.7            1         14  0.673801   \n","14                        AMR  -0.25  0.7            1         15  0.599819   \n","15                        AMR  -0.25  0.7            1         16  0.489576   \n","16                        AMR  -0.25  0.7            1         17  0.706022   \n","17                        AMR  -0.25  0.7            1         18  0.616083   \n","18                        AMR  -0.25  0.7            1         19  0.637430   \n","19                        AMR  -0.25  0.7            1         20  0.617050   \n","\n","    val      test  \n","0    -1  0.010879  \n","1    -1  0.000896  \n","2    -1  0.067494  \n","3    -1  0.006602  \n","4    -1  0.003441  \n","5    -1  0.744645  \n","6    -1  0.565706  \n","7    -1  0.000238  \n","8    -1  0.455787  \n","9    -1  0.013110  \n","10   -1  0.009120  \n","11   -1  0.249916  \n","12   -1  0.897293  \n","13   -1  0.014960  \n","14   -1  0.619572  \n","15   -1  0.000912  \n","16   -1  0.081225  \n","17   -1  0.433669  \n","18   -1  0.331899  \n","19   -1  0.737221  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5048150412747155\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.15420523600745192\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141828 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141828 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4366116078619864\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.025454273676344015\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8233009165759769\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014755286952505187\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7451594621056018\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.756429976741842e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6503931267368304\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.45050913404789045\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6345812339999908\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.012672725754245677\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9021834774045245\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01614896472567015\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.878950160245014\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005404418722561172\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6839276530774862\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04410092589437436\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5431768653804034\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1941863575450005\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8425140725641262\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.811610086730654e-07\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141834 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.42415220131006526\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.1525251126730865\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6485371826990086\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4482054151645393\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141797 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5357522353103172\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.3990274595430096\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8848065650448167\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08229309919461655\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.42211056938757663\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07917523807193583\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8895896176421428\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09056757970552393\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.654385021380019\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.55952371889785\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141830 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141830 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8612229586915107\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09931068264920119\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8649099729998707\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0007757650235715473\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.7_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.7_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732   1.99884  0.444920          1\n","1     HG01052   1.87222 -0.670996          1\n","2     HG01064   1.19091  0.625901          1\n","3     HG01069  -1.70326  0.076555          1\n","4     HG01079   2.93276 -1.153797          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -4.19529  0.485648         20\n","1096  NA19741  12.83090 -0.230099         20\n","1097  NA19752 -13.07830  0.243584         20\n","1098  NA19759  10.91340  0.066414         20\n","1099  NA19770   9.12625  0.353253         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.7           10          1  0.504815   \n","1                         AMR  -0.25  0.7           10          2  0.436612   \n","2                         AMR  -0.25  0.7           10          3  0.823301   \n","3                         AMR  -0.25  0.7           10          4  0.745159   \n","4                         AMR  -0.25  0.7           10          5  0.650393   \n","5                         AMR  -0.25  0.7           10          6  0.634581   \n","6                         AMR  -0.25  0.7           10          7  0.902183   \n","7                         AMR  -0.25  0.7           10          8  0.878950   \n","8                         AMR  -0.25  0.7           10          9  0.683928   \n","9                         AMR  -0.25  0.7           10         10  0.543177   \n","10                        AMR  -0.25  0.7           10         11  0.842514   \n","11                        AMR  -0.25  0.7           10         12  0.424152   \n","12                        AMR  -0.25  0.7           10         13  0.648537   \n","13                        AMR  -0.25  0.7           10         14  0.535752   \n","14                        AMR  -0.25  0.7           10         15  0.884807   \n","15                        AMR  -0.25  0.7           10         16  0.422111   \n","16                        AMR  -0.25  0.7           10         17  0.889590   \n","17                        AMR  -0.25  0.7           10         18  0.654385   \n","18                        AMR  -0.25  0.7           10         19  0.861223   \n","19                        AMR  -0.25  0.7           10         20  0.864910   \n","\n","    val          test  \n","0    -1  1.542052e-01  \n","1    -1  2.545427e-02  \n","2    -1  1.475529e-02  \n","3    -1  1.756430e-05  \n","4    -1  4.505091e-01  \n","5    -1  1.267273e-02  \n","6    -1  1.614896e-02  \n","7    -1  5.404419e-03  \n","8    -1  4.410093e-02  \n","9    -1  1.941864e-01  \n","10   -1  3.811610e-07  \n","11   -1  1.525251e-01  \n","12   -1  4.482054e-01  \n","13   -1  3.990275e-01  \n","14   -1  8.229310e-02  \n","15   -1  7.917524e-02  \n","16   -1  9.056758e-02  \n","17   -1  5.595237e-01  \n","18   -1  9.931068e-02  \n","19   -1  7.757650e-04  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6200294232293141\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031656541497841986\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8667656634342737\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.26871139440867786\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5644432653846195\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08985148593909666\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5594275893473397\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030155167913527185\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6657768557670035\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.032697485515908696\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6786105815493457\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006458372517905315\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8176002182847217\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03579316500511854\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7623678676379926\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.019480057085633013\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6285127438322029\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.057866920761164176\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141700 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141700 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7273273757002782\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0031793691015121754\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8698346593318772\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005008819582294394\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141872 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141872 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.41076062915313494\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.21591115307958278\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141725 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141725 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6924992014270718\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.043116962473704286\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5738539748976049\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4904856119409899\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5025128562572304\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.180355316763543\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9230847989107122\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03912397251631714\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6390427289681028\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0647757784541331\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7765058467813278\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0042966461546176836\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46044532724312587\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09231823684013077\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6319388354590101\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08940956616237294\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.7_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.7_num-causals=100_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  41.54220  1.859904          1\n","1     HG01052  18.69490 -0.836597          1\n","2     HG01064   9.42469 -0.554533          1\n","3     HG01069   7.20891 -0.612246          1\n","4     HG01079  45.90960  0.174492          1\n","...       ...       ...       ...        ...\n","1095  NA19734   5.27969  0.155001         20\n","1096  NA19741 -37.40550  0.804489         20\n","1097  NA19752  58.17290 -0.505531         20\n","1098  NA19759 -48.11280 -0.919567         20\n","1099  NA19770  33.14440  0.144056         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.7          100          1  0.620029   \n","1                         AMR  -0.25  0.7          100          2  0.866766   \n","2                         AMR  -0.25  0.7          100          3  0.564443   \n","3                         AMR  -0.25  0.7          100          4  0.559428   \n","4                         AMR  -0.25  0.7          100          5  0.665777   \n","5                         AMR  -0.25  0.7          100          6  0.678611   \n","6                         AMR  -0.25  0.7          100          7  0.817600   \n","7                         AMR  -0.25  0.7          100          8  0.762368   \n","8                         AMR  -0.25  0.7          100          9  0.628513   \n","9                         AMR  -0.25  0.7          100         10  0.727327   \n","10                        AMR  -0.25  0.7          100         11  0.869835   \n","11                        AMR  -0.25  0.7          100         12  0.410761   \n","12                        AMR  -0.25  0.7          100         13  0.692499   \n","13                        AMR  -0.25  0.7          100         14  0.573854   \n","14                        AMR  -0.25  0.7          100         15  0.502513   \n","15                        AMR  -0.25  0.7          100         16  0.923085   \n","16                        AMR  -0.25  0.7          100         17  0.639043   \n","17                        AMR  -0.25  0.7          100         18  0.776506   \n","18                        AMR  -0.25  0.7          100         19  0.460445   \n","19                        AMR  -0.25  0.7          100         20  0.631939   \n","\n","    val      test  \n","0    -1  0.031657  \n","1    -1  0.268711  \n","2    -1  0.089851  \n","3    -1  0.030155  \n","4    -1  0.032697  \n","5    -1  0.006458  \n","6    -1  0.035793  \n","7    -1  0.019480  \n","8    -1  0.057867  \n","9    -1  0.003179  \n","10   -1  0.005009  \n","11   -1  0.215911  \n","12   -1  0.043117  \n","13   -1  0.490486  \n","14   -1  0.180355  \n","15   -1  0.039124  \n","16   -1  0.064776  \n","17   -1  0.004297  \n","18   -1  0.092318  \n","19   -1  0.089410  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5845422353300751\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.01487787389399348\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5516206112634627\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07041448616844179\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8900693981629213\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03558926896939568\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.47389277568322696\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008363604282134514\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8302165312404275\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03358244591684977\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5684105810613067\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0737073463530113\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5578918748442615\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1031839026888295\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.700558722808918\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08042591593128064\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7280113123548388\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06968228023962166\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8347162146538988\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.029598455471637572\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8809996222595975\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023517258807393235\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.47944107879471104\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014539697147908011\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6065036607888521\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010756691593628466\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6522762606569441\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07087504982998089\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6055504455802628\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11329102061091412\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5466729430603593\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.025845785710006775\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.876976627761094\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11859767728562992\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5565549645176017\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.023062131606041557\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5209695598379994\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.04162488744193468\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6448646227034929\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0068798066286867915\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.7_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.7_num-causals=250_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   1.942290  0.628898          1\n","1     HG01052   1.618600 -0.865945          1\n","2     HG01064  -1.550670 -2.041939          1\n","3     HG01069  -1.947700  0.218634          1\n","4     HG01079   1.209380  0.792814          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.437756  0.261949         20\n","1096  NA19741 -36.147800  0.085458         20\n","1097  NA19752  44.258200  0.126946         20\n","1098  NA19759 -31.420000 -0.204031         20\n","1099  NA19770  31.183000  0.145765         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.7          250          1  0.584542   \n","1                         AMR  -0.25  0.7          250          2  0.551621   \n","2                         AMR  -0.25  0.7          250          3  0.890069   \n","3                         AMR  -0.25  0.7          250          4  0.473893   \n","4                         AMR  -0.25  0.7          250          5  0.830217   \n","5                         AMR  -0.25  0.7          250          6  0.568411   \n","6                         AMR  -0.25  0.7          250          7  0.557892   \n","7                         AMR  -0.25  0.7          250          8  0.700559   \n","8                         AMR  -0.25  0.7          250          9  0.728011   \n","9                         AMR  -0.25  0.7          250         10  0.834716   \n","10                        AMR  -0.25  0.7          250         11  0.881000   \n","11                        AMR  -0.25  0.7          250         12  0.479441   \n","12                        AMR  -0.25  0.7          250         13  0.606504   \n","13                        AMR  -0.25  0.7          250         14  0.652276   \n","14                        AMR  -0.25  0.7          250         15  0.605550   \n","15                        AMR  -0.25  0.7          250         16  0.546673   \n","16                        AMR  -0.25  0.7          250         17  0.876977   \n","17                        AMR  -0.25  0.7          250         18  0.556555   \n","18                        AMR  -0.25  0.7          250         19  0.520970   \n","19                        AMR  -0.25  0.7          250         20  0.644865   \n","\n","    val      test  \n","0    -1  0.014878  \n","1    -1  0.070414  \n","2    -1  0.035589  \n","3    -1  0.000836  \n","4    -1  0.033582  \n","5    -1  0.073707  \n","6    -1  0.103184  \n","7    -1  0.080426  \n","8    -1  0.069682  \n","9    -1  0.029598  \n","10   -1  0.023517  \n","11   -1  0.014540  \n","12   -1  0.010757  \n","13   -1  0.070875  \n","14   -1  0.113291  \n","15   -1  0.025846  \n","16   -1  0.118598  \n","17   -1  0.023062  \n","18   -1  0.041625  \n","19   -1  0.006880  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5245532723900539\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.10683426881595927\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7142660318696785\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008491759814734353\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5097207517765684\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03855904480820265\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5639242696278675\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01751065071564903\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5364030005502461\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04540535107139472\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5409095863378461\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017309333883633298\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7221006604147033\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01826121929837037\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6148789404402945\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0003039395747252103\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5346040389228439\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02350829984037832\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48365179622990906\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0006953445457281316\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5245072775886525\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04687693811093941\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7015133547026222\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06273257917853146\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.544117611849741\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02448321392156085\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5588809018010025\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03139198970299615\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4669484038135029\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03133720424907548\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7023471787962373\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.036050030774309946\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6509387713855954\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016287350002297273\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5985473781048868\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.12498403990101188\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6132408164481195\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07132299100463321\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6387452067903565\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07664658482438336\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.7_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.7_num-causals=512_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   5.320440 -0.268823          1\n","1     HG01052   0.782888 -1.796597          1\n","2     HG01064  -0.014163  0.162594          1\n","3     HG01069  -0.153712 -0.401727          1\n","4     HG01079  11.208600 -0.198962          1\n","...       ...        ...       ...        ...\n","1095  NA19734 -22.498600 -1.070969         20\n","1096  NA19741 -37.927700  0.644665         20\n","1097  NA19752  40.683200  0.163813         20\n","1098  NA19759 -44.185600  0.509841         20\n","1099  NA19770  15.794700  0.282698         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.7          512          1  0.524553   \n","1                         AMR  -0.25  0.7          512          2  0.714266   \n","2                         AMR  -0.25  0.7          512          3  0.509721   \n","3                         AMR  -0.25  0.7          512          4  0.563924   \n","4                         AMR  -0.25  0.7          512          5  0.536403   \n","5                         AMR  -0.25  0.7          512          6  0.540910   \n","6                         AMR  -0.25  0.7          512          7  0.722101   \n","7                         AMR  -0.25  0.7          512          8  0.614879   \n","8                         AMR  -0.25  0.7          512          9  0.534604   \n","9                         AMR  -0.25  0.7          512         10  0.483652   \n","10                        AMR  -0.25  0.7          512         11  0.524507   \n","11                        AMR  -0.25  0.7          512         12  0.701513   \n","12                        AMR  -0.25  0.7          512         13  0.544118   \n","13                        AMR  -0.25  0.7          512         14  0.558881   \n","14                        AMR  -0.25  0.7          512         15  0.466948   \n","15                        AMR  -0.25  0.7          512         16  0.702347   \n","16                        AMR  -0.25  0.7          512         17  0.650939   \n","17                        AMR  -0.25  0.7          512         18  0.598547   \n","18                        AMR  -0.25  0.7          512         19  0.613241   \n","19                        AMR  -0.25  0.7          512         20  0.638745   \n","\n","    val      test  \n","0    -1  0.106834  \n","1    -1  0.008492  \n","2    -1  0.038559  \n","3    -1  0.017511  \n","4    -1  0.045405  \n","5    -1  0.017309  \n","6    -1  0.018261  \n","7    -1  0.000304  \n","8    -1  0.023508  \n","9    -1  0.000695  \n","10   -1  0.046877  \n","11   -1  0.062733  \n","12   -1  0.024483  \n","13   -1  0.031392  \n","14   -1  0.031337  \n","15   -1  0.036050  \n","16   -1  0.016287  \n","17   -1  0.124984  \n","18   -1  0.071323  \n","19   -1  0.076647  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7210726016866733\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009455948744381734\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4994135020992902\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0010734520001108943\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6469205504353215\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00862794053540095\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6552701601783846\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09063154532741756\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6751971049133316\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06509794583051849\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7632017545867944\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7929627839175889\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141820 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141820 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7137670427494663\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6873130603756894\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.786454125142069\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03960225652996597\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7283469324571703\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.696271142382178\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7073520185925664\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8603561523416724\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5676718159728606\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0037038287103055853\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141850 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141850 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5244917738134222\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.24097031101474872\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9501296777742821\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.9664012869158347\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141825 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141825 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9231701610277461\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024931711853032133\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141828 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141828 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7179879715494839\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6425222689949626\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141870 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141870 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5844015802790995\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.1974544467573044\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7987501180972072\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7704059811877686\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6872803397180592\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.43147526072086867\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6933727785487882\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.5410991271529368\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6768299662889076\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7164600424068804\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.9_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.9_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  50.323000 -0.798145          1\n","1     HG01052  41.863500  0.203646          1\n","2     HG01064  29.144400  0.015730          1\n","3     HG01069  23.432300 -0.008074          1\n","4     HG01079  54.311500 -0.278244          1\n","...       ...        ...       ...        ...\n","1095  NA19734   1.283670  1.693201         20\n","1096  NA19741   0.842389  1.123548         20\n","1097  NA19752  -2.282860 -2.219292         20\n","1098  NA19759   0.414829  1.058637         20\n","1099  NA19770   1.283920  1.007270         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.9            1          1  0.721073   \n","1                         AMR  -0.25  0.9            1          2  0.499414   \n","2                         AMR  -0.25  0.9            1          3  0.646921   \n","3                         AMR  -0.25  0.9            1          4  0.655270   \n","4                         AMR  -0.25  0.9            1          5  0.675197   \n","5                         AMR  -0.25  0.9            1          6  0.763202   \n","6                         AMR  -0.25  0.9            1          7  0.713767   \n","7                         AMR  -0.25  0.9            1          8  0.786454   \n","8                         AMR  -0.25  0.9            1          9  0.728347   \n","9                         AMR  -0.25  0.9            1         10  0.707352   \n","10                        AMR  -0.25  0.9            1         11  0.567672   \n","11                        AMR  -0.25  0.9            1         12  0.524492   \n","12                        AMR  -0.25  0.9            1         13  0.950130   \n","13                        AMR  -0.25  0.9            1         14  0.923170   \n","14                        AMR  -0.25  0.9            1         15  0.717988   \n","15                        AMR  -0.25  0.9            1         16  0.584402   \n","16                        AMR  -0.25  0.9            1         17  0.798750   \n","17                        AMR  -0.25  0.9            1         18  0.687280   \n","18                        AMR  -0.25  0.9            1         19  0.693373   \n","19                        AMR  -0.25  0.9            1         20  0.676830   \n","\n","    val      test  \n","0    -1  0.000946  \n","1    -1  0.001073  \n","2    -1  0.008628  \n","3    -1  0.090632  \n","4    -1  0.065098  \n","5    -1  0.792963  \n","6    -1  0.687313  \n","7    -1  0.039602  \n","8    -1  0.696271  \n","9    -1  0.860356  \n","10   -1  0.003704  \n","11   -1  0.240970  \n","12   -1  0.966401  \n","13   -1  0.024932  \n","14   -1  0.642522  \n","15   -1  0.197454  \n","16   -1  0.770406  \n","17   -1  0.431475  \n","18   -1  0.541099  \n","19   -1  0.716460  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5308311517197223\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.5219109573385591\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.44727738899684066\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.11790437410142085\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8736946404942879\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07174661290806443\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.777431624794215\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.840348799851159e-11\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141842 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141842 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6826497473614865\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.462200530328435\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6903254383184418\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05596131934322383\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9141485328845648\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00046145228163104845\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8504550045495846\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.16359990021541074\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6888244789048837\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.6469047991108768\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6058708855524788\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8283110811792441\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6438845178100036\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.12007010979444198\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46198881292594624\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.18952318305645302\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7282430116865047\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6984657491315482\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141853 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141853 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6742215613697403\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.3884450260961377\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7072041649933438\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01162429917857735\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141845 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141845 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4654025755771073\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.13104335034812445\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8859459361033402\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02000272257275139\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.817054826151977\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8302284529723069\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.848809158191522\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.9123505411901167\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9376548370615153\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005564571079656949\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.9_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.9_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  0.808846 -0.149965          1\n","1     HG01052  1.205450 -0.724920          1\n","2     HG01064  2.192900  0.181330          1\n","3     HG01069  0.808846 -1.216972          1\n","4     HG01079  1.205450 -0.876219          1\n","...       ...       ...       ...        ...\n","1095  NA19734  2.168440  0.334620         20\n","1096  NA19741  0.333124  0.179203         20\n","1097  NA19752 -4.253740  0.567723         20\n","1098  NA19759  0.861876 -0.329925         20\n","1099  NA19770  9.314320  0.553430         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.9           10          1  0.530831   \n","1                         AMR  -0.25  0.9           10          2  0.447277   \n","2                         AMR  -0.25  0.9           10          3  0.873695   \n","3                         AMR  -0.25  0.9           10          4  0.777432   \n","4                         AMR  -0.25  0.9           10          5  0.682650   \n","5                         AMR  -0.25  0.9           10          6  0.690325   \n","6                         AMR  -0.25  0.9           10          7  0.914149   \n","7                         AMR  -0.25  0.9           10          8  0.850455   \n","8                         AMR  -0.25  0.9           10          9  0.688824   \n","9                         AMR  -0.25  0.9           10         10  0.605871   \n","10                        AMR  -0.25  0.9           10         11  0.643885   \n","11                        AMR  -0.25  0.9           10         12  0.461989   \n","12                        AMR  -0.25  0.9           10         13  0.728243   \n","13                        AMR  -0.25  0.9           10         14  0.674222   \n","14                        AMR  -0.25  0.9           10         15  0.707204   \n","15                        AMR  -0.25  0.9           10         16  0.465403   \n","16                        AMR  -0.25  0.9           10         17  0.885946   \n","17                        AMR  -0.25  0.9           10         18  0.817055   \n","18                        AMR  -0.25  0.9           10         19  0.848809   \n","19                        AMR  -0.25  0.9           10         20  0.937655   \n","\n","    val          test  \n","0    -1  5.219110e-01  \n","1    -1  1.179044e-01  \n","2    -1  7.174661e-02  \n","3    -1  5.840349e-11  \n","4    -1  4.622005e-01  \n","5    -1  5.596132e-02  \n","6    -1  4.614523e-04  \n","7    -1  1.635999e-01  \n","8    -1  6.469048e-01  \n","9    -1  8.283111e-01  \n","10   -1  1.200701e-01  \n","11   -1  1.895232e-01  \n","12   -1  6.984657e-01  \n","13   -1  3.884450e-01  \n","14   -1  1.162430e-02  \n","15   -1  1.310434e-01  \n","16   -1  2.000272e-02  \n","17   -1  8.302285e-01  \n","18   -1  9.123505e-01  \n","19   -1  5.564571e-03  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6581703587838622\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05100247869394709\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9175489536563873\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.27513127562478557\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6695419873628747\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.6639772405328798\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5155291266308324\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.018840894738341326\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8341793849283982\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11956174805156113\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7080656258094902\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=9.190130691553059e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6443945145258104\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004947625512718636\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8746063374202836\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05523501454481095\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7545949083165375\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06383800655671582\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141712 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141712 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5979663597472802\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.21859242542957422\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7867818424039417\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025450263986310884\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141832 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141832 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4180584360387766\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.18756074012659904\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.858606801482165\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.14133444126862732\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.639613326281975\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.5170422046442632\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5552469231143972\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.2889886651575803\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9266751947639025\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014146183022729134\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141832 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141832 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5469233947391399\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05426308567672185\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6777692091173805\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.012350084559683632\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.51729183352522\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1527710500959211\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8670607981487505\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08101910236415996\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.9_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.9_num-causals=100_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  35.234000  2.530688          1\n","1     HG01052  22.527200 -0.135487          1\n","2     HG01064   8.302560  0.135626          1\n","3     HG01069  -3.751870 -0.779672          1\n","4     HG01079  33.149800  1.305204          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.356271 -0.441415         20\n","1096  NA19741 -17.854300  0.750976         20\n","1097  NA19752  17.034100 -0.984733         20\n","1098  NA19759 -15.833500 -0.847033         20\n","1099  NA19770   7.878430  0.235735         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.9          100          1  0.658170   \n","1                         AMR  -0.25  0.9          100          2  0.917549   \n","2                         AMR  -0.25  0.9          100          3  0.669542   \n","3                         AMR  -0.25  0.9          100          4  0.515529   \n","4                         AMR  -0.25  0.9          100          5  0.834179   \n","5                         AMR  -0.25  0.9          100          6  0.708066   \n","6                         AMR  -0.25  0.9          100          7  0.644395   \n","7                         AMR  -0.25  0.9          100          8  0.874606   \n","8                         AMR  -0.25  0.9          100          9  0.754595   \n","9                         AMR  -0.25  0.9          100         10  0.597966   \n","10                        AMR  -0.25  0.9          100         11  0.786782   \n","11                        AMR  -0.25  0.9          100         12  0.418058   \n","12                        AMR  -0.25  0.9          100         13  0.858607   \n","13                        AMR  -0.25  0.9          100         14  0.639613   \n","14                        AMR  -0.25  0.9          100         15  0.555247   \n","15                        AMR  -0.25  0.9          100         16  0.926675   \n","16                        AMR  -0.25  0.9          100         17  0.546923   \n","17                        AMR  -0.25  0.9          100         18  0.677769   \n","18                        AMR  -0.25  0.9          100         19  0.517292   \n","19                        AMR  -0.25  0.9          100         20  0.867061   \n","\n","    val      test  \n","0    -1  0.051002  \n","1    -1  0.275131  \n","2    -1  0.663977  \n","3    -1  0.018841  \n","4    -1  0.119562  \n","5    -1  0.000092  \n","6    -1  0.004948  \n","7    -1  0.055235  \n","8    -1  0.063838  \n","9    -1  0.218592  \n","10   -1  0.025450  \n","11   -1  0.187561  \n","12   -1  0.141334  \n","13   -1  0.517042  \n","14   -1  0.288989  \n","15   -1  0.014146  \n","16   -1  0.054263  \n","17   -1  0.012350  \n","18   -1  0.152771  \n","19   -1  0.081019  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5765169977494212\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002556015856336391\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5335746432584547\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.282966225417107\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6808637434225412\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0399138306378512\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141823 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5045961798247708\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024248822298986072\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7500045618011855\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030167642309149104\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141851 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5434350092765807\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.3913394149249461\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.537055948037576\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.25429166111377416\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.697404028953609\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.049654732361762315\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5270411141931775\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.12456061117405474\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8728139938014218\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05411769327570657\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8478872815187867\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.135846885605435\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5435222205385495\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.12365542778268389\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5931998688957181\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.1272568950632659\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6127687209631029\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.04835964452355329\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.598256210658862\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09188124160945377\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.612226463203263\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.1524749423359275\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.727551018752698\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11673070912576596\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5777930484009308\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.16955168733071901\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141749 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7153552395044259\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.11275784654827156\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6146576609797528\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.15311646089106548\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.9_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.9_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  48.13600  0.959192          1\n","1     HG01052  23.46500 -2.348548          1\n","2     HG01064  15.15180 -1.490464          1\n","3     HG01069   8.96991 -0.428314          1\n","4     HG01079  49.09660  1.068811          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -4.04269 -0.146929         20\n","1096  NA19741  -7.83801  0.139604         20\n","1097  NA19752   6.48531  0.008507         20\n","1098  NA19759  -7.89381 -1.073591         20\n","1099  NA19770   3.10688  0.258948         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.9          250          1  0.576517   \n","1                         AMR  -0.25  0.9          250          2  0.533575   \n","2                         AMR  -0.25  0.9          250          3  0.680864   \n","3                         AMR  -0.25  0.9          250          4  0.504596   \n","4                         AMR  -0.25  0.9          250          5  0.750005   \n","5                         AMR  -0.25  0.9          250          6  0.543435   \n","6                         AMR  -0.25  0.9          250          7  0.537056   \n","7                         AMR  -0.25  0.9          250          8  0.697404   \n","8                         AMR  -0.25  0.9          250          9  0.527041   \n","9                         AMR  -0.25  0.9          250         10  0.872814   \n","10                        AMR  -0.25  0.9          250         11  0.847887   \n","11                        AMR  -0.25  0.9          250         12  0.543522   \n","12                        AMR  -0.25  0.9          250         13  0.593200   \n","13                        AMR  -0.25  0.9          250         14  0.612769   \n","14                        AMR  -0.25  0.9          250         15  0.598256   \n","15                        AMR  -0.25  0.9          250         16  0.612226   \n","16                        AMR  -0.25  0.9          250         17  0.727551   \n","17                        AMR  -0.25  0.9          250         18  0.577793   \n","18                        AMR  -0.25  0.9          250         19  0.715355   \n","19                        AMR  -0.25  0.9          250         20  0.614658   \n","\n","    val      test  \n","0    -1  0.002556  \n","1    -1  0.282966  \n","2    -1  0.039914  \n","3    -1  0.024249  \n","4    -1  0.030168  \n","5    -1  0.391339  \n","6    -1  0.254292  \n","7    -1  0.049655  \n","8    -1  0.124561  \n","9    -1  0.054118  \n","10   -1  0.135847  \n","11   -1  0.123655  \n","12   -1  0.127257  \n","13   -1  0.048360  \n","14   -1  0.091881  \n","15   -1  0.152475  \n","16   -1  0.116731  \n","17   -1  0.169552  \n","18   -1  0.112758  \n","19   -1  0.153116  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5228372995435876\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05566964518327754\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5599981242599035\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.05819021244989931\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5358800221792164\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08067239561622262\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5349726509935141\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.05675316448932781\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5004443847882726\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0726816388785724\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5704740976934496\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0697634450364179\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.559871067095401\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05188756081393762\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5955858547087418\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05774856655936068\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5130083056418757\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.08908292375737008\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5344463553519444\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.011203271256556088\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5555057832338111\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05191246863521717\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.515108000563645\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.08280261969774753\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5693738023685727\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0880105915061197\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5039489853704605\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.38025009499289764\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5526845964144131\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.044558425860283066\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5470884616435859\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05448521239696667\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5609181791492166\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.08036399586514921\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5397687516865699\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.044932653637383364\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5634985633427109\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.055296273720659796\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6118169116322161\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04430373646284209\n","Combined prediction data has been written to: combined_predict_AMR_power=-0.25_her=0.9_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-0.25_her=0.9_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  59.74180  0.865936          1\n","1     HG01052  37.05290 -2.561170          1\n","2     HG01064  22.53350  0.050937          1\n","3     HG01069  14.16750 -0.593585          1\n","4     HG01079  54.79480 -0.636546          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -8.37011 -1.441265         20\n","1096  NA19741 -34.04030 -0.233845         20\n","1097  NA19752  38.57190 -0.991436         20\n","1098  NA19759 -38.33320  0.195152         20\n","1099  NA19770  22.82380  1.616123         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR  -0.25  0.9          512          1  0.522837   \n","1                         AMR  -0.25  0.9          512          2  0.559998   \n","2                         AMR  -0.25  0.9          512          3  0.535880   \n","3                         AMR  -0.25  0.9          512          4  0.534973   \n","4                         AMR  -0.25  0.9          512          5  0.500444   \n","5                         AMR  -0.25  0.9          512          6  0.570474   \n","6                         AMR  -0.25  0.9          512          7  0.559871   \n","7                         AMR  -0.25  0.9          512          8  0.595586   \n","8                         AMR  -0.25  0.9          512          9  0.513008   \n","9                         AMR  -0.25  0.9          512         10  0.534446   \n","10                        AMR  -0.25  0.9          512         11  0.555506   \n","11                        AMR  -0.25  0.9          512         12  0.515108   \n","12                        AMR  -0.25  0.9          512         13  0.569374   \n","13                        AMR  -0.25  0.9          512         14  0.503949   \n","14                        AMR  -0.25  0.9          512         15  0.552685   \n","15                        AMR  -0.25  0.9          512         16  0.547088   \n","16                        AMR  -0.25  0.9          512         17  0.560918   \n","17                        AMR  -0.25  0.9          512         18  0.539769   \n","18                        AMR  -0.25  0.9          512         19  0.563499   \n","19                        AMR  -0.25  0.9          512         20  0.611817   \n","\n","    val      test  \n","0    -1  0.055670  \n","1    -1  0.058190  \n","2    -1  0.080672  \n","3    -1  0.056753  \n","4    -1  0.072682  \n","5    -1  0.069763  \n","6    -1  0.051888  \n","7    -1  0.057749  \n","8    -1  0.089083  \n","9    -1  0.011203  \n","10   -1  0.051912  \n","11   -1  0.082803  \n","12   -1  0.088011  \n","13   -1  0.380250  \n","14   -1  0.044558  \n","15   -1  0.054485  \n","16   -1  0.080364  \n","17   -1  0.044933  \n","18   -1  0.055296  \n","19   -1  0.044304  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141718 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8899813840625828\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00011650245540452784\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141867 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141867 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8678938722597145\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07387008248470427\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5483655153749682\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00037834143352664116\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9026341169302994\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008597194550318604\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7043997313894408\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.01705499561552811\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8529227650677185\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0007021914048533104\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141840 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141840 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.666185565823765\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005196577366007864\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.924329934537552\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0005804069313646186\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8459892885549126\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005728694114764442\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141730 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7116068852818762\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.002151592292269678\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5084431214911161\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04425498744706503\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.431587016223837\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0009146672826833022\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8050646071076082\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0004926206035640092\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8710178592644913\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010327518389903608\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141727 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141727 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5958432344755591\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.03544598798310704\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8632618821893001\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0020155504367519346\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8984911710989717\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009715030173937945\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5992336643096312\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07659073277733901\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6909816304195474\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003496113413288803\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141853 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141853 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6097331954412928\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08513158180906183\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.1_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.1_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  -7.484340  0.561632          1\n","1     HG01052   2.082760  0.351465          1\n","2     HG01064  -0.723988  0.199119          1\n","3     HG01069  -7.301450 -1.143221          1\n","4     HG01079   0.029637 -0.578011          1\n","...       ...        ...       ...        ...\n","1095  NA19734   2.991180  0.497374         20\n","1096  NA19741  30.920800  1.115122         20\n","1097  NA19752 -43.115400 -0.405085         20\n","1098  NA19759  49.792300  0.402972         20\n","1099  NA19770  -2.298540 -0.964737         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.1            1          1  0.889981   \n","1                         AMR     -1  0.1            1          2  0.867894   \n","2                         AMR     -1  0.1            1          3  0.548366   \n","3                         AMR     -1  0.1            1          4  0.902634   \n","4                         AMR     -1  0.1            1          5  0.704400   \n","5                         AMR     -1  0.1            1          6  0.852923   \n","6                         AMR     -1  0.1            1          7  0.666186   \n","7                         AMR     -1  0.1            1          8  0.924330   \n","8                         AMR     -1  0.1            1          9  0.845989   \n","9                         AMR     -1  0.1            1         10  0.711607   \n","10                        AMR     -1  0.1            1         11  0.508443   \n","11                        AMR     -1  0.1            1         12  0.431587   \n","12                        AMR     -1  0.1            1         13  0.805065   \n","13                        AMR     -1  0.1            1         14  0.871018   \n","14                        AMR     -1  0.1            1         15  0.595843   \n","15                        AMR     -1  0.1            1         16  0.863262   \n","16                        AMR     -1  0.1            1         17  0.898491   \n","17                        AMR     -1  0.1            1         18  0.599234   \n","18                        AMR     -1  0.1            1         19  0.690982   \n","19                        AMR     -1  0.1            1         20  0.609733   \n","\n","    val      test  \n","0    -1  0.000117  \n","1    -1  0.073870  \n","2    -1  0.000378  \n","3    -1  0.008597  \n","4    -1  0.017055  \n","5    -1  0.000702  \n","6    -1  0.005197  \n","7    -1  0.000580  \n","8    -1  0.005729  \n","9    -1  0.002152  \n","10   -1  0.044255  \n","11   -1  0.000915  \n","12   -1  0.000493  \n","13   -1  0.010328  \n","14   -1  0.035446  \n","15   -1  0.002016  \n","16   -1  0.009715  \n","17   -1  0.076591  \n","18   -1  0.003496  \n","19   -1  0.085132  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7413775202059991\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0158335623405602\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8003518975921475\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026123948887720357\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7718074174525309\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.027178185533642995\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141713 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141713 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7767131533057599\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003032112278908793\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4899855481921882\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09861832937795724\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8960920293168257\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0499455818306438\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7431010134859656\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023427720502765696\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7169228418744833\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0034108891458291047\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7828736519603114\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0001416888592316636\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6994369071928584\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01745980246449581\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9117402830634108\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.001805165643086704\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4531724339397786\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.00010312066628617202\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7198247714573465\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00013494152420824836\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6155549158334895\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016584575582032208\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7465746805078147\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008768056036174827\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5483297910016212\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03162313824229741\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7830430506877488\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002485814920725877\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5741769589065177\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.018885129057018727\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9166500615027278\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.029209501075651473\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7900345110909144\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004186223921832577\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.1_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.1_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  29.43940  0.058808          1\n","1     HG01052  37.27630 -0.176159          1\n","2     HG01064  33.17740  0.853608          1\n","3     HG01069  25.31090  0.002745          1\n","4     HG01079  23.03320 -0.188158          1\n","...       ...       ...       ...        ...\n","1095  NA19734   5.35437  1.215148         20\n","1096  NA19741  12.22030  1.430426         20\n","1097  NA19752 -11.42270  0.720558         20\n","1098  NA19759   7.93099  1.085257         20\n","1099  NA19770 -10.70830  1.504600         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.1           10          1  0.741378   \n","1                         AMR     -1  0.1           10          2  0.800352   \n","2                         AMR     -1  0.1           10          3  0.771807   \n","3                         AMR     -1  0.1           10          4  0.776713   \n","4                         AMR     -1  0.1           10          5  0.489986   \n","5                         AMR     -1  0.1           10          6  0.896092   \n","6                         AMR     -1  0.1           10          7  0.743101   \n","7                         AMR     -1  0.1           10          8  0.716923   \n","8                         AMR     -1  0.1           10          9  0.782874   \n","9                         AMR     -1  0.1           10         10  0.699437   \n","10                        AMR     -1  0.1           10         11  0.911740   \n","11                        AMR     -1  0.1           10         12  0.453172   \n","12                        AMR     -1  0.1           10         13  0.719825   \n","13                        AMR     -1  0.1           10         14  0.615555   \n","14                        AMR     -1  0.1           10         15  0.746575   \n","15                        AMR     -1  0.1           10         16  0.548330   \n","16                        AMR     -1  0.1           10         17  0.783043   \n","17                        AMR     -1  0.1           10         18  0.574177   \n","18                        AMR     -1  0.1           10         19  0.916650   \n","19                        AMR     -1  0.1           10         20  0.790035   \n","\n","    val      test  \n","0    -1  0.015834  \n","1    -1  0.026124  \n","2    -1  0.027178  \n","3    -1  0.003032  \n","4    -1  0.098618  \n","5    -1  0.049946  \n","6    -1  0.023428  \n","7    -1  0.003411  \n","8    -1  0.000142  \n","9    -1  0.017460  \n","10   -1  0.001805  \n","11   -1  0.000103  \n","12   -1  0.000135  \n","13   -1  0.016585  \n","14   -1  0.008768  \n","15   -1  0.031623  \n","16   -1  0.002486  \n","17   -1  0.018885  \n","18   -1  0.029210  \n","19   -1  0.004186  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9055526110155738\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026657125517905975\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.879351139784797\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0021041042168788394\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141686 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141686 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.888917621571584\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004742718303040602\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6007277009062411\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023131574817911563\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7664375112975071\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0021543281267040223\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4857280309212552\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.019203136087289584\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8445010706703013\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01176989922211538\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8509404862515698\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0003313694966785031\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141713 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141713 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8973387729946991\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04276040513368025\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6600171739691784\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0028947992587962652\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8121951711829564\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024717625222284594\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5448023370887893\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05729331616067613\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7396171419249343\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0056538075379338\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5011896774386375\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05184805639568952\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7441462604058985\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004383547344242747\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141835 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141835 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9047943880574489\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.018877585414509828\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7308934516789902\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.011968671755455362\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8720351319743086\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008726780893463609\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141836 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141836 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6895820500018932\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008112835488237054\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.729960731571085\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06195247103270422\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.1_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.1_num-causals=100_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732   1.66421  0.931990          1\n","1     HG01052  -2.69572 -0.283532          1\n","2     HG01064 -19.50880  1.122290          1\n","3     HG01069 -20.90350  0.786841          1\n","4     HG01079   2.79799  1.064079          1\n","...       ...       ...       ...        ...\n","1095  NA19734   2.89331 -0.422592         20\n","1096  NA19741  25.63750  0.420297         20\n","1097  NA19752 -15.29150  0.726918         20\n","1098  NA19759  42.37010  0.178449         20\n","1099  NA19770 -13.71170 -1.728402         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.1          100          1  0.905553   \n","1                         AMR     -1  0.1          100          2  0.879351   \n","2                         AMR     -1  0.1          100          3  0.888918   \n","3                         AMR     -1  0.1          100          4  0.600728   \n","4                         AMR     -1  0.1          100          5  0.766438   \n","5                         AMR     -1  0.1          100          6  0.485728   \n","6                         AMR     -1  0.1          100          7  0.844501   \n","7                         AMR     -1  0.1          100          8  0.850940   \n","8                         AMR     -1  0.1          100          9  0.897339   \n","9                         AMR     -1  0.1          100         10  0.660017   \n","10                        AMR     -1  0.1          100         11  0.812195   \n","11                        AMR     -1  0.1          100         12  0.544802   \n","12                        AMR     -1  0.1          100         13  0.739617   \n","13                        AMR     -1  0.1          100         14  0.501190   \n","14                        AMR     -1  0.1          100         15  0.744146   \n","15                        AMR     -1  0.1          100         16  0.904794   \n","16                        AMR     -1  0.1          100         17  0.730893   \n","17                        AMR     -1  0.1          100         18  0.872035   \n","18                        AMR     -1  0.1          100         19  0.689582   \n","19                        AMR     -1  0.1          100         20  0.729961   \n","\n","    val      test  \n","0    -1  0.026657  \n","1    -1  0.002104  \n","2    -1  0.004743  \n","3    -1  0.023132  \n","4    -1  0.002154  \n","5    -1  0.019203  \n","6    -1  0.011770  \n","7    -1  0.000331  \n","8    -1  0.042760  \n","9    -1  0.002895  \n","10   -1  0.024718  \n","11   -1  0.057293  \n","12   -1  0.005654  \n","13   -1  0.051848  \n","14   -1  0.004384  \n","15   -1  0.018878  \n","16   -1  0.011969  \n","17   -1  0.008727  \n","18   -1  0.008113  \n","19   -1  0.061952  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9384761933645881\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009152323472481261\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6997325953277574\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07336363288002572\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5585473056928315\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005054311201272855\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8482548304006532\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0010239969883819625\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9235553839274564\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016977211668011173\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8496486903732546\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0001002685706602781\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6720925774442597\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0037807220823954566\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8136855016363309\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008283925923457734\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7983878896733098\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0033911422771864485\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8006592353170711\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002249341065493175\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6564243769764757\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01217250523368837\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6363335265746247\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024800615708525736\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8046909775430761\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01672278509394657\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141689 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141689 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7946269144506941\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.019923557630874212\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.896474327706869\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09103787225904603\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8430374567296599\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007358913678685157\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9302357408084024\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04025715451161996\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8849905798964142\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01317510510892156\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7783548341562383\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008958273729594981\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141696 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141696 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8576325757593029\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005113606424780911\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.1_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.1_num-causals=250_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   6.034370  0.746926          1\n","1     HG01052  16.332300 -2.081018          1\n","2     HG01064   5.758760 -0.530738          1\n","3     HG01069   1.139030 -0.274356          1\n","4     HG01079   4.440020  1.397227          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.992419  0.535487         20\n","1096  NA19741  -6.779970  0.174986         20\n","1097  NA19752   5.381320  0.121014         20\n","1098  NA19759  -7.939610 -0.966032         20\n","1099  NA19770 -14.279200 -0.119518         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.1          250          1  0.938476   \n","1                         AMR     -1  0.1          250          2  0.699733   \n","2                         AMR     -1  0.1          250          3  0.558547   \n","3                         AMR     -1  0.1          250          4  0.848255   \n","4                         AMR     -1  0.1          250          5  0.923555   \n","5                         AMR     -1  0.1          250          6  0.849649   \n","6                         AMR     -1  0.1          250          7  0.672093   \n","7                         AMR     -1  0.1          250          8  0.813686   \n","8                         AMR     -1  0.1          250          9  0.798388   \n","9                         AMR     -1  0.1          250         10  0.800659   \n","10                        AMR     -1  0.1          250         11  0.656424   \n","11                        AMR     -1  0.1          250         12  0.636334   \n","12                        AMR     -1  0.1          250         13  0.804691   \n","13                        AMR     -1  0.1          250         14  0.794627   \n","14                        AMR     -1  0.1          250         15  0.896474   \n","15                        AMR     -1  0.1          250         16  0.843037   \n","16                        AMR     -1  0.1          250         17  0.930236   \n","17                        AMR     -1  0.1          250         18  0.884991   \n","18                        AMR     -1  0.1          250         19  0.778355   \n","19                        AMR     -1  0.1          250         20  0.857633   \n","\n","    val      test  \n","0    -1  0.009152  \n","1    -1  0.073364  \n","2    -1  0.005054  \n","3    -1  0.001024  \n","4    -1  0.016977  \n","5    -1  0.000100  \n","6    -1  0.003781  \n","7    -1  0.000828  \n","8    -1  0.003391  \n","9    -1  0.002249  \n","10   -1  0.012173  \n","11   -1  0.024801  \n","12   -1  0.016723  \n","13   -1  0.019924  \n","14   -1  0.091038  \n","15   -1  0.007359  \n","16   -1  0.040257  \n","17   -1  0.013175  \n","18   -1  0.008958  \n","19   -1  0.005114  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9124076567082537\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.85053708203868e-08\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7607640751468657\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0029261734508533735\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9280349793638791\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00397578960168636\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9090102802659595\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.084022106429397\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8828711487183822\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010901237451771805\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.660995424391815\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0002496666161057343\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8913038723767192\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00019691118354277047\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8613229439497606\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.020436337400330518\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8630050401785218\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0003639533984672423\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6171142483368309\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0067375617237478425\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7551712315602068\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006032763667662437\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8557451660039648\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0013693774506871321\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8694495720093051\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008345184378102586\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7350747808900104\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0011775266312654352\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7694580358831403\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05425956153416596\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7733885237147408\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.02181611911810638\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.78371072463044\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.036454482953482054\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8224457012540088\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.014564357575479338\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8843575109896662\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005965694897924581\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7716032852332821\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0012976095171492394\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.1_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.1_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732   4.55476  0.886228          1\n","1     HG01052  -3.01379  0.327446          1\n","2     HG01064  -9.36226 -1.362904          1\n","3     HG01069  -8.81982 -1.310776          1\n","4     HG01079   2.75749 -0.611876          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -9.89205 -1.386987         20\n","1096  NA19741 -34.17780 -0.217449         20\n","1097  NA19752  12.45220 -0.032663         20\n","1098  NA19759 -14.63410 -1.498684         20\n","1099  NA19770  -2.42202 -0.703894         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.1          512          1  0.912408   \n","1                         AMR     -1  0.1          512          2  0.760764   \n","2                         AMR     -1  0.1          512          3  0.928035   \n","3                         AMR     -1  0.1          512          4  0.909010   \n","4                         AMR     -1  0.1          512          5  0.882871   \n","5                         AMR     -1  0.1          512          6  0.660995   \n","6                         AMR     -1  0.1          512          7  0.891304   \n","7                         AMR     -1  0.1          512          8  0.861323   \n","8                         AMR     -1  0.1          512          9  0.863005   \n","9                         AMR     -1  0.1          512         10  0.617114   \n","10                        AMR     -1  0.1          512         11  0.755171   \n","11                        AMR     -1  0.1          512         12  0.855745   \n","12                        AMR     -1  0.1          512         13  0.869450   \n","13                        AMR     -1  0.1          512         14  0.735075   \n","14                        AMR     -1  0.1          512         15  0.769458   \n","15                        AMR     -1  0.1          512         16  0.773389   \n","16                        AMR     -1  0.1          512         17  0.783711   \n","17                        AMR     -1  0.1          512         18  0.822446   \n","18                        AMR     -1  0.1          512         19  0.884358   \n","19                        AMR     -1  0.1          512         20  0.771603   \n","\n","    val          test  \n","0    -1  4.850537e-08  \n","1    -1  2.926173e-03  \n","2    -1  3.975790e-03  \n","3    -1  8.402211e-02  \n","4    -1  1.090124e-02  \n","5    -1  2.496666e-04  \n","6    -1  1.969112e-04  \n","7    -1  2.043634e-02  \n","8    -1  3.639534e-04  \n","9    -1  6.737562e-03  \n","10   -1  6.032764e-03  \n","11   -1  1.369377e-03  \n","12   -1  8.345184e-03  \n","13   -1  1.177527e-03  \n","14   -1  5.425956e-02  \n","15   -1  2.181612e-02  \n","16   -1  3.645448e-02  \n","17   -1  1.456436e-02  \n","18   -1  5.965695e-03  \n","19   -1  1.297610e-03  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6427519288826414\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00023552203504386394\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5049276116560943\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007407947361468799\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7906811576494023\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0010735698986766256\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.788319357515542\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005912409371854425\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8401511141723786\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0036228883426832766\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6481372076718764\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00013298754338742928\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7800121599927806\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00041464335738793524\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6007594529208463\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004454971310756419\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5358749394268868\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.01426721791956811\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7994349497754027\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007135310768193403\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8612912717505995\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0006039527648781862\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141830 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141830 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4303719021476401\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.04642766517285367\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6834737522378286\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006266586101505722\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5228029833613296\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04404692749676673\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8039582477331857\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09294693014223498\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6669523043184125\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03182687998889249\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7106198759871736\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0023030075927979224\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5307632311947771\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.11290329386212257\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6415872962610891\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009755504040458765\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6138229763463804\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05850819451399546\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.3_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.3_num-causals=1_pheno=1-20.csv\n","          IID  predict    actual  pheno_num\n","0     HG00732  40.6529  0.756351          1\n","1     HG01052  34.5812 -0.242036          1\n","2     HG01064  23.8463  1.131629          1\n","3     HG01069  17.6734 -1.872450          1\n","4     HG01079  52.0032 -0.125801          1\n","...       ...      ...       ...        ...\n","1095  NA19734  10.4611  0.212142         20\n","1096  NA19741  28.2405  0.102626         20\n","1097  NA19752 -58.3374  0.073826         20\n","1098  NA19759  43.4287  0.658375         20\n","1099  NA19770 -20.7993  1.763488         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.3            1          1  0.642752   \n","1                         AMR     -1  0.3            1          2  0.504928   \n","2                         AMR     -1  0.3            1          3  0.790681   \n","3                         AMR     -1  0.3            1          4  0.788319   \n","4                         AMR     -1  0.3            1          5  0.840151   \n","5                         AMR     -1  0.3            1          6  0.648137   \n","6                         AMR     -1  0.3            1          7  0.780012   \n","7                         AMR     -1  0.3            1          8  0.600759   \n","8                         AMR     -1  0.3            1          9  0.535875   \n","9                         AMR     -1  0.3            1         10  0.799435   \n","10                        AMR     -1  0.3            1         11  0.861291   \n","11                        AMR     -1  0.3            1         12  0.430372   \n","12                        AMR     -1  0.3            1         13  0.683474   \n","13                        AMR     -1  0.3            1         14  0.522803   \n","14                        AMR     -1  0.3            1         15  0.803958   \n","15                        AMR     -1  0.3            1         16  0.666952   \n","16                        AMR     -1  0.3            1         17  0.710620   \n","17                        AMR     -1  0.3            1         18  0.530763   \n","18                        AMR     -1  0.3            1         19  0.641587   \n","19                        AMR     -1  0.3            1         20  0.613823   \n","\n","    val      test  \n","0    -1  0.000236  \n","1    -1  0.007408  \n","2    -1  0.001074  \n","3    -1  0.005912  \n","4    -1  0.003623  \n","5    -1  0.000133  \n","6    -1  0.000415  \n","7    -1  0.004455  \n","8    -1  0.014267  \n","9    -1  0.007135  \n","10   -1  0.000604  \n","11   -1  0.046428  \n","12   -1  0.006267  \n","13   -1  0.044047  \n","14   -1  0.092947  \n","15   -1  0.031827  \n","16   -1  0.002303  \n","17   -1  0.112903  \n","18   -1  0.009756  \n","19   -1  0.058508  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141850 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141850 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4924325712074751\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004683852939022117\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5036038646700849\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007465387137562508\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8343940555626358\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.001331620752428799\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8896039373275443\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0013080918234201874\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.49086069749778827\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.08721692098348469\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8782772819400869\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06839471349303371\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141801 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8977548781277599\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007267196882286005\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8970822882147962\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0002924689769515354\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7067565157592914\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.047763879025254086\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8492138473615104\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0012643068724348555\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7485221677991253\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004553447554093194\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.41684430107535264\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.02337833934660781\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7593806310217871\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.021701040774165972\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4722374841429886\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013615418486203285\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8726690377020226\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009166811922150497\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4590423568230311\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.21335915504677933\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7654619419923823\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06344392963919705\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9052244076825601\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0031635658506034467\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7132718698368039\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0037587925279821232\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7317228358570308\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=8.92427846288733e-05\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.3_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.3_num-causals=10_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  73.862900  0.526962          1\n","1     HG01052  50.642600  1.641565          1\n","2     HG01064  44.453900  0.367933          1\n","3     HG01069  20.904300 -1.694887          1\n","4     HG01079  54.856600 -0.974518          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.489654  0.101854         20\n","1096  NA19741 -22.997600 -0.688834         20\n","1097  NA19752  30.864800  0.534054         20\n","1098  NA19759 -14.595500  0.659710         20\n","1099  NA19770  17.509200  1.639384         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.3           10          1  0.492433   \n","1                         AMR     -1  0.3           10          2  0.503604   \n","2                         AMR     -1  0.3           10          3  0.834394   \n","3                         AMR     -1  0.3           10          4  0.889604   \n","4                         AMR     -1  0.3           10          5  0.490861   \n","5                         AMR     -1  0.3           10          6  0.878277   \n","6                         AMR     -1  0.3           10          7  0.897755   \n","7                         AMR     -1  0.3           10          8  0.897082   \n","8                         AMR     -1  0.3           10          9  0.706757   \n","9                         AMR     -1  0.3           10         10  0.849214   \n","10                        AMR     -1  0.3           10         11  0.748522   \n","11                        AMR     -1  0.3           10         12  0.416844   \n","12                        AMR     -1  0.3           10         13  0.759381   \n","13                        AMR     -1  0.3           10         14  0.472237   \n","14                        AMR     -1  0.3           10         15  0.872669   \n","15                        AMR     -1  0.3           10         16  0.459042   \n","16                        AMR     -1  0.3           10         17  0.765462   \n","17                        AMR     -1  0.3           10         18  0.905224   \n","18                        AMR     -1  0.3           10         19  0.713272   \n","19                        AMR     -1  0.3           10         20  0.731723   \n","\n","    val      test  \n","0    -1  0.004684  \n","1    -1  0.007465  \n","2    -1  0.001332  \n","3    -1  0.001308  \n","4    -1  0.087217  \n","5    -1  0.068395  \n","6    -1  0.007267  \n","7    -1  0.000292  \n","8    -1  0.047764  \n","9    -1  0.001264  \n","10   -1  0.004553  \n","11   -1  0.023378  \n","12   -1  0.021701  \n","13   -1  0.013615  \n","14   -1  0.000917  \n","15   -1  0.213359  \n","16   -1  0.063444  \n","17   -1  0.003164  \n","18   -1  0.003759  \n","19   -1  0.000089  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8410856394895369\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0027374001440375164\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7231431500536382\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002273816791254009\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6533672450499697\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.012811002245671423\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.49903368874906606\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.022118955996121115\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8643756999044726\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07487465366706239\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5637223467342697\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01949634472245569\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8808480567100229\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00910007430333016\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8137704795642915\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0017222730482835654\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6451485052578259\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=6.372124906739824e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6505633414445797\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0629541386351717\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8915224889957888\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03828803200465796\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.43103518400551366\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09786312909617947\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141835 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141835 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7494228183835446\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0012152845494887042\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5011190704660728\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0827485372899881\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5598366386278283\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008246329629651846\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6922250065983212\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.05365221548771841\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4553962862664812\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0018730830153257706\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8609780518260565\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03296630023286084\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141822 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8079869085766054\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03656002723697185\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9244253748040152\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.032598180652337196\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.3_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.3_num-causals=100_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732 -16.10410  0.369079          1\n","1     HG01052  -8.26738 -2.125669          1\n","2     HG01064  -7.01542 -0.151841          1\n","3     HG01069   1.69406 -1.010977          1\n","4     HG01079 -11.75840  1.578228          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -2.69526 -0.726116         20\n","1096  NA19741   3.31993  1.785216         20\n","1097  NA19752  19.70370  1.282351         20\n","1098  NA19759   4.40472 -0.867918         20\n","1099  NA19770  14.66650  1.186345         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.3          100          1  0.841086   \n","1                         AMR     -1  0.3          100          2  0.723143   \n","2                         AMR     -1  0.3          100          3  0.653367   \n","3                         AMR     -1  0.3          100          4  0.499034   \n","4                         AMR     -1  0.3          100          5  0.864376   \n","5                         AMR     -1  0.3          100          6  0.563722   \n","6                         AMR     -1  0.3          100          7  0.880848   \n","7                         AMR     -1  0.3          100          8  0.813770   \n","8                         AMR     -1  0.3          100          9  0.645149   \n","9                         AMR     -1  0.3          100         10  0.650563   \n","10                        AMR     -1  0.3          100         11  0.891522   \n","11                        AMR     -1  0.3          100         12  0.431035   \n","12                        AMR     -1  0.3          100         13  0.749423   \n","13                        AMR     -1  0.3          100         14  0.501119   \n","14                        AMR     -1  0.3          100         15  0.559837   \n","15                        AMR     -1  0.3          100         16  0.692225   \n","16                        AMR     -1  0.3          100         17  0.455396   \n","17                        AMR     -1  0.3          100         18  0.860978   \n","18                        AMR     -1  0.3          100         19  0.807987   \n","19                        AMR     -1  0.3          100         20  0.924425   \n","\n","    val      test  \n","0    -1  0.002737  \n","1    -1  0.002274  \n","2    -1  0.012811  \n","3    -1  0.022119  \n","4    -1  0.074875  \n","5    -1  0.019496  \n","6    -1  0.009100  \n","7    -1  0.001722  \n","8    -1  0.000064  \n","9    -1  0.062954  \n","10   -1  0.038288  \n","11   -1  0.097863  \n","12   -1  0.001215  \n","13   -1  0.082749  \n","14   -1  0.000825  \n","15   -1  0.053652  \n","16   -1  0.001873  \n","17   -1  0.032966  \n","18   -1  0.036560  \n","19   -1  0.032598  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141687 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141687 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7137734866590799\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0009936104002804392\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9158977740782402\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.8662631369866397e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4294522410331076\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06823820479219572\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5383102997901765\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00024360545117328337\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8781581089169969\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02280475758919674\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6201842962572087\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013611980805056057\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141785 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8341243108445765\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.013756147191070068\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141720 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.867371858114847\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00014057179322807825\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9100599394498485\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004744060345951134\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7628758086866565\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017014631008503322\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7596822830135929\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0474686057339609\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7045447443309416\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032475627292600446\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9110088258271135\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01084296671776319\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8190729688823828\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010981183272141414\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141695 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141695 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7350815520963557\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0005443769964031119\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7863319018577979\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0016014538359727045\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8664708012080847\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.020137961058200968\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8736289528280462\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0022301173889118647\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8467101257326268\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0004082756647241337\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.82926816977824\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006215341253432193\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.3_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.3_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  23.07150  0.335653          1\n","1     HG01052  -1.94840 -1.271184          1\n","2     HG01064   8.16978 -1.890582          1\n","3     HG01069  17.72110 -0.710113          1\n","4     HG01079  33.56470  1.179666          1\n","...       ...       ...       ...        ...\n","1095  NA19734   8.86734  0.470520         20\n","1096  NA19741 -17.99120 -1.139121         20\n","1097  NA19752  38.06060  0.434777         20\n","1098  NA19759  -1.99859  0.628531         20\n","1099  NA19770  23.50830  1.091789         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.3          250          1  0.713773   \n","1                         AMR     -1  0.3          250          2  0.915898   \n","2                         AMR     -1  0.3          250          3  0.429452   \n","3                         AMR     -1  0.3          250          4  0.538310   \n","4                         AMR     -1  0.3          250          5  0.878158   \n","5                         AMR     -1  0.3          250          6  0.620184   \n","6                         AMR     -1  0.3          250          7  0.834124   \n","7                         AMR     -1  0.3          250          8  0.867372   \n","8                         AMR     -1  0.3          250          9  0.910060   \n","9                         AMR     -1  0.3          250         10  0.762876   \n","10                        AMR     -1  0.3          250         11  0.759682   \n","11                        AMR     -1  0.3          250         12  0.704545   \n","12                        AMR     -1  0.3          250         13  0.911009   \n","13                        AMR     -1  0.3          250         14  0.819073   \n","14                        AMR     -1  0.3          250         15  0.735082   \n","15                        AMR     -1  0.3          250         16  0.786332   \n","16                        AMR     -1  0.3          250         17  0.866471   \n","17                        AMR     -1  0.3          250         18  0.873629   \n","18                        AMR     -1  0.3          250         19  0.846710   \n","19                        AMR     -1  0.3          250         20  0.829268   \n","\n","    val      test  \n","0    -1  0.000994  \n","1    -1  0.000019  \n","2    -1  0.068238  \n","3    -1  0.000244  \n","4    -1  0.022805  \n","5    -1  0.013612  \n","6    -1  0.013756  \n","7    -1  0.000141  \n","8    -1  0.004744  \n","9    -1  0.017015  \n","10   -1  0.047469  \n","11   -1  0.003248  \n","12   -1  0.010843  \n","13   -1  0.010981  \n","14   -1  0.000544  \n","15   -1  0.001601  \n","16   -1  0.020138  \n","17   -1  0.002230  \n","18   -1  0.000408  \n","19   -1  0.006215  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6586558550566488\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003505898321200145\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8244750228425957\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0017449504979403882\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141717 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141717 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7181893421076043\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.022362548916021675\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8837945326422846\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=2.122758968467804e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7006288934223737\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0026843089593599377\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8234440182794184\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0065946166581223355\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.896888287097732\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007236205866569532\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8251278832901638\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01682884202535474\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6346441429208988\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02148090953813165\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9159005458883736\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07327573782002851\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8724433128853066\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.7351910367584496e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8964783719918358\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0005253402001734098\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7312254269682777\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05596832516780404\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6059973131459404\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02651736715121749\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9159944355066907\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009915291971574964\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6901315680532574\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=5.7670421896913945e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7176041166570936\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.026653219308030513\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8491143281985828\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0003114524334298\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8855286717707487\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003503166848074097\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7986294776820451\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00014835927084291852\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.3_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.3_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  50.82760 -0.051201          1\n","1     HG01052  26.04060  0.285355          1\n","2     HG01064  21.02930 -2.228845          1\n","3     HG01069   5.63492 -1.367164          1\n","4     HG01079  34.75880 -1.597806          1\n","...       ...       ...       ...        ...\n","1095  NA19734 -12.47050 -0.497295         20\n","1096  NA19741 -18.79350  0.427934         20\n","1097  NA19752  11.77590  0.599626         20\n","1098  NA19759 -21.79670  1.792580         20\n","1099  NA19770   2.22242  0.161886         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.3          512          1  0.658656   \n","1                         AMR     -1  0.3          512          2  0.824475   \n","2                         AMR     -1  0.3          512          3  0.718189   \n","3                         AMR     -1  0.3          512          4  0.883795   \n","4                         AMR     -1  0.3          512          5  0.700629   \n","5                         AMR     -1  0.3          512          6  0.823444   \n","6                         AMR     -1  0.3          512          7  0.896888   \n","7                         AMR     -1  0.3          512          8  0.825128   \n","8                         AMR     -1  0.3          512          9  0.634644   \n","9                         AMR     -1  0.3          512         10  0.915901   \n","10                        AMR     -1  0.3          512         11  0.872443   \n","11                        AMR     -1  0.3          512         12  0.896478   \n","12                        AMR     -1  0.3          512         13  0.731225   \n","13                        AMR     -1  0.3          512         14  0.605997   \n","14                        AMR     -1  0.3          512         15  0.915994   \n","15                        AMR     -1  0.3          512         16  0.690132   \n","16                        AMR     -1  0.3          512         17  0.717604   \n","17                        AMR     -1  0.3          512         18  0.849114   \n","18                        AMR     -1  0.3          512         19  0.885529   \n","19                        AMR     -1  0.3          512         20  0.798629   \n","\n","    val      test  \n","0    -1  0.003506  \n","1    -1  0.001745  \n","2    -1  0.022363  \n","3    -1  0.000021  \n","4    -1  0.002684  \n","5    -1  0.006595  \n","6    -1  0.007236  \n","7    -1  0.016829  \n","8    -1  0.021481  \n","9    -1  0.073276  \n","10   -1  0.000017  \n","11   -1  0.000525  \n","12   -1  0.055968  \n","13   -1  0.026517  \n","14   -1  0.009915  \n","15   -1  0.000006  \n","16   -1  0.026653  \n","17   -1  0.000311  \n","18   -1  0.003503  \n","19   -1  0.000148  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8743515541934356\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.062077177715606786\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45061202681474494\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0694052552686179\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.706210702381439\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0006717290307503053\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7682723994799819\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.010870287586336957\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5456590135375553\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.022351768438164988\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5994322129955663\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.011080077667247222\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5830848660574534\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01526333841938221\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6578576453314123\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.006415913524447638\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141844 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141844 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5887703660935352\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4639470608928864\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4921848067077662\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0065152717741872295\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141812 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141812 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7464722980417854\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017231094310324793\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141819 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.413739894203838\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.03884562393317637\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141754 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7980448153257939\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002323399107040962\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141809 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8812023063115801\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016099818796595167\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6803501030269642\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04067622854582912\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141858 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141858 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4978453307356736\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06214865282574809\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7354434776876843\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.006002403889650193\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.608921916146773\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.038762340180465676\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5840971930807031\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.2866817495278726\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5220246097517358\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.13482752858236882\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.5_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.5_num-causals=1_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  14.61660  1.014668          1\n","1     HG01052   9.36657 -0.206036          1\n","2     HG01064   2.25508 -0.217853          1\n","3     HG01069 -10.56250  0.574502          1\n","4     HG01079  14.84960  0.018479          1\n","...       ...       ...       ...        ...\n","1095  NA19734 -10.60180  1.738732         20\n","1096  NA19741  30.53210 -0.038429         20\n","1097  NA19752 -57.57360 -0.321365         20\n","1098  NA19759  38.07020  0.579301         20\n","1099  NA19770 -24.11990 -0.263901         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.5            1          1  0.874352   \n","1                         AMR     -1  0.5            1          2  0.450612   \n","2                         AMR     -1  0.5            1          3  0.706211   \n","3                         AMR     -1  0.5            1          4  0.768272   \n","4                         AMR     -1  0.5            1          5  0.545659   \n","5                         AMR     -1  0.5            1          6  0.599432   \n","6                         AMR     -1  0.5            1          7  0.583085   \n","7                         AMR     -1  0.5            1          8  0.657858   \n","8                         AMR     -1  0.5            1          9  0.588770   \n","9                         AMR     -1  0.5            1         10  0.492185   \n","10                        AMR     -1  0.5            1         11  0.746472   \n","11                        AMR     -1  0.5            1         12  0.413740   \n","12                        AMR     -1  0.5            1         13  0.798045   \n","13                        AMR     -1  0.5            1         14  0.881202   \n","14                        AMR     -1  0.5            1         15  0.680350   \n","15                        AMR     -1  0.5            1         16  0.497845   \n","16                        AMR     -1  0.5            1         17  0.735443   \n","17                        AMR     -1  0.5            1         18  0.608922   \n","18                        AMR     -1  0.5            1         19  0.584097   \n","19                        AMR     -1  0.5            1         20  0.522025   \n","\n","    val      test  \n","0    -1  0.062077  \n","1    -1  0.069405  \n","2    -1  0.000672  \n","3    -1  0.010870  \n","4    -1  0.022352  \n","5    -1  0.011080  \n","6    -1  0.015263  \n","7    -1  0.006416  \n","8    -1  0.463947  \n","9    -1  0.006515  \n","10   -1  0.017231  \n","11   -1  0.038846  \n","12   -1  0.002323  \n","13   -1  0.016100  \n","14   -1  0.040676  \n","15   -1  0.062149  \n","16   -1  0.006002  \n","17   -1  0.038762  \n","18   -1  0.286682  \n","19   -1  0.134828  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48692608994684194\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.029374902495392246\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5742502998471187\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04617745534884474\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8747867582827041\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0019493847407792886\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7400223409444854\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0030083384163297503\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141869 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141869 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5432693352908803\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.32388954307246315\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5694313345143804\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04014289666882358\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8702486579586041\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.011188304451021973\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8911077175486815\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.020324949090953615\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8042563908116255\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.008422469514949373\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.905450272466719\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002019379209499422\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7508673171780285\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003333763749695783\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.4734713620993916\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.10463945524993618\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5917969521859187\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.46489736186556135\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141803 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.469951194526206\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.12677445404476373\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141845 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141845 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9160106839432751\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03199565239305603\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.3976997383855024\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.03683321883762066\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141746 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.611461317309606\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.1316943600331118\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8969698972522311\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0068088564032567425\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6186596426766334\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.698205944627648e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.881663559044982\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00011773451060226921\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.5_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.5_num-causals=10_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  47.566000 -1.238253          1\n","1     HG01052  41.452200  0.758769          1\n","2     HG01064  27.391100  0.263587          1\n","3     HG01069   6.179430 -0.445744          1\n","4     HG01079  41.415700 -0.080350          1\n","...       ...        ...       ...        ...\n","1095  NA19734  11.729000 -0.021752         20\n","1096  NA19741  18.190100  0.975288         20\n","1097  NA19752   0.503416  1.313239         20\n","1098  NA19759   9.062480 -0.491356         20\n","1099  NA19770  -4.727130  0.598020         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.5           10          1  0.486926   \n","1                         AMR     -1  0.5           10          2  0.574250   \n","2                         AMR     -1  0.5           10          3  0.874787   \n","3                         AMR     -1  0.5           10          4  0.740022   \n","4                         AMR     -1  0.5           10          5  0.543269   \n","5                         AMR     -1  0.5           10          6  0.569431   \n","6                         AMR     -1  0.5           10          7  0.870249   \n","7                         AMR     -1  0.5           10          8  0.891108   \n","8                         AMR     -1  0.5           10          9  0.804256   \n","9                         AMR     -1  0.5           10         10  0.905450   \n","10                        AMR     -1  0.5           10         11  0.750867   \n","11                        AMR     -1  0.5           10         12  0.473471   \n","12                        AMR     -1  0.5           10         13  0.591797   \n","13                        AMR     -1  0.5           10         14  0.469951   \n","14                        AMR     -1  0.5           10         15  0.916011   \n","15                        AMR     -1  0.5           10         16  0.397700   \n","16                        AMR     -1  0.5           10         17  0.611461   \n","17                        AMR     -1  0.5           10         18  0.896970   \n","18                        AMR     -1  0.5           10         19  0.618660   \n","19                        AMR     -1  0.5           10         20  0.881664   \n","\n","    val      test  \n","0    -1  0.029375  \n","1    -1  0.046177  \n","2    -1  0.001949  \n","3    -1  0.003008  \n","4    -1  0.323890  \n","5    -1  0.040143  \n","6    -1  0.011188  \n","7    -1  0.020325  \n","8    -1  0.008422  \n","9    -1  0.002019  \n","10   -1  0.003334  \n","11   -1  0.104639  \n","12   -1  0.464897  \n","13   -1  0.126774  \n","14   -1  0.031996  \n","15   -1  0.036833  \n","16   -1  0.131694  \n","17   -1  0.006809  \n","18   -1  0.000047  \n","19   -1  0.000118  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8558323457324057\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03995963133818202\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.863296994553836\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.018100273246571784\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5145944464205121\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04184244497793953\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4447323550136196\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030851439662637533\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8524915965442647\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.13520056722382845\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7164543318930636\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.001965199181412672\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8680974256551718\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004622038341539926\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.667461719295161\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00467222133757736\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6135781251706829\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.025211388826444952\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.659699331540377\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06670543258696936\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7462707501632198\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00028552371313093504\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45156942738792377\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.27284190534172365\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7778687278819932\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024683482247354383\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4680466343682262\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.29109157136569974\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4885194097995229\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=9.449664540738313e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141729 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.665811992618363\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.016436769391749616\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.677956855303651\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07164279988527801\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.899974531284648\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0567794991255218\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5739312571496848\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023788206125084553\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8140558123888288\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002269551056975723\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.5_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.5_num-causals=100_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  19.45990  2.467742          1\n","1     HG01052  21.98430 -0.700991          1\n","2     HG01064   2.32436  1.147002          1\n","3     HG01069   5.91957 -0.938706          1\n","4     HG01079  20.05050  0.252217          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -3.81550 -0.632308         20\n","1096  NA19741   4.62854  1.023281         20\n","1097  NA19752 -15.51680  0.330000         20\n","1098  NA19759  19.61200  0.196553         20\n","1099  NA19770 -14.04160  0.481078         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.5          100          1  0.855832   \n","1                         AMR     -1  0.5          100          2  0.863297   \n","2                         AMR     -1  0.5          100          3  0.514594   \n","3                         AMR     -1  0.5          100          4  0.444732   \n","4                         AMR     -1  0.5          100          5  0.852492   \n","5                         AMR     -1  0.5          100          6  0.716454   \n","6                         AMR     -1  0.5          100          7  0.868097   \n","7                         AMR     -1  0.5          100          8  0.667462   \n","8                         AMR     -1  0.5          100          9  0.613578   \n","9                         AMR     -1  0.5          100         10  0.659699   \n","10                        AMR     -1  0.5          100         11  0.746271   \n","11                        AMR     -1  0.5          100         12  0.451569   \n","12                        AMR     -1  0.5          100         13  0.777869   \n","13                        AMR     -1  0.5          100         14  0.468047   \n","14                        AMR     -1  0.5          100         15  0.488519   \n","15                        AMR     -1  0.5          100         16  0.665812   \n","16                        AMR     -1  0.5          100         17  0.677957   \n","17                        AMR     -1  0.5          100         18  0.899975   \n","18                        AMR     -1  0.5          100         19  0.573931   \n","19                        AMR     -1  0.5          100         20  0.814056   \n","\n","    val      test  \n","0    -1  0.039960  \n","1    -1  0.018100  \n","2    -1  0.041842  \n","3    -1  0.030851  \n","4    -1  0.135201  \n","5    -1  0.001965  \n","6    -1  0.004622  \n","7    -1  0.004672  \n","8    -1  0.025211  \n","9    -1  0.066705  \n","10   -1  0.000286  \n","11   -1  0.272842  \n","12   -1  0.024683  \n","13   -1  0.291092  \n","14   -1  0.000094  \n","15   -1  0.016437  \n","16   -1  0.071643  \n","17   -1  0.056779  \n","18   -1  0.023788  \n","19   -1  0.002270  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7692004022281085\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=2.8044598318948637e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6042501678557248\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03130747582298428\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.49235283052452455\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.028402193408148046\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8129664754158477\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.057237893398540796\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8924009110891648\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005197955558542712\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5508389505783174\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05743570779261949\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7251632068201229\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.06169676665031945\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6156215229817269\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0021587296833554676\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9085447448271914\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.415836882491532e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5898940614635291\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0718992671604637\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141826 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141826 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6983154495926764\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.046937344762661266\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9130572133926126\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0027794971781014027\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8609864836234931\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002312755304217461\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141738 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9166077171346022\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005043677974301238\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5684555460865108\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04298258780958834\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8030629386778327\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03459827838330842\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.858738436161895\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04333424618948953\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141814 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7632677251817962\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00251147406651983\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.5415767720916521\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0027495894516551657\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7212308995614626\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002434227157523936\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.5_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.5_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732   8.14338  0.211403          1\n","1     HG01052  -6.44434 -0.092892          1\n","2     HG01064 -20.19320 -1.244714          1\n","3     HG01069   0.35507 -0.431231          1\n","4     HG01079  12.86150  1.022567          1\n","...       ...       ...       ...        ...\n","1095  NA19734 -13.17690 -0.839360         20\n","1096  NA19741 -36.35780  0.578447         20\n","1097  NA19752  22.63900  0.022519         20\n","1098  NA19759 -28.86060 -0.326972         20\n","1099  NA19770  15.15710  0.362310         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.5          250          1  0.769200   \n","1                         AMR     -1  0.5          250          2  0.604250   \n","2                         AMR     -1  0.5          250          3  0.492353   \n","3                         AMR     -1  0.5          250          4  0.812966   \n","4                         AMR     -1  0.5          250          5  0.892401   \n","5                         AMR     -1  0.5          250          6  0.550839   \n","6                         AMR     -1  0.5          250          7  0.725163   \n","7                         AMR     -1  0.5          250          8  0.615622   \n","8                         AMR     -1  0.5          250          9  0.908545   \n","9                         AMR     -1  0.5          250         10  0.589894   \n","10                        AMR     -1  0.5          250         11  0.698315   \n","11                        AMR     -1  0.5          250         12  0.913057   \n","12                        AMR     -1  0.5          250         13  0.860986   \n","13                        AMR     -1  0.5          250         14  0.916608   \n","14                        AMR     -1  0.5          250         15  0.568456   \n","15                        AMR     -1  0.5          250         16  0.803063   \n","16                        AMR     -1  0.5          250         17  0.858738   \n","17                        AMR     -1  0.5          250         18  0.763268   \n","18                        AMR     -1  0.5          250         19  0.541577   \n","19                        AMR     -1  0.5          250         20  0.721231   \n","\n","    val      test  \n","0    -1  0.000028  \n","1    -1  0.031307  \n","2    -1  0.028402  \n","3    -1  0.057238  \n","4    -1  0.005198  \n","5    -1  0.057436  \n","6    -1  0.061697  \n","7    -1  0.002159  \n","8    -1  0.000034  \n","9    -1  0.071899  \n","10   -1  0.046937  \n","11   -1  0.002779  \n","12   -1  0.002313  \n","13   -1  0.005044  \n","14   -1  0.042983  \n","15   -1  0.034598  \n","16   -1  0.043334  \n","17   -1  0.002511  \n","18   -1  0.002750  \n","19   -1  0.002434  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8210016475240521\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0016671580322389849\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.636170199790581\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=2.4854475226827873e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141680 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141680 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8755722569592499\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.004760964419481759\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141768 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7494835848977441\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.08956459891358237\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8061502645133486\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.022307845194629243\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141807 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8574398600692786\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.042195100928268836\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8885636942051679\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00022194051640060317\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.677053133881462\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030318477744725555\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8666946919450187\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05671326331810866\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.84082234910491\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0012823482445178747\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8975687219515762\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.049687403448935026\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.895629332812227\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02502114377544338\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141793 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8263623080004461\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014456765619314452\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141735 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.850695461775801\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0004845535483914572\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7775590855410529\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0006697583334611881\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141743 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.927842619319862\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03072810164965502\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141716 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141716 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.9168282455182155\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0010798855873216926\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.908390119219977\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.018617343123967307\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8088629305065365\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00973917631116833\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.911039992019892\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00304068956787816\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.5_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.5_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732 -16.70540  1.039254          1\n","1     HG01052 -19.69990 -1.819275          1\n","2     HG01064 -41.79040 -1.532652          1\n","3     HG01069  -2.34036 -1.078952          1\n","4     HG01079 -16.28290  1.798863          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -6.90710 -1.033272         20\n","1096  NA19741  -1.19050 -1.097010         20\n","1097  NA19752 -17.26240  0.191931         20\n","1098  NA19759   6.89221  0.282842         20\n","1099  NA19770   5.51799  0.392857         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.5          512          1  0.821002   \n","1                         AMR     -1  0.5          512          2  0.636170   \n","2                         AMR     -1  0.5          512          3  0.875572   \n","3                         AMR     -1  0.5          512          4  0.749484   \n","4                         AMR     -1  0.5          512          5  0.806150   \n","5                         AMR     -1  0.5          512          6  0.857440   \n","6                         AMR     -1  0.5          512          7  0.888564   \n","7                         AMR     -1  0.5          512          8  0.677053   \n","8                         AMR     -1  0.5          512          9  0.866695   \n","9                         AMR     -1  0.5          512         10  0.840822   \n","10                        AMR     -1  0.5          512         11  0.897569   \n","11                        AMR     -1  0.5          512         12  0.895629   \n","12                        AMR     -1  0.5          512         13  0.826362   \n","13                        AMR     -1  0.5          512         14  0.850695   \n","14                        AMR     -1  0.5          512         15  0.777559   \n","15                        AMR     -1  0.5          512         16  0.927843   \n","16                        AMR     -1  0.5          512         17  0.916828   \n","17                        AMR     -1  0.5          512         18  0.908390   \n","18                        AMR     -1  0.5          512         19  0.808863   \n","19                        AMR     -1  0.5          512         20  0.911040   \n","\n","    val      test  \n","0    -1  0.001667  \n","1    -1  0.000025  \n","2    -1  0.004761  \n","3    -1  0.089565  \n","4    -1  0.022308  \n","5    -1  0.042195  \n","6    -1  0.000222  \n","7    -1  0.030318  \n","8    -1  0.056713  \n","9    -1  0.001282  \n","10   -1  0.049687  \n","11   -1  0.025021  \n","12   -1  0.014457  \n","13   -1  0.000485  \n","14   -1  0.000670  \n","15   -1  0.030728  \n","16   -1  0.001080  \n","17   -1  0.018617  \n","18   -1  0.009739  \n","19   -1  0.003041  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.822134437479258\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.002846246303663226\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5118703039374217\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.006688510023276805\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.771876360490081\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=3.2735840416562586e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6147312217989953\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.012113109770477184\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7164977352617818\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=6.656494738963499e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6826055841737553\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.7057707899867823\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5466504056853441\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.3484139928379641\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7803405328906056\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=2.916962869097222e-06\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6756005293072572\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.554906507358378\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5007002087660227\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0911093610903761\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.6013023320076559\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.027442378087096864\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141870 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141870 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.497589197237154\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.15982550385424887\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8454841566454466\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8913247310601947\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7181292664302262\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.10089989297584877\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7252690624174485\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6478933926987811\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141860 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141860 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5108166734692305\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.04594178293946057\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8944134364810417\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01080321434895379\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.631326613046048\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4522644881813537\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6398032009425428\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.35262196787948685\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5616952125351484\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.457470177801763\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.7_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.7_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  29.502900 -0.927383          1\n","1     HG01052  24.666400 -0.268158          1\n","2     HG01064  10.369700  0.236677          1\n","3     HG01069   8.484730  0.774524          1\n","4     HG01079  37.122900 -1.213590          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.861749  0.745869         20\n","1096  NA19741   0.408789  0.628143         20\n","1097  NA19752  -2.291110 -0.836776         20\n","1098  NA19759   0.463272  0.385249         20\n","1099  NA19770  -0.011294  1.194447         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.7            1          1  0.822134   \n","1                         AMR     -1  0.7            1          2  0.511870   \n","2                         AMR     -1  0.7            1          3  0.771876   \n","3                         AMR     -1  0.7            1          4  0.614731   \n","4                         AMR     -1  0.7            1          5  0.716498   \n","5                         AMR     -1  0.7            1          6  0.682606   \n","6                         AMR     -1  0.7            1          7  0.546650   \n","7                         AMR     -1  0.7            1          8  0.780341   \n","8                         AMR     -1  0.7            1          9  0.675601   \n","9                         AMR     -1  0.7            1         10  0.500700   \n","10                        AMR     -1  0.7            1         11  0.601302   \n","11                        AMR     -1  0.7            1         12  0.497589   \n","12                        AMR     -1  0.7            1         13  0.845484   \n","13                        AMR     -1  0.7            1         14  0.718129   \n","14                        AMR     -1  0.7            1         15  0.725269   \n","15                        AMR     -1  0.7            1         16  0.510817   \n","16                        AMR     -1  0.7            1         17  0.894413   \n","17                        AMR     -1  0.7            1         18  0.631327   \n","18                        AMR     -1  0.7            1         19  0.639803   \n","19                        AMR     -1  0.7            1         20  0.561695   \n","\n","    val      test  \n","0    -1  0.002846  \n","1    -1  0.006689  \n","2    -1  0.000033  \n","3    -1  0.012113  \n","4    -1  0.000067  \n","5    -1  0.705771  \n","6    -1  0.348414  \n","7    -1  0.000003  \n","8    -1  0.554907  \n","9    -1  0.091109  \n","10   -1  0.027442  \n","11   -1  0.159826  \n","12   -1  0.891325  \n","13   -1  0.100900  \n","14   -1  0.647893  \n","15   -1  0.045942  \n","16   -1  0.010803  \n","17   -1  0.452264  \n","18   -1  0.352622  \n","19   -1  0.457470  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4629221646146812\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.10345345335826106\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4339745608194272\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.1085988279815744\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7972990634525781\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08271557351300213\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7865153979282162\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032970730509141485\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141873 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141873 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6384607120105854\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.43594104866930905\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8112343384760854\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03923384142208724\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.923861235859889\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04605658286807627\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.8854828326496772\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05999439983594186\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6943135258367088\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.007552822697389583\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6732554405183395\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.09322880664492571\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7241866143836198\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03437239853261358\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4170054573359787\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.13184977711440832\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6716728632780766\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6231288855834671\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141808 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5330125310344216\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.3278490338942788\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.7168669186567422\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09509061015477947\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141855 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141855 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45532879939032533\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.056736175097502985\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7838218389874414\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04925586171197588\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8438978768706994\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.008206528672487046\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7645870624905057\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.201989034883125e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9175932083390175\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=7.033696489705684e-05\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.7_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.7_num-causals=10_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732   0.998598 -1.161807          1\n","1     HG01052   0.831978 -0.628439          1\n","2     HG01064   1.410110  0.117391          1\n","3     HG01069   1.414590 -0.729677          1\n","4     HG01079   2.740260  0.950227          1\n","...       ...        ...       ...        ...\n","1095  NA19734   4.646740 -0.856961         20\n","1096  NA19741   4.935180  0.614434         20\n","1097  NA19752  -1.907870 -0.053596         20\n","1098  NA19759  11.184600  0.400326         20\n","1099  NA19770  17.075700 -0.014717         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.7           10          1  0.462922   \n","1                         AMR     -1  0.7           10          2  0.433975   \n","2                         AMR     -1  0.7           10          3  0.797299   \n","3                         AMR     -1  0.7           10          4  0.786515   \n","4                         AMR     -1  0.7           10          5  0.638461   \n","5                         AMR     -1  0.7           10          6  0.811234   \n","6                         AMR     -1  0.7           10          7  0.923861   \n","7                         AMR     -1  0.7           10          8  0.885483   \n","8                         AMR     -1  0.7           10          9  0.694314   \n","9                         AMR     -1  0.7           10         10  0.673255   \n","10                        AMR     -1  0.7           10         11  0.724187   \n","11                        AMR     -1  0.7           10         12  0.417005   \n","12                        AMR     -1  0.7           10         13  0.671673   \n","13                        AMR     -1  0.7           10         14  0.533013   \n","14                        AMR     -1  0.7           10         15  0.716867   \n","15                        AMR     -1  0.7           10         16  0.455329   \n","16                        AMR     -1  0.7           10         17  0.783822   \n","17                        AMR     -1  0.7           10         18  0.843898   \n","18                        AMR     -1  0.7           10         19  0.764587   \n","19                        AMR     -1  0.7           10         20  0.917593   \n","\n","    val      test  \n","0    -1  0.103453  \n","1    -1  0.108599  \n","2    -1  0.082716  \n","3    -1  0.003297  \n","4    -1  0.435941  \n","5    -1  0.039234  \n","6    -1  0.046057  \n","7    -1  0.059994  \n","8    -1  0.007553  \n","9    -1  0.093229  \n","10   -1  0.034372  \n","11   -1  0.131850  \n","12   -1  0.623129  \n","13   -1  0.327849  \n","14   -1  0.095091  \n","15   -1  0.056736  \n","16   -1  0.049256  \n","17   -1  0.008207  \n","18   -1  0.000012  \n","19   -1  0.000070  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141764 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"metadata":{"tags":null},"name":"stdout","output_type":"stream","text":["0.598807401363031\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07510731889077443\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8742251065978511\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.14983957062673003\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141775 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46961674460346575\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.12214989291897047\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141794 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45176849350209924\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.020381940294921342\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9192817750648263\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06132545114939693\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6570978276014375\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0004163395985318297\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9067446353166231\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01035545978617519\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141731 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8827922256959172\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06599196705171204\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6093663389284717\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.049205021371640824\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141787 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45508472547751067\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.17249963270809887\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8375493634369932\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003813465585169661\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141839 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141839 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.41424545249686556\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.17214303973143028\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"metadata":{"tags":null},"name":"stderr","output_type":"stream","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"output_type":"stream","name":"stdout","text":["0.8998863143365514\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.2421502447833505\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141772 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.510101466439929\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.31898862008037165\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6880338554582749\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.14542207680017488\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5783823183993688\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.11894982246154633\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141813 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141813 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5315939252896577\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.038456987565097314\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7795010635238829\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03114968865038675\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5607030425408653\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.06616554510980027\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141802 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7361775030757931\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04688185831638921\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.7_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.7_num-causals=100_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  48.175000  1.340105          1\n","1     HG01052  34.426600 -0.649956          1\n","2     HG01064  28.839300 -1.152701          1\n","3     HG01069  10.292200 -1.657362          1\n","4     HG01079  50.138500  0.990140          1\n","...       ...        ...       ...        ...\n","1095  NA19734   0.934474 -0.215544         20\n","1096  NA19741  13.772600  0.934609         20\n","1097  NA19752 -36.250800 -0.252540         20\n","1098  NA19759  25.538700 -0.996952         20\n","1099  NA19770 -18.352600  0.005501         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.7          100          1  0.598807   \n","1                         AMR     -1  0.7          100          2  0.874225   \n","2                         AMR     -1  0.7          100          3  0.469617   \n","3                         AMR     -1  0.7          100          4  0.451768   \n","4                         AMR     -1  0.7          100          5  0.919282   \n","5                         AMR     -1  0.7          100          6  0.657098   \n","6                         AMR     -1  0.7          100          7  0.906745   \n","7                         AMR     -1  0.7          100          8  0.882792   \n","8                         AMR     -1  0.7          100          9  0.609366   \n","9                         AMR     -1  0.7          100         10  0.455085   \n","10                        AMR     -1  0.7          100         11  0.837549   \n","11                        AMR     -1  0.7          100         12  0.414245   \n","12                        AMR     -1  0.7          100         13  0.899886   \n","13                        AMR     -1  0.7          100         14  0.510101   \n","14                        AMR     -1  0.7          100         15  0.688034   \n","15                        AMR     -1  0.7          100         16  0.578382   \n","16                        AMR     -1  0.7          100         17  0.531594   \n","17                        AMR     -1  0.7          100         18  0.779501   \n","18                        AMR     -1  0.7          100         19  0.560703   \n","19                        AMR     -1  0.7          100         20  0.736178   \n","\n","    val      test  \n","0    -1  0.075107  \n","1    -1  0.149840  \n","2    -1  0.122150  \n","3    -1  0.020382  \n","4    -1  0.061325  \n","5    -1  0.000416  \n","6    -1  0.010355  \n","7    -1  0.065992  \n","8    -1  0.049205  \n","9    -1  0.172500  \n","10   -1  0.003813  \n","11   -1  0.172143  \n","12   -1  0.242150  \n","13   -1  0.318989  \n","14   -1  0.145422  \n","15   -1  0.118950  \n","16   -1  0.038457  \n","17   -1  0.031150  \n","18   -1  0.066166  \n","19   -1  0.046882  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5787986925074666\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.001962579740789786\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7334071318494153\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0019181562103318271\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.709861592852988\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.046531223181098716\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8774024991377243\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04712339600437659\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.891000997539529\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030571554911601722\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141839 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141839 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4171449423442707\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09900670569603114\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6500220417539394\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0668018648497818\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141706 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6496734114456771\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.012329704996607209\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7409345291933641\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.051016229565041295\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141810 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6544123497337216\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.14610952291036627\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5810511065231969\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013723729926876656\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.895990314912403\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03032961094930607\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.933675583572025\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06911076606908041\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141701 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141701 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8895128348730873\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0543935949232318\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7030506511118149\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04358717865401318\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141733 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7465933387049098\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.028388337777605622\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8507891288877016\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0020259048209188774\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.883815021126758\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05619820343137351\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141723 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.847415521566199\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01945595592509334\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8034578177712033\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.034184167435898354\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.7_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.7_num-causals=250_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  33.336200  1.748489          1\n","1     HG01052  13.754500 -1.564939          1\n","2     HG01064  19.460700 -0.203629          1\n","3     HG01069  13.740700 -0.026826          1\n","4     HG01079  41.136600  0.684538          1\n","...       ...        ...       ...        ...\n","1095  NA19734  -0.859213 -1.187577         20\n","1096  NA19741 -29.895600  1.322055         20\n","1097  NA19752  29.152800 -0.033973         20\n","1098  NA19759 -22.222200 -0.417020         20\n","1099  NA19770  13.861600 -0.095466         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.7          250          1  0.578799   \n","1                         AMR     -1  0.7          250          2  0.733407   \n","2                         AMR     -1  0.7          250          3  0.709862   \n","3                         AMR     -1  0.7          250          4  0.877402   \n","4                         AMR     -1  0.7          250          5  0.891001   \n","5                         AMR     -1  0.7          250          6  0.417145   \n","6                         AMR     -1  0.7          250          7  0.650022   \n","7                         AMR     -1  0.7          250          8  0.649673   \n","8                         AMR     -1  0.7          250          9  0.740935   \n","9                         AMR     -1  0.7          250         10  0.654412   \n","10                        AMR     -1  0.7          250         11  0.581051   \n","11                        AMR     -1  0.7          250         12  0.895990   \n","12                        AMR     -1  0.7          250         13  0.933676   \n","13                        AMR     -1  0.7          250         14  0.889513   \n","14                        AMR     -1  0.7          250         15  0.703051   \n","15                        AMR     -1  0.7          250         16  0.746593   \n","16                        AMR     -1  0.7          250         17  0.850789   \n","17                        AMR     -1  0.7          250         18  0.883815   \n","18                        AMR     -1  0.7          250         19  0.847416   \n","19                        AMR     -1  0.7          250         20  0.803458   \n","\n","    val      test  \n","0    -1  0.001963  \n","1    -1  0.001918  \n","2    -1  0.046531  \n","3    -1  0.047123  \n","4    -1  0.030572  \n","5    -1  0.099007  \n","6    -1  0.066802  \n","7    -1  0.012330  \n","8    -1  0.051016  \n","9    -1  0.146110  \n","10   -1  0.013724  \n","11   -1  0.030330  \n","12   -1  0.069111  \n","13   -1  0.054394  \n","14   -1  0.043587  \n","15   -1  0.028388  \n","16   -1  0.002026  \n","17   -1  0.056198  \n","18   -1  0.019456  \n","19   -1  0.034184  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8596346842963352\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.04169846101825911\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141734 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8658215551710701\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014103397645436744\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141755 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9028991449129129\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04964956798351034\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8573098187980198\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0032657589642576612\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9230385149572651\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00044981073792331656\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8538754041410888\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09725020009863478\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141766 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8326549408338083\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03146778203334165\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8563607276499259\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.011523047664498924\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141781 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8869238460856715\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06990644435072871\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8833387812318371\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05024109026671952\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8892678041249691\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.02435534264424724\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141716 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141716 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9257133710258583\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.005485388759482081\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141719 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7933090738937899\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03180227368928441\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141757 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9158153513609036\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04467997363226497\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141779 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8207573794486199\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.047832484056473556\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141722 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9140433134770088\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009004258947335033\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141739 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8613783392857537\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.037853545965889426\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6375723511581665\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03484637899968036\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"output_type":"stream","name":"stdout","text":["0.8831101461613146\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0006726850029018885\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141776 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8915233415021755\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03807271903365554\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.7_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.7_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  -4.26226  0.608461          1\n","1     HG01052 -10.61690 -3.260704          1\n","2     HG01064 -18.45570  0.004789          1\n","3     HG01069  -4.86909 -0.430276          1\n","4     HG01079 -11.37320 -0.251122          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -3.10530 -0.424294         20\n","1096  NA19741   2.79443 -0.204009         20\n","1097  NA19752   5.49639  0.222701         20\n","1098  NA19759  12.10790  0.942121         20\n","1099  NA19770   5.40653  0.351564         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.7          512          1  0.859635   \n","1                         AMR     -1  0.7          512          2  0.865822   \n","2                         AMR     -1  0.7          512          3  0.902899   \n","3                         AMR     -1  0.7          512          4  0.857310   \n","4                         AMR     -1  0.7          512          5  0.923039   \n","5                         AMR     -1  0.7          512          6  0.853875   \n","6                         AMR     -1  0.7          512          7  0.832655   \n","7                         AMR     -1  0.7          512          8  0.856361   \n","8                         AMR     -1  0.7          512          9  0.886924   \n","9                         AMR     -1  0.7          512         10  0.883339   \n","10                        AMR     -1  0.7          512         11  0.889268   \n","11                        AMR     -1  0.7          512         12  0.925713   \n","12                        AMR     -1  0.7          512         13  0.793309   \n","13                        AMR     -1  0.7          512         14  0.915815   \n","14                        AMR     -1  0.7          512         15  0.820757   \n","15                        AMR     -1  0.7          512         16  0.914043   \n","16                        AMR     -1  0.7          512         17  0.861378   \n","17                        AMR     -1  0.7          512         18  0.637572   \n","18                        AMR     -1  0.7          512         19  0.883110   \n","19                        AMR     -1  0.7          512         20  0.891523   \n","\n","    val      test  \n","0    -1  0.041698  \n","1    -1  0.014103  \n","2    -1  0.049650  \n","3    -1  0.003266  \n","4    -1  0.000450  \n","5    -1  0.097250  \n","6    -1  0.031468  \n","7    -1  0.011523  \n","8    -1  0.069906  \n","9    -1  0.050241  \n","10   -1  0.024355  \n","11   -1  0.005485  \n","12   -1  0.031802  \n","13   -1  0.044680  \n","14   -1  0.047832  \n","15   -1  0.009004  \n","16   -1  0.037854  \n","17   -1  0.034846  \n","18   -1  0.000673  \n","19   -1  0.038073  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141728 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6833860118759243\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0017959110900737992\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5362491500446428\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.0004209951898929949\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141756 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7147639975024723\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0016399437817798115\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.662283085816983\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06049531465403438\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141740 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6545154330160344\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03977825388068103\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141745 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7829539553436802\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8394844390088537\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6243363309130132\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4384725899765496\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141815 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7893435742589949\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003473282324846464\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141795 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6787000047799028\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.5439649069535968\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7320590453421657\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.8323577883000033\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n","/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"output_type":"stream","name":"stdout","text":["0.7373529756521396\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03475278645505974\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141862 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141862 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5109384228728294\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.18177682039811138\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141750 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9516809292140954\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.9671670263410473\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"output_type":"stream","name":"stdout","text":["0.7329263562880528\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.013718870489452148\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141817 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7342884916743347\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.649748359712963\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5653386588311069\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.17107947013252234\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7672860038790534\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6976782665070909\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7073108216948075\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.44548897407657306\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6539494780316282\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.36952681647857677\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141796 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6473775759568716\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6673013132745127\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.9_num-causals=1_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.9_num-causals=1_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  55.004200 -0.033580          1\n","1     HG01052  40.602300  0.560574          1\n","2     HG01064  33.413400 -0.146600          1\n","3     HG01069  22.484300  0.216360          1\n","4     HG01079  55.653000 -0.695851          1\n","...       ...        ...       ...        ...\n","1095  NA19734   1.759060  0.891737         20\n","1096  NA19741   0.866931  1.154377         20\n","1097  NA19752  -3.966740 -1.832428         20\n","1098  NA19759   0.540457  1.218987         20\n","1099  NA19770   1.872400  0.593821         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.9            1          1  0.683386   \n","1                         AMR     -1  0.9            1          2  0.536249   \n","2                         AMR     -1  0.9            1          3  0.714764   \n","3                         AMR     -1  0.9            1          4  0.662283   \n","4                         AMR     -1  0.9            1          5  0.654515   \n","5                         AMR     -1  0.9            1          6  0.782954   \n","6                         AMR     -1  0.9            1          7  0.624336   \n","7                         AMR     -1  0.9            1          8  0.789344   \n","8                         AMR     -1  0.9            1          9  0.678700   \n","9                         AMR     -1  0.9            1         10  0.732059   \n","10                        AMR     -1  0.9            1         11  0.737353   \n","11                        AMR     -1  0.9            1         12  0.510938   \n","12                        AMR     -1  0.9            1         13  0.951681   \n","13                        AMR     -1  0.9            1         14  0.732926   \n","14                        AMR     -1  0.9            1         15  0.734288   \n","15                        AMR     -1  0.9            1         16  0.565339   \n","16                        AMR     -1  0.9            1         17  0.767286   \n","17                        AMR     -1  0.9            1         18  0.707311   \n","18                        AMR     -1  0.9            1         19  0.653949   \n","19                        AMR     -1  0.9            1         20  0.647378   \n","\n","    val      test  \n","0    -1  0.001796  \n","1    -1  0.000421  \n","2    -1  0.001640  \n","3    -1  0.060495  \n","4    -1  0.039778  \n","5    -1  0.839484  \n","6    -1  0.438473  \n","7    -1  0.003473  \n","8    -1  0.543965  \n","9    -1  0.832358  \n","10   -1  0.034753  \n","11   -1  0.181777  \n","12   -1  0.967167  \n","13   -1  0.013719  \n","14   -1  0.649748  \n","15   -1  0.171079  \n","16   -1  0.697678  \n","17   -1  0.445489  \n","18   -1  0.369527  \n","19   -1  0.667301  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141829 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141829 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.48137019050725205\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.04537568811270453\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.46140941862921425\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1012636304561336\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141759 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8973696422026657\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=2.394628560160491e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7810277207410387\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06068241034824947\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6435849695531979\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.41429033859391434\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6136791581037816\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.18531376505499186\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9244170745404113\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00910561989771339\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.860910144262895\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0766004417263747\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141748 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7108389527130106\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02061570606299176\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5491540266875415\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07466231327928467\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.796242310220994\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=4.32517432570771e-05\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141841 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141841 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4544963530133978\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.10829208372808961\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141805 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7117990613187906\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.6082662581095406\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141821 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6495223057661804\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.4362793293983537\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6600982956345219\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.1209745566665715\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141799 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.47050660636066594\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.13110952042268795\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7101852482919876\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02973585169537127\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141773 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8646276094085369\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00034250179833182187\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141827 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6935528101065045\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.5022655678715381\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141710 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141710 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9403350142553294\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=1.529315405921787e-05\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.9_num-causals=10_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.9_num-causals=10_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732   5.81022 -0.251069          1\n","1     HG01052   6.62659 -0.491385          1\n","2     HG01064   3.07878  0.104101          1\n","3     HG01069   1.93359 -0.045691          1\n","4     HG01079   6.35397 -1.138669          1\n","...       ...       ...       ...        ...\n","1095  NA19734  16.99980  0.204176         20\n","1096  NA19741  15.55370  0.368839         20\n","1097  NA19752  18.09800  0.339787         20\n","1098  NA19759  16.13870 -0.020641         20\n","1099  NA19770  20.24930  0.547772         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.9           10          1  0.481370   \n","1                         AMR     -1  0.9           10          2  0.461409   \n","2                         AMR     -1  0.9           10          3  0.897370   \n","3                         AMR     -1  0.9           10          4  0.781028   \n","4                         AMR     -1  0.9           10          5  0.643585   \n","5                         AMR     -1  0.9           10          6  0.613679   \n","6                         AMR     -1  0.9           10          7  0.924417   \n","7                         AMR     -1  0.9           10          8  0.860910   \n","8                         AMR     -1  0.9           10          9  0.710839   \n","9                         AMR     -1  0.9           10         10  0.549154   \n","10                        AMR     -1  0.9           10         11  0.796242   \n","11                        AMR     -1  0.9           10         12  0.454496   \n","12                        AMR     -1  0.9           10         13  0.711799   \n","13                        AMR     -1  0.9           10         14  0.649522   \n","14                        AMR     -1  0.9           10         15  0.660098   \n","15                        AMR     -1  0.9           10         16  0.470507   \n","16                        AMR     -1  0.9           10         17  0.710185   \n","17                        AMR     -1  0.9           10         18  0.864628   \n","18                        AMR     -1  0.9           10         19  0.693553   \n","19                        AMR     -1  0.9           10         20  0.940335   \n","\n","    val      test  \n","0    -1  0.045376  \n","1    -1  0.101264  \n","2    -1  0.000024  \n","3    -1  0.060682  \n","4    -1  0.414290  \n","5    -1  0.185314  \n","6    -1  0.009106  \n","7    -1  0.076600  \n","8    -1  0.020616  \n","9    -1  0.074662  \n","10   -1  0.000043  \n","11   -1  0.108292  \n","12   -1  0.608266  \n","13   -1  0.436279  \n","14   -1  0.120975  \n","15   -1  0.131110  \n","16   -1  0.029736  \n","17   -1  0.000343  \n","18   -1  0.502266  \n","19   -1  0.000015  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141736 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6073164664040633\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01041372942271718\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141762 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8892755777849046\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.1680450745568588\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141784 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.559212168866312\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.2854931977297432\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141790 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.45126941258244824\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.009790808245061297\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141761 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8963786631787061\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.16302159198566737\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141770 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5965574852501649\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.028328977414452842\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141680 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141680 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7477521624355304\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.003585826741874981\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141717 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141717 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7212701954974372\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0144888131885153\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141727 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141727 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7760503268243485\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.15063675293099463\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141752 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5060507760704994\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.21240048274160658\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8450908228853123\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03934942450142424\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141800 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.3961017738887464\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.1525409034450928\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8811578734468931\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.14595373205194742\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5968445974667526\n","Validation: \n","Best:  0.000001 -1\n","Testing: \n","pval=0.000001, R2=0.46085074718112057\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141806 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.569595987852976\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08130274272384358\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141742 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6869235695557692\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08479833587116017\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141818 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5676283902447131\n","Validation: \n","Best:  0.00001 -1\n","Testing: \n","pval=0.00001, R2=0.23732632298120168\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8675681616465224\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03376000209187779\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5540595462317887\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.07820610516912081\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141811 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8668413215749523\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04513165921158216\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.9_num-causals=100_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.9_num-causals=100_pheno=1-20.csv\n","          IID    predict    actual  pheno_num\n","0     HG00732  34.490100  2.631737          1\n","1     HG01052  27.553300 -0.749258          1\n","2     HG01064  19.497400 -0.112811          1\n","3     HG01069   0.708801 -1.565981          1\n","4     HG01079  40.914300  1.061480          1\n","...       ...        ...       ...        ...\n","1095  NA19734 -17.486800 -0.276593         20\n","1096  NA19741 -18.743700  1.517305         20\n","1097  NA19752 -15.558500 -0.325964         20\n","1098  NA19759  -6.534290 -0.724665         20\n","1099  NA19770 -24.769500  0.328467         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.9          100          1  0.607316   \n","1                         AMR     -1  0.9          100          2  0.889276   \n","2                         AMR     -1  0.9          100          3  0.559212   \n","3                         AMR     -1  0.9          100          4  0.451269   \n","4                         AMR     -1  0.9          100          5  0.896379   \n","5                         AMR     -1  0.9          100          6  0.596557   \n","6                         AMR     -1  0.9          100          7  0.747752   \n","7                         AMR     -1  0.9          100          8  0.721270   \n","8                         AMR     -1  0.9          100          9  0.776050   \n","9                         AMR     -1  0.9          100         10  0.506051   \n","10                        AMR     -1  0.9          100         11  0.845091   \n","11                        AMR     -1  0.9          100         12  0.396102   \n","12                        AMR     -1  0.9          100         13  0.881158   \n","13                        AMR     -1  0.9          100         14  0.596845   \n","14                        AMR     -1  0.9          100         15  0.569596   \n","15                        AMR     -1  0.9          100         16  0.686924   \n","16                        AMR     -1  0.9          100         17  0.567628   \n","17                        AMR     -1  0.9          100         18  0.867568   \n","18                        AMR     -1  0.9          100         19  0.554060   \n","19                        AMR     -1  0.9          100         20  0.866841   \n","\n","    val      test  \n","0    -1  0.010414  \n","1    -1  0.168045  \n","2    -1  0.285493  \n","3    -1  0.009791  \n","4    -1  0.163022  \n","5    -1  0.028329  \n","6    -1  0.003586  \n","7    -1  0.014489  \n","8    -1  0.150637  \n","9    -1  0.212400  \n","10   -1  0.039349  \n","11   -1  0.152541  \n","12   -1  0.145954  \n","13   -1  0.460851  \n","14   -1  0.081303  \n","15   -1  0.084798  \n","16   -1  0.237326  \n","17   -1  0.033760  \n","18   -1  0.078206  \n","19   -1  0.045132  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6120457356515104\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02430135409991383\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141804 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6967185305637831\n","Validation: \n","Best:  0.001 -1\n","Testing: \n","pval=0.001, R2=0.12158434288244323\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141824 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.4801796534344593\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.10263655442176262\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141751 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8283389885842329\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06278425114903326\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141712 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141712 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8375086109018697\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08637491248033204\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141846 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.49740878039824915\n","Validation: \n","Best:  0.0001 -1\n","Testing: \n","pval=0.0001, R2=0.1909937186049243\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141726 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6206955285781076\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.023227859135106606\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141725 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141725 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5989395554822994\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0008998564585348097\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141747 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7178673529020383\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03917572915889323\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141833 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.60393501999174\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08100151242856397\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141798 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.5779495129093749\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031192624607560938\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141771 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8773242392521744\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.019369102708867384\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9094912237557773\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.10732733711785926\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141732 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8742683678917872\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.047360397340866195\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141763 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.6436181515729128\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06385712971400108\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/scipy/stats/_stats_py.py:4781: ConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n","  warnings.warn(stats.ConstantInputWarning(msg))\n"]},{"output_type":"stream","name":"stdout","text":["0.8854971693650642\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.06604135155314461\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141724 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141724 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8765071909163107\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.00034806272030139595\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141791 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.872581875028852\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.1372800317990429\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141760 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8246838937490616\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.02585429372919684\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141788 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7665964195138997\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0492344643432613\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.9_num-causals=250_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.9_num-causals=250_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  40.77270  1.493736          1\n","1     HG01052  23.02620 -1.522665          1\n","2     HG01064  14.11790 -1.241989          1\n","3     HG01069  10.26370 -0.391749          1\n","4     HG01079  45.50560  0.739235          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -1.65791 -0.338403         20\n","1096  NA19741 -22.45700  0.475752         20\n","1097  NA19752  30.29160 -0.100797         20\n","1098  NA19759 -21.84340 -0.680704         20\n","1099  NA19770  19.99390  0.569435         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.9          250          1  0.612046   \n","1                         AMR     -1  0.9          250          2  0.696719   \n","2                         AMR     -1  0.9          250          3  0.480180   \n","3                         AMR     -1  0.9          250          4  0.828339   \n","4                         AMR     -1  0.9          250          5  0.837509   \n","5                         AMR     -1  0.9          250          6  0.497409   \n","6                         AMR     -1  0.9          250          7  0.620696   \n","7                         AMR     -1  0.9          250          8  0.598940   \n","8                         AMR     -1  0.9          250          9  0.717867   \n","9                         AMR     -1  0.9          250         10  0.603935   \n","10                        AMR     -1  0.9          250         11  0.577950   \n","11                        AMR     -1  0.9          250         12  0.877324   \n","12                        AMR     -1  0.9          250         13  0.909491   \n","13                        AMR     -1  0.9          250         14  0.874268   \n","14                        AMR     -1  0.9          250         15  0.643618   \n","15                        AMR     -1  0.9          250         16  0.885497   \n","16                        AMR     -1  0.9          250         17  0.876507   \n","17                        AMR     -1  0.9          250         18  0.872582   \n","18                        AMR     -1  0.9          250         19  0.824684   \n","19                        AMR     -1  0.9          250         20  0.766596   \n","\n","    val      test  \n","0    -1  0.024301  \n","1    -1  0.121584  \n","2    -1  0.102637  \n","3    -1  0.062784  \n","4    -1  0.086375  \n","5    -1  0.190994  \n","6    -1  0.023228  \n","7    -1  0.000900  \n","8    -1  0.039176  \n","9    -1  0.081002  \n","10   -1  0.031193  \n","11   -1  0.019369  \n","12   -1  0.107327  \n","13   -1  0.047360  \n","14   -1  0.063857  \n","15   -1  0.066041  \n","16   -1  0.000348  \n","17   -1  0.137280  \n","18   -1  0.025854  \n","19   -1  0.049234  \n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141737 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.7914741036802806\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.01577104273704233\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141715 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9195026523632478\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04917996508217359\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141831 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8828477565168439\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05244658612497103\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141780 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8174558274803616\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08017199019927884\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141816 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8880154843497756\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03613147846308299\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141789 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.905899675354051\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.014598682757155267\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141758 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8077870971781262\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.024697166231049242\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141767 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.893018959257029\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.030378350645989594\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141777 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.842042177819589\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.03295673896826527\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141741 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8274599163342675\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031189267622895457\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141774 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9016706784718863\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.07943766188919861\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141782 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8692077726717776\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.017136274549303086\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141753 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8786142212721048\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.04717918674651798\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141786 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9176238953741268\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.05652244283430511\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141783 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8712768647452561\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.09982121095852833\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141765 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8564677210786114\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.031754866714248875\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141792 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.88990697153884\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.08688060391214848\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141744 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8841200495500985\n","Validation: \n","Best:  0.05 -1\n","Testing: \n","pval=0.05, R2=0.0112269237136658\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141778 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.8999383503444892\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.00447149980693129\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Error: This run requires decent allele frequencies, but they aren't being\n","loaded with --read-freq, and less than 50 samples are available to impute them\n","from.\n","You should generate (with --freq) or obtain an allele frequency file based on a\n","larger similar-population reference dataset, and load it with --read-freq.\n","Warning: 1 line skipped in --q-score-range data file.\n","Warning: 141769 --score file entries were skipped due to missing variant IDs.\n","(Add the 'list-variants' modifier to see which variants were actually used for\n","scoring.)\n","Training: \n","0.9147984434036016\n","Validation: \n","Best:  0.1 -1\n","Testing: \n","pval=0.1, R2=0.0692507245649325\n","Combined prediction data has been written to: combined_predict_AMR_power=-1_her=0.9_num-causals=512_pheno=1-20.csv\n","Combined result data has been written to: combined_result_AMR_power=-1_her=0.9_num-causals=512_pheno=1-20.csv\n","          IID   predict    actual  pheno_num\n","0     HG00732  20.34720  0.190761          1\n","1     HG01052  11.89800 -2.476303          1\n","2     HG01064  -3.32182 -0.926528          1\n","3     HG01069  13.52760 -0.984452          1\n","4     HG01079  20.56130 -0.627941          1\n","...       ...       ...       ...        ...\n","1095  NA19734  -6.26558 -1.672316         20\n","1096  NA19741  -7.20459  0.366708         20\n","1097  NA19752  -3.24545 -0.480351         20\n","1098  NA19759   1.93403  0.418198         20\n","1099  NA19770   4.29576  0.718949         20\n","\n","[1100 rows x 4 columns]\n","   population/superpopulation  power  her  num_causals  pheno_num     train  \\\n","0                         AMR     -1  0.9          512          1  0.791474   \n","1                         AMR     -1  0.9          512          2  0.919503   \n","2                         AMR     -1  0.9          512          3  0.882848   \n","3                         AMR     -1  0.9          512          4  0.817456   \n","4                         AMR     -1  0.9          512          5  0.888015   \n","5                         AMR     -1  0.9          512          6  0.905900   \n","6                         AMR     -1  0.9          512          7  0.807787   \n","7                         AMR     -1  0.9          512          8  0.893019   \n","8                         AMR     -1  0.9          512          9  0.842042   \n","9                         AMR     -1  0.9          512         10  0.827460   \n","10                        AMR     -1  0.9          512         11  0.901671   \n","11                        AMR     -1  0.9          512         12  0.869208   \n","12                        AMR     -1  0.9          512         13  0.878614   \n","13                        AMR     -1  0.9          512         14  0.917624   \n","14                        AMR     -1  0.9          512         15  0.871277   \n","15                        AMR     -1  0.9          512         16  0.856468   \n","16                        AMR     -1  0.9          512         17  0.889907   \n","17                        AMR     -1  0.9          512         18  0.884120   \n","18                        AMR     -1  0.9          512         19  0.899938   \n","19                        AMR     -1  0.9          512         20  0.914798   \n","\n","    val      test  \n","0    -1  0.015771  \n","1    -1  0.049180  \n","2    -1  0.052447  \n","3    -1  0.080172  \n","4    -1  0.036131  \n","5    -1  0.014599  \n","6    -1  0.024697  \n","7    -1  0.030378  \n","8    -1  0.032957  \n","9    -1  0.031189  \n","10   -1  0.079438  \n","11   -1  0.017136  \n","12   -1  0.047179  \n","13   -1  0.056522  \n","14   -1  0.099821  \n","15   -1  0.031755  \n","16   -1  0.086881  \n","17   -1  0.011227  \n","18   -1  0.004471  \n","19   -1  0.069251  \n"]}],"source":["! cp /content/drive/MyDrive/CSE-284-Final-Project/data/1000g_by_chrom/chr19.* .\n","\n","PREFIX = '/content'\n","chr19 = f'{PREFIX}/chr19'\n","\n","! echo \"0.000001 0 0.000001\" > range_list\n","! echo \"0.00001 0 0.00001\" >> range_list\n","! echo \"0.0001 0 0.0001\" >> range_list\n","! echo \"0.001 0 0.001\" >> range_list\n","! echo \"0.05 0 0.05\" >> range_list\n","! echo \"0.1 0 0.1\" >> range_list\n","\n","for POP in POP_LIST:\n","  ! cp /content/drive/MyDrive/CSE-284-Final-Project/data/split_ids/by_superpopulation/{POP}_* .\n","  ! cut -d ' ' -f 2 {POP}_train.txt > {POP}_train_IID_only.txt\n","  ! cut -d ' ' -f 2 {POP}_val.txt > {POP}_val_IID_only.txt\n","  ! cut -d ' ' -f 2 {POP}_test.txt > {POP}_test_IID_only.txt\n","\n","  keep_train = f'{POP}_train_IID_only.txt'\n","  out_train = f'{POP}_train'\n","\n","  keep_val = f'{POP}_val_IID_only.txt'\n","  out_val = f'{POP}_val'\n","\n","  keep_test = f'{POP}_test_IID_only.txt'\n","  out_test = f'{POP}_test'\n","\n","  # Make bed\n","\n","  sh = f'''\n","  ./plink2 \\\n","    --make-bed \\\n","    --bfile {chr19} \\\n","    --keep {keep_train} \\\n","    --out {out_train}\n","\n","  ./plink2 \\\n","    --make-bed \\\n","    --bfile {chr19} \\\n","    --keep {keep_val} \\\n","    --out {out_val}\n","\n","  ./plink2 \\\n","    --make-bed \\\n","    --bfile {chr19} \\\n","    --keep {keep_test} \\\n","    --out {out_test}\n","\n","  '''\n","  with open('makebed.sh', 'w') as file:\n","      file.write(sh)\n","  !bash makebed.sh\n","\n","  GENO_TRAIN = f'{PREFIX}/{POP}_train'\n","  GENO_VAL = f'{PREFIX}/{POP}_val'\n","  GENO_TEST = f'{PREFIX}/{POP}_test'\n","\n","  for POWER in POWER_LIST:\n","    for HER in HER_LIST:\n","      for NUM_CAUSALS in NUM_CAUSALS_LIST:\n","        ! cp /content/drive/MyDrive/CSE-284-Final-Project/data/chr19_ldl_pheno/power={POWER}_her={HER}_num-causals={NUM_CAUSALS}.pheno .\n","        PHENO = f'{PREFIX}/power={POWER}_her={HER}_num-causals={NUM_CAUSALS}.pheno'\n","\n","        # GWAS (GLM)\n","        sh = f'''\n","        ./plink2 --bfile {GENO_TRAIN} \\\n","                --pheno {PHENO} \\\n","                --glm allow-no-covars \\\n","                --maf 0.05 \\\n","                --out {POP}_gwas \\\n","                --silent\n","        '''\n","\n","        with open('gwas.sh', 'w') as file:\n","            file.write(sh)\n","        !bash gwas.sh\n","\n","\n","        combined_prediction_data = []\n","        combined_result_data = []\n","\n","        for PHENO_NUM in range(PHENO_NUM_START, PHENO_NUM_END + 1):\n","\n","            GLM = f'/content/{POP}_gwas.PHENO{PHENO_NUM}.glm.linear'\n","\n","            # Clumping\n","\n","            sh = f'''\n","            ./plink2 \\\n","                --bfile {GENO_TRAIN} \\\n","                --clump-p1 1 \\\n","                --clump-r2 0.1 \\\n","                --clump-kb 250 \\\n","                --clump {GLM} \\\n","                --out {POP}_gwas.PHENO{PHENO_NUM} \\\n","                --silent\n","\n","            awk 'NR!=1{{print $3}}' {POP}_gwas.PHENO{PHENO_NUM}.clumps > {POP}_gwas.PHENO{PHENO_NUM}.valid.snp\n","            cat {POP}_gwas.PHENO{PHENO_NUM}.clumps | awk '{{print $3 \"\\t\" $4}}' > {POP}_gwas.PHENO{PHENO_NUM}.pvals\n","            '''\n","\n","            with open('clump.sh', 'w') as file:\n","                file.write(sh)\n","            !bash clump.sh\n","\n","            GLM = f'/content/{POP}_gwas.PHENO{PHENO_NUM}.glm.linear'\n","            PVALS = f'/content/{POP}_gwas.PHENO{PHENO_NUM}.pvals'\n","            VAL = f'/content/{POP}_gwas.PHENO{PHENO_NUM}.valid.snp'\n","\n","            # Scoring\n","\n","            sh = f'''\n","\n","            ./plink2 \\\n","            --bfile {GENO_TRAIN} \\\n","            --score {GLM} 3 7 12 header cols=+scoresums\\\n","            --q-score-range range_list {PVALS} \\\n","            --extract {VAL} \\\n","            --out {POP}_PHENO{PHENO_NUM}_train \\\n","            --silent\n","\n","            ./plink2 \\\n","            --bfile {GENO_VAL} \\\n","            --score {GLM} 3 7 12 header cols=+scoresums\\\n","            --q-score-range range_list {PVALS} \\\n","            --extract {VAL} \\\n","            --out {POP}_PHENO{PHENO_NUM}_val \\\n","            --silent\n","\n","            ./plink2 \\\n","            --bfile {GENO_TEST} \\\n","            --score {GLM} 3 7 12 header cols=+scoresums\\\n","            --q-score-range range_list {PVALS} \\\n","            --extract {VAL} \\\n","            --out {POP}_PHENO{PHENO_NUM}_test \\\n","            --silent\n","            '''\n","\n","            with open('score.sh', 'w') as file:\n","              file.write(sh)\n","            !bash score.sh\n","\n","            # Thresholding\n","\n","            pval_list = [\"0.000001\", \"0.00001\", \"0.0001\", \"0.001\", \"0.05\", \"0.1\"]\n","            best_pval_idx = 0\n","            train_scores = []\n","            val_scores = []\n","            test_scores = []\n","            train_score = 0\n","            val_score = 0\n","            test_score = 0\n","\n","            print(\"Training: \")\n","            for pval in pval_list:\n","                score_file = f'{POP}_PHENO{PHENO_NUM}_train.{pval}.sscore'\n","                if os.path.isfile(score_file) == False:\n","                  continue\n","                prs = pd.read_csv(score_file, delim_whitespace=True)\n","                phen = pd.read_csv(f'{PHENO}',\n","                              delim_whitespace=True, usecols=[0, 1, PHENO_NUM + 1], names=[\"FID\",\"IID\",\"phen\"])\n","                d = pd.merge(prs, phen, on=[\"IID\"])\n","                score = scipy.stats.pearsonr(d[\"phen\"], d[\"SCORE1_SUM\"])[0] ** 2\n","                train_scores.append(score)\n","                # print(\"pval=%s, R2=%s\"%(pval, score))\n","            train_score = np.nanmax(train_scores)\n","            print(train_score)\n","\n","            print(\"Validation: \")\n","            for pval in pval_list:\n","                score_file = f'{POP}_PHENO{PHENO_NUM}_val.{pval}.sscore'\n","                if os.path.isfile(score_file) == False:\n","                  val_scores.append(-1)\n","                  continue\n","                prs = pd.read_csv(score_file, delim_whitespace=True)\n","                phen = pd.read_csv(f'{PHENO}',\n","                              delim_whitespace=True, usecols=[0, 1, PHENO_NUM + 1], names=[\"FID\",\"IID\",\"phen\"])\n","                d = pd.merge(prs, phen, on=[\"IID\"])\n","                score = scipy.stats.pearsonr(d[\"phen\"], d[\"SCORE1_SUM\"])[0] ** 2\n","                val_scores.append(score)\n","                # print(\"pval=%s, R2=%s\"%(pval, score))\n","            val_score = np.nanmax(val_scores)\n","            best_pval_idx = val_scores.index(val_score)\n","            # Validation Sample Size < 50: PLINK2 will not calculate the score (error)\n","            if val_score == -1:\n","              best_pval_idx = train_scores.index(train_score)\n","            print(\"Best: \", pval_list[best_pval_idx], val_score)\n","\n","            print(\"Testing: \")\n","            best_pval = pval_list[best_pval_idx]\n","            prs = pd.read_csv(f'{POP}_PHENO{PHENO_NUM}_test.{best_pval}.sscore', delim_whitespace=True)\n","            phen = pd.read_csv(f'{PHENO}',\n","                              delim_whitespace=True, usecols=[0, 1, PHENO_NUM + 1], names=[\"FID\",\"IID\",\"phen\"])\n","            d = pd.merge(prs, phen, on=[\"IID\"])\n","            test_score = scipy.stats.pearsonr(d[\"phen\"], d[\"SCORE1_SUM\"])[0] ** 2\n","            print(\"pval=%s, R2=%s\"%(best_pval, test_score))\n","            predict_df = d[['IID', 'SCORE1_SUM', 'phen']].copy()\n","            predict_df.rename(columns={'IID': 'IID', 'SCORE1_SUM': 'predict', 'phen': 'actual'}, inplace=True)\n","            predict_df[\"pheno_num\"] = PHENO_NUM\n","            combined_prediction_data.append(predict_df)\n","\n","            result_dict = {\n","                \"population/superpopulation\": POP,\n","                \"power\": POWER,\n","                \"her\": HER,\n","                \"num_causals\": NUM_CAUSALS,\n","                \"pheno_num\": PHENO_NUM,\n","                \"train\": train_score,\n","                \"val\": val_score,\n","                \"test\": test_score\n","            }\n","            combined_result_data.append(result_dict)\n","\n","        combined_prediction_df = pd.concat(combined_prediction_data, ignore_index=True)\n","        combined_result_df = pd.DataFrame(combined_result_data)\n","\n","        # Write combined prediction data to file\n","        combined_prediction_file = f'combined_predict_{POP}_power={POWER}_her={HER}_num-causals={NUM_CAUSALS}_pheno={PHENO_NUM_START}-{PHENO_NUM_END}.csv'\n","        combined_prediction_df.to_csv(combined_prediction_file, index=False)\n","        print(\"Combined prediction data has been written to:\", combined_prediction_file)\n","\n","        # Write combined result data to file\n","        combined_result_file = f'combined_result_{POP}_power={POWER}_her={HER}_num-causals={NUM_CAUSALS}_pheno={PHENO_NUM_START}-{PHENO_NUM_END}.csv'\n","        combined_result_df.to_csv(combined_result_file, index=False)\n","        print(\"Combined result data has been written to:\", combined_result_file)\n","\n","        print(combined_prediction_df)\n","        print(combined_result_df)\n","\n","        ! cp combined_predict_{POP}_power={POWER}_her={HER}_num-causals={NUM_CAUSALS}_pheno={PHENO_NUM_START}-{PHENO_NUM_END}.csv /content/drive/MyDrive/CSE-284-Final-Project/data/CT/superpopulation\n","        ! cp combined_result_{POP}_power={POWER}_her={HER}_num-causals={NUM_CAUSALS}_pheno={PHENO_NUM_START}-{PHENO_NUM_END}.csv /content/drive/MyDrive/CSE-284-Final-Project/data/CT/superpopulation\n","\n"]}],"metadata":{"accelerator":"TPU","colab":{"provenance":[],"authorship_tag":"ABX9TyNjV48m9AKIUoDh6kvzmTjb"},"kernelspec":{"display_name":"Python 3","name":"python3"},"language_info":{"name":"python"}},"nbformat":4,"nbformat_minor":0}